## The List of Bioinformatics Ruby Gems

* https://github.com/ruby-on-bioc/bio-gems

## bio

[![Gem Version](https://badge.fury.io/rb/bio.svg)](https://badge.fury.io/rb/bio)
![Year](https://img.shields.io/static/v1?label=date&message=2023-09-27&color=blue)
![Code Size](https://tokei.rs/b1/github/bioruby/bioruby/)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio)

BioRuby is a library for bioinformatics (biology + information science).

* URL: [http://bioruby.org/](http://bioruby.org/)
* Authors: BioRuby project

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio|
|downloads|737877|
|version|2.0.5|
|version_created_at|2023-09-27T14:50:07.538Z|
|version_downloads|50786|
|platform|ruby|
|licenses|["Ruby"]|
|metadata|{}|
|yanked|false|
|sha|36001f30e792ba3e8b0b9ad4ed15941ac5b12a16835c02dc99f3ab34c7455a6b|
|spec_sha|ba00c3e1ff7be2349b1b81bd13d47e31d9a274cbdf220499d3a9436da47105a3|
|project_uri|https://rubygems.org/gems/bio|
|gem_uri|https://rubygems.org/gems/bio-2.0.5.gem|
|wiki_uri|http://bioruby.open-bio.org/wiki/|
|documentation_uri|http://bioruby.org/rdoc/|
|mailing_list_uri|http://lists.open-bio.org/mailman/listinfo/bioruby|
|source_code_uri|https://github.com/bioruby/bioruby/|
|bug_tracker_uri|https://github.com/bioruby/bioruby/issues|
|dependencies|{"development"=>[], "runtime"=>[]}|

</details>

## bio-samtools

[![Gem Version](https://badge.fury.io/rb/bio-samtools.svg)](https://badge.fury.io/rb/bio-samtools)
![Year](https://img.shields.io/static/v1?label=date&message=2018-01-11&color=blue)
![Code Size](https://tokei.rs/b1/github/helios/bioruby-samtools)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-samtools)

Binder of samtools for ruby, on the top of FFI. 

  This project was born from the need to add support of BAM files to 
  the gee_fu genome browser (http://github.com/danmaclean/gee_fu).

* URL: [http://github.com/helios/bioruby-samtools](http://github.com/helios/bioruby-samtools)
* Authors: Ricardo Ramirez-Gonzalez, Dan MacLean, Raoul J.P. Bonnal

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-samtools|
|downloads|123040|
|version|2.6.2|
|version_created_at|2018-01-11T00:55:22.555Z|
|version_downloads|4137|
|platform|ruby|
|licenses|["GPL-3.0"]|
|metadata|{}|
|yanked|false|
|sha|0643ef0f131b29ab786280e68fc639e0db18689ffec77f655e3ab4b5a49420ad|
|spec_sha|6f49b625d00349bcf676f0975135cea74ecf72e0807688846e8d1ff21b56aa28|
|project_uri|https://rubygems.org/gems/bio-samtools|
|gem_uri|https://rubygems.org/gems/bio-samtools-2.6.2.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-samtools/2.6.2|
|dependencies|{"development"=>[{"name"=>"juwelier", "requirements"=>">= 0"}, {"name"=>"rack", "requirements"=>"= 1.6.4"}, {"name"=>"shoulda", "requirements"=>"= 2.10"}, {"name"=>"test-unit", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bio-svgenes", "requirements"=>">= 0.4.1"}]}|

</details>

## bio-gem

[![Gem Version](https://badge.fury.io/rb/bio-gem.svg)](https://badge.fury.io/rb/bio-gem)
![Year](https://img.shields.io/static/v1?label=date&message=2014-05-20&color=blue)
![Code Size](https://tokei.rs/b1/github/helios/bioruby-gem)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-gem)

Biogem is a software generator for those bioinformaticans who want to start coding an application or a library for using/extending BioRuby core library and sharing it through rubygems.org .
  The basic idea is to simplify and promote a modular approach to bioinformatics software development

* URL: [http://github.com/helios/bioruby-gem](http://github.com/helios/bioruby-gem)
* Authors: Raoul J.P. Bonnal, Pjotr Prins

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-gem|
|downloads|97980|
|version|1.3.6|
|version_created_at|2014-05-20T04:31:57.244Z|
|version_downloads|4513|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|a96253804a8ac3a9bce64f560bac5a41a1492d83bf33e73ad72050aa8223649d|
|spec_sha|b05180abf78ae288c56dc61edaae26c1433dbc90080773add1e8e6d29b1a5b8f|
|project_uri|https://rubygems.org/gems/bio-gem|
|gem_uri|https://rubygems.org/gems/bio-gem-1.3.6.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-gem/1.3.6|
|dependencies|{"development"=>[], "runtime"=>[{"name"=>"bundler", "requirements"=>">= 1.6.2, ~> 1.6"}, {"name"=>"jeweler", "requirements"=>">= 2.0.1, ~> 2.0"}, {"name"=>"rdoc", "requirements"=>">= 4.1.1, ~> 4.1.1"}]}|

</details>

## bio-gadget

[![Gem Version](https://badge.fury.io/rb/bio-gadget.svg)](https://badge.fury.io/rb/bio-gadget)
![Year](https://img.shields.io/static/v1?label=date&message=2017-02-01&color=blue)
![Code Size](https://tokei.rs/b1/github/shka/ruby-bio-gadget)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-gadget)

Gadgets for bioinformatics

* URL: [https://github.com/shka/ruby-bio-gadget](https://github.com/shka/ruby-bio-gadget)
* Authors: Shintaro Katayama

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-gadget|
|downloads|83893|
|version|0.5.0|
|version_created_at|2017-02-01T14:25:16.869Z|
|version_downloads|2914|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|0dfe50b252972194227b888ef6b3a4af3bccef986feeb90970b7abf40c2dd08e|
|spec_sha|1184d9248a60e3c69c1572b54ff189c168b89607633946df355cef79c0fb9841|
|project_uri|https://rubygems.org/gems/bio-gadget|
|gem_uri|https://rubygems.org/gems/bio-gadget-0.5.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-gadget/0.5.0|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.12"}, {"name"=>"minitest", "requirements"=>"~> 5.0"}, {"name"=>"rake", "requirements"=>"~> 10.0"}, {"name"=>"rake-compiler", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 0"}, {"name"=>"damerau-levenshtein", "requirements"=>">= 0"}, {"name"=>"mkfifo", "requirements"=>">= 0"}, {"name"=>"parallel", "requirements"=>">= 0"}, {"name"=>"thor", "requirements"=>"~> 0.19.3"}]}|

</details>

## bio-fastqc

[![Gem Version](https://badge.fury.io/rb/bio-fastqc.svg)](https://badge.fury.io/rb/bio-fastqc)
![Year](https://img.shields.io/static/v1?label=date&message=2018-07-26&color=blue)
![Code Size](https://tokei.rs/b1/github/inutano/bioruby-fastqc)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-fastqc)

ruby parser for FastQC, a quality control software for high-throughput sequencing data.

* URL: [http://github.com/inutano/bioruby-fastqc](http://github.com/inutano/bioruby-fastqc)
* Authors: Tazro Inutano Ohta

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-fastqc|
|downloads|61789|
|version|0.10.8|
|version_created_at|2018-07-26T06:15:47.507Z|
|version_downloads|2474|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|fee9b5551f7fe8c0bd3598a23c7b963f55807f0ee3839fb9388d68557aad8d4e|
|spec_sha|3ba43d40926a8f172f4f53cabff8910d239ee18b68a08dfe54ff745899ac26ed|
|project_uri|https://rubygems.org/gems/bio-fastqc|
|gem_uri|https://rubygems.org/gems/bio-fastqc-0.10.8.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-fastqc/0.10.8|
|dependencies|{"development"=>[{"name"=>"bio-fastqc", "requirements"=>">= 0"}, {"name"=>"bundler", "requirements"=>">= 1.8.0"}, {"name"=>"jeweler", "requirements"=>"~> 2.0"}, {"name"=>"pry", "requirements"=>"~> 0.10"}, {"name"=>"rake", "requirements"=>"~> 10.0"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rspec", "requirements"=>"~> 3.3"}, {"name"=>"shoulda", "requirements"=>"~> 3.5"}, {"name"=>"simplecov", "requirements"=>"~> 0.10"}], "runtime"=>[{"name"=>"json-ld", "requirements"=>"~> 1.99"}, {"name"=>"rdf-turtle", "requirements"=>"~> 1.99"}, {"name"=>"rubyzip", "requirements"=>">= 1.1.0, ~> 1.1"}, {"name"=>"thor", "requirements"=>"~> 0.19.1"}]}|

</details>

## bio-svgenes

[![Gem Version](https://badge.fury.io/rb/bio-svgenes.svg)](https://badge.fury.io/rb/bio-svgenes)
![Year](https://img.shields.io/static/v1?label=date&message=2014-01-06&color=blue)
![Code Size](https://tokei.rs/b1/github/danmaclean/bioruby-svgenes)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-svgenes)

This bio-gem facilitates the creation of pretty, publication quality SVG images from feature data.

* URL: [http://github.com/danmaclean/bioruby-svgenes](http://github.com/danmaclean/bioruby-svgenes)
* Authors: Dan MacLean

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-svgenes|
|downloads|61078|
|version|0.4.1|
|version_created_at|2014-01-06T09:45:10.714Z|
|version_downloads|13205|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|86e17e70fe5a5ae0c28896b0897eb5d7815ab74f7f045ebffaa05ad8d2c19eda|
|spec_sha|0626c5fb6f0d8558b5669714d8dcd1d5b1f054440ca5339d7281639303fd5f78|
|project_uri|https://rubygems.org/gems/bio-svgenes|
|gem_uri|https://rubygems.org/gems/bio-svgenes-0.4.1.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-svgenes/0.4.1|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.4"}, {"name"=>"json", "requirements"=>">= 1.7"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-bgzf

[![Gem Version](https://badge.fury.io/rb/bio-bgzf.svg)](https://badge.fury.io/rb/bio-bgzf)
![Year](https://img.shields.io/static/v1?label=date&message=2012-08-06&color=blue)
![Code Size](https://tokei.rs/b1/github/csw/bioruby-bgzf)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-bgzf)

This library provides support for BGZF (Blocked GZip Format) in Ruby. BGZF, originally defined as part of the SAM/BAM specification, is used to compress record-oriented data in a way that facilitates random access, unlike plain gzip. BGZF is principally used for bioinformatics data but would be useful in other contexts as well.

* URL: [http://github.com/csw/bioruby-bgzf](http://github.com/csw/bioruby-bgzf)
* Authors: Artem Tarasov, Clayton Wheeler

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-bgzf|
|downloads|57762|
|version|0.2.1|
|version_created_at|2012-08-06T19:05:37.870Z|
|version_downloads|45982|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/csw/bioruby-bgzf"}|
|yanked|false|
|sha|29556916ce8a4b69d7bd69db7748df04d4c3c1ea94285f70160e4f545608812c|
|spec_sha|1cd14258c71b6ffaa3d340b0aef9b357fe8cb060a1c80035569636333a5c283e|
|project_uri|https://rubygems.org/gems/bio-bgzf|
|gem_uri|https://rubygems.org/gems/bio-bgzf-0.2.1.gem|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.1.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.3"}, {"name"=>"rspec", "requirements"=>"~> 2.10.0"}], "runtime"=>[]}|

</details>

## bio-rocker

[![Gem Version](https://badge.fury.io/rb/bio-rocker.svg)](https://badge.fury.io/rb/bio-rocker)
![Year](https://img.shields.io/static/v1?label=date&message=2019-08-15&color=blue)
![Code Size]()
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-rocker)

Detecting and quantifying functional genes in short-read metagenomic datasets

* URL: [http://enve-omics.ce.gatech.edu/rocker](http://enve-omics.ce.gatech.edu/rocker)
* Authors: Luis (Coto) Orellana, Luis M. Rodriguez-R

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-rocker|
|downloads|56054|
|version|1.5.2|
|version_created_at|2019-08-15T00:50:41.492Z|
|version_downloads|2969|
|platform|ruby|
|licenses|["Artistic-2.0"]|
|metadata|{}|
|yanked|false|
|sha|b500c32eaed5f6fabf64cb384d18ef5a7f03e17e016d46a6b5c8c56ac118119d|
|spec_sha|4b68009cd99214b6414410f5fb5cf3cc91250e1930bed1056d63cdc3dc731962|
|project_uri|https://rubygems.org/gems/bio-rocker|
|gem_uri|https://rubygems.org/gems/bio-rocker-1.5.2.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-rocker/1.5.2|
|dependencies|{"development"=>[], "runtime"=>[{"name"=>"json", "requirements"=>"~> 1.8"}, {"name"=>"rest-client", "requirements"=>"~> 1.7"}]}|

</details>

## bio-maf

[![Gem Version](https://badge.fury.io/rb/bio-maf.svg)](https://badge.fury.io/rb/bio-maf)
![Year](https://img.shields.io/static/v1?label=date&message=2012-08-21&color=blue)
![Code Size](https://tokei.rs/b1/github/csw/bioruby-maf)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-maf)

Multiple Alignment Format parser for BioRuby.

* URL: [http://github.com/csw/bioruby-maf](http://github.com/csw/bioruby-maf)
* Authors: Clayton Wheeler

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-maf|
|downloads|52944|
|version|1.0.1|
|version_created_at|2012-08-21T17:38:13.637Z|
|version_downloads|5462|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/csw/bioruby-maf"}|
|yanked|false|
|sha|b0568a723f04c62211c0eda85baad5a314333ae06d866cd8f4f4b42cf9016f26|
|spec_sha|c7c7716808f578ff96523be68c5a7a5ab9a4673fca8e05f52746123d46ca7843|
|project_uri|https://rubygems.org/gems/bio-maf|
|gem_uri|https://rubygems.org/gems/bio-maf-1.0.1.gem|
|dependencies|{"development"=>[], "runtime"=>[{"name"=>"bio-alignment", "requirements"=>"~> 0.0.7"}, {"name"=>"bio-bgzf", "requirements"=>"~> 0.2.1"}, {"name"=>"bio-genomic-interval", "requirements"=>"~> 0.1.2"}, {"name"=>"bio-logger", "requirements"=>"~> 1.0.1"}, {"name"=>"kyotocabinet-ruby", "requirements"=>"~> 1.27.1"}]}|

</details>

## bio-blastxmlparser

[![Gem Version](https://badge.fury.io/rb/bio-blastxmlparser.svg)](https://badge.fury.io/rb/bio-blastxmlparser)
![Year](https://img.shields.io/static/v1?label=date&message=2015-09-16&color=blue)
![Code Size](https://tokei.rs/b1/github/pjotrp/blastxmlparser)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-blastxmlparser)

Fast big data BLAST XML parser and library; this libxml2 based version is 50x faster than BioRuby and comes with a nice CLI

* URL: [http://github.com/pjotrp/blastxmlparser](http://github.com/pjotrp/blastxmlparser)
* Authors: Pjotr Prins

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-blastxmlparser|
|downloads|52252|
|version|2.0.4|
|version_created_at|2015-09-16T06:44:58.703Z|
|version_downloads|7886|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|2299c27e40c4853f361643d6611c4dfa85ec9ed2bb52672a664fb6ca9ec7c4f1|
|spec_sha|e87c46bf88bd2459d7c0bbf992921be3a179dbba1e18bd10d8f646ac108dcd9d|
|project_uri|https://rubygems.org/gems/bio-blastxmlparser|
|gem_uri|https://rubygems.org/gems/bio-blastxmlparser-2.0.4.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-blastxmlparser/2.0.4|
|dependencies|{"development"=>[], "runtime"=>[{"name"=>"bio-logger", "requirements"=>">= 0"}, {"name"=>"nokogiri", "requirements"=>">= 0"}]}|

</details>

## bio-grid

[![Gem Version](https://badge.fury.io/rb/bio-grid.svg)](https://badge.fury.io/rb/bio-grid)
![Year](https://img.shields.io/static/v1?label=date&message=2012-10-25&color=blue)
![Code Size](https://tokei.rs/b1/github/fstrozzi/bioruby-grid)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-grid)

A BioGem to submit jobs on a queue system

* URL: [http://github.com/fstrozzi/bioruby-grid](http://github.com/fstrozzi/bioruby-grid)
* Authors: Francesco Strozzi

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-grid|
|downloads|47253|
|version|0.3.3|
|version_created_at|2012-10-25T13:35:19.450Z|
|version_downloads|4899|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/fstrozzi/bioruby-grid"}|
|yanked|false|
|sha|e4f9fb68abf87ba7f057ce036bfe591d539af6bb28c0cf10f560b2513c867da1|
|spec_sha|546acee358848e1e01f78a5257587e1aa119e264277a573c55626c527bf55712|
|project_uri|https://rubygems.org/gems/bio-grid|
|gem_uri|https://rubygems.org/gems/bio-grid-0.3.3.gem|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.4"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rspec", "requirements"=>"~> 2.8.0"}, {"name"=>"uuid", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"uuid", "requirements"=>">= 0"}]}|

</details>

## bio-gff3

[![Gem Version](https://badge.fury.io/rb/bio-gff3.svg)](https://badge.fury.io/rb/bio-gff3)
![Year](https://img.shields.io/static/v1?label=date&message=2012-08-20&color=blue)
![Code Size](https://tokei.rs/b1/github/pjotrp/bioruby-gff3-plugin)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-gff3)

GFF3 (genome browser) information and digest mRNA and CDS sequences.
Options for low memory use and caching of records.
Support for external FASTA files.


* URL: [https://github.com/pjotrp/bioruby-gff3-plugin](https://github.com/pjotrp/bioruby-gff3-plugin)
* Authors: Pjotr Prins

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-gff3|
|downloads|45616|
|version|0.9.1|
|version_created_at|2012-08-20T09:29:57.889Z|
|version_downloads|5451|
|platform|ruby|
|metadata|{"homepage_uri"=>"https://github.com/pjotrp/bioruby-gff3-plugin"}|
|yanked|false|
|sha|392e6a9f4ef3f60fc3fa16ee9a9a5679b12f05a32f1d12f83dd4dd4fa6a2846d|
|spec_sha|b9d849128c24177b339573593eb369aaaecc2c1973383e5c7f23cc373a46e05c|
|project_uri|https://rubygems.org/gems/bio-gff3|
|gem_uri|https://rubygems.org/gems/bio-gff3-0.9.1.gem|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.1.5"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.3"}, {"name"=>"regressiontest", "requirements"=>">= 0"}, {"name"=>"rspec", "requirements"=>">= 2.10.0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.3.1"}, {"name"=>"bio-logger", "requirements"=>"> 0.8.0"}, {"name"=>"log4r", "requirements"=>"> 1.1.6"}]}|

</details>

## bio-vcf

[![Gem Version](https://badge.fury.io/rb/bio-vcf.svg)](https://badge.fury.io/rb/bio-vcf)
![Year](https://img.shields.io/static/v1?label=date&message=2021-01-18&color=blue)
![Code Size](https://tokei.rs/b1/github/vcflib/bio-vcf)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-vcf)

Smart lazy multi-threaded parser for VCF format with useful filtering and output rewriting (JSON, RDF etc.)

* URL: [http://github.com/vcflib/bio-vcf](http://github.com/vcflib/bio-vcf)
* Authors: Pjotr Prins

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-vcf|
|downloads|40609|
|version|0.9.5|
|version_created_at|2021-01-18T08:20:08.520Z|
|version_downloads|4799|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|cfbc2d226f09df69cbe063b863d4a3c902f41576de9bd9af1ee1239dec2d9cbe|
|spec_sha|be9541c8a5d9e8f346a644acb3739a901ebab97ce7570c4a9aa0f5d16b6c2c1f|
|project_uri|https://rubygems.org/gems/bio-vcf|
|gem_uri|https://rubygems.org/gems/bio-vcf-0.9.5.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-vcf/0.9.5|
|dependencies|{"development"=>[], "runtime"=>[]}|

</details>

## bio-band

[![Gem Version](https://badge.fury.io/rb/bio-band.svg)](https://badge.fury.io/rb/bio-band)
![Year](https://img.shields.io/static/v1?label=date&message=2013-09-16&color=blue)
![Code Size](https://tokei.rs/b1/github/arrigonialberto86/bioruby-band)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-band)

Data mining and machine learning algorithms for JRuby 

* URL: [http://github.com/arrigonialberto86/bioruby-band](http://github.com/arrigonialberto86/bioruby-band)
* Authors: arrigonialberto86

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-band|
|downloads|38684|
|version|0.1.11|
|version_created_at|2013-09-16T10:06:12.118Z|
|version_downloads|4626|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|8d3da73dd15a41cdc4c53731eba742b15eb6925d66d8ab18be500b8e7f41a08d|
|spec_sha|37f93439c944effc5b72a4d1f55ad5640d0d72d9c86870011be4269bf369c74a|
|project_uri|https://rubygems.org/gems/bio-band|
|gem_uri|https://rubygems.org/gems/bio-band-0.1.11.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-band/0.1.11|
|dependencies|{"development"=>[], "runtime"=>[{"name"=>"activesupport", "requirements"=>"= 3.2.13"}, {"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>"~> 1.3.5"}, {"name"=>"cucumber", "requirements"=>">= 0"}, {"name"=>"gherkin", "requirements"=>">= 0"}, {"name"=>"git", "requirements"=>">= 0"}, {"name"=>"i18n", "requirements"=>"= 0.6.1"}, {"name"=>"jbundler", "requirements"=>"= 0.4.3"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.4"}, {"name"=>"json", "requirements"=>">= 0"}, {"name"=>"multi_json", "requirements"=>">= 0"}, {"name"=>"rake", "requirements"=>">= 0"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rspec", "requirements"=>">= 0"}, {"name"=>"ruport", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"simplecov", "requirements"=>">= 0"}, {"name"=>"test-unit", "requirements"=>">= 0"}]}|

</details>

## bio-alignment

[![Gem Version](https://badge.fury.io/rb/bio-alignment.svg)](https://badge.fury.io/rb/bio-alignment)
![Year](https://img.shields.io/static/v1?label=date&message=2017-10-10&color=blue)
![Code Size](https://tokei.rs/b1/github/pjotrp/bioruby-alignment)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-alignment)

Support for multiple sequence alignments (MSA)

* URL: [http://github.com/pjotrp/bioruby-alignment](http://github.com/pjotrp/bioruby-alignment)
* Authors: Pjotr Prins

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-alignment|
|downloads|36977|
|version|0.1.0|
|version_created_at|2017-10-10T09:36:09.884Z|
|version_downloads|2764|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|5b0ebd3ae6e28c74454ed7f72102a27403fbecbf81c180a46febf82d8039d6dd|
|spec_sha|b7fe690d68e1c0dfdfe3e914c2fda492f14d974623b1abe8dc894e18f16330b1|
|project_uri|https://rubygems.org/gems/bio-alignment|
|gem_uri|https://rubygems.org/gems/bio-alignment-0.1.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-alignment/0.1.0|
|dependencies|{"development"=>[], "runtime"=>[]}|

</details>

## bio-logger

[![Gem Version](https://badge.fury.io/rb/bio-logger.svg)](https://badge.fury.io/rb/bio-logger)
![Year](https://img.shields.io/static/v1?label=date&message=2012-04-02&color=blue)
![Code Size](https://tokei.rs/b1/github/pjotrp/bioruby-logger-plugin)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-logger)

Log4r wrapper for BioRuby

* URL: [https://github.com/pjotrp/bioruby-logger-plugin](https://github.com/pjotrp/bioruby-logger-plugin)
* Authors: Pjotr Prins

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-logger|
|downloads|36262|
|version|1.0.1|
|version_created_at|2012-04-02T11:04:58.719Z|
|version_downloads|17469|
|platform|ruby|
|metadata|{"homepage_uri"=>"https://github.com/pjotrp/bioruby-logger-plugin"}|
|yanked|false|
|sha|4834e677a7350a276486128f587f90d3d59f59b0d06e7abfffb84d8bbca3fe0a|
|spec_sha|2e9331b0fa0e3d3061c1898dca861bba50421bcc69802174a7aa33a5be239b6b|
|project_uri|https://rubygems.org/gems/bio-logger|
|gem_uri|https://rubygems.org/gems/bio-logger-1.0.1.gem|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 0"}, {"name"=>"jeweler", "requirements"=>">= 0"}, {"name"=>"rdoc", "requirements"=>">= 3.12"}, {"name"=>"rspec", "requirements"=>">= 2.3.0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"log4r", "requirements"=>">= 1.1.9"}]}|

</details>

## bio-table

[![Gem Version](https://badge.fury.io/rb/bio-table.svg)](https://badge.fury.io/rb/bio-table)
![Year](https://img.shields.io/static/v1?label=date&message=2014-05-31&color=blue)
![Code Size](https://tokei.rs/b1/github/pjotrp/bioruby-table)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-table)

Functions and tools for tranforming and changing tab delimited and comma separated table files - useful for Excel sheets and SQL/RDF output

* URL: [http://github.com/pjotrp/bioruby-table](http://github.com/pjotrp/bioruby-table)
* Authors: Pjotr Prins

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-table|
|downloads|35633|
|version|1.0.0|
|version_created_at|2014-05-31T11:17:14.624Z|
|version_downloads|4893|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|d9d85c99d3afa788c81c4f48f5782e93846c0f39c20beb5c0ea44d328ea697ca|
|spec_sha|4cd863b588c188004389544317cf6986fbdd590f36b2e918b60836f034db5f0c|
|project_uri|https://rubygems.org/gems/bio-table|
|gem_uri|https://rubygems.org/gems/bio-table-1.0.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-table/1.0.0|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 0"}, {"name"=>"cucumber", "requirements"=>">= 0"}, {"name"=>"jeweler", "requirements"=>"~> 2.0.0"}, {"name"=>"rdoc", "requirements"=>">= 0"}, {"name"=>"regressiontest", "requirements"=>">= 0.0.2"}, {"name"=>"rspec", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio-logger", "requirements"=>">= 0"}]}|

</details>

## bio-publisci

[![Gem Version](https://badge.fury.io/rb/bio-publisci.svg)](https://badge.fury.io/rb/bio-publisci)
![Year](https://img.shields.io/static/v1?label=date&message=2013-09-20&color=blue)
![Code Size](https://tokei.rs/b1/github/wstrinz/bioruby-publisci)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-publisci)

A toolkit for publishing scientific results and datasets using RDF, OWL, and related technologies 

* URL: [http://github.com/wstrinz/bioruby-publisci](http://github.com/wstrinz/bioruby-publisci)
* Authors: Will Strinz

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-publisci|
|downloads|32359|
|version|0.1.0|
|version_created_at|2013-09-20T03:34:59.554Z|
|version_downloads|4600|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|f202b23f14c56ab3b3446b9610e03429c30562ff370007ba260b147f122c2da8|
|spec_sha|664d743d2d2e83289e27c38ac98efaf32e469cd4c4679cc82f9606bb696ad869|
|project_uri|https://rubygems.org/gems/bio-publisci|
|gem_uri|https://rubygems.org/gems/bio-publisci-0.1.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-publisci/0.1.0|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"cucumber", "requirements"=>">= 0"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.4"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rspec", "requirements"=>"~> 2.8.0"}, {"name"=>"spira", "requirements"=>">= 0"}, {"name"=>"spoon", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"json-ld", "requirements"=>">= 0"}, {"name"=>"rack-flash3", "requirements"=>">= 0"}, {"name"=>"rdf", "requirements"=>">= 0"}, {"name"=>"rdf-4store", "requirements"=>">= 0"}, {"name"=>"rdf-rdfxml", "requirements"=>">= 0"}, {"name"=>"rdf-turtle", "requirements"=>">= 0"}, {"name"=>"rest-client", "requirements"=>">= 0"}, {"name"=>"rserve-client", "requirements"=>">= 0"}, {"name"=>"sinatra", "requirements"=>">= 0"}, {"name"=>"sinatra-contrib", "requirements"=>">= 0"}, {"name"=>"sinatra-cross_origin", "requirements"=>">= 0"}, {"name"=>"sinatra-linkeddata", "requirements"=>">= 0"}, {"name"=>"slop", "requirements"=>">= 0"}, {"name"=>"sparql", "requirements"=>">= 0"}]}|

</details>

## bio-faster

[![Gem Version](https://badge.fury.io/rb/bio-faster.svg)](https://badge.fury.io/rb/bio-faster)
![Year](https://img.shields.io/static/v1?label=date&message=2012-06-13&color=blue)
![Code Size](https://tokei.rs/b1/github/fstrozzi/bioruby-faster)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-faster)

A fast parser for FastQ files

* URL: [http://github.com/fstrozzi/bioruby-faster](http://github.com/fstrozzi/bioruby-faster)
* Authors: Francesco Strozzi

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-faster|
|downloads|30497|
|version|0.4.5|
|version_created_at|2012-06-13T10:53:06.185Z|
|version_downloads|5754|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/fstrozzi/bioruby-faster"}|
|yanked|false|
|sha|a3fa411c33e019428c79faa5cf97f1c5ca5eb2797174cad985944f8d8fdeb97a|
|spec_sha|95130c6d9753a25380853cbb9a62d70a7d1d49e22924779baedf92b10dcae942|
|project_uri|https://rubygems.org/gems/bio-faster|
|gem_uri|https://rubygems.org/gems/bio-faster-0.4.5.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"ffi", "requirements"=>">= 0"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.4"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"rspec", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"ffi", "requirements"=>">= 0"}]}|

</details>

## bio-ngs

[![Gem Version](https://badge.fury.io/rb/bio-ngs.svg)](https://badge.fury.io/rb/bio-ngs)
![Year](https://img.shields.io/static/v1?label=date&message=2012-06-06&color=blue)
![Code Size](https://tokei.rs/b1/github/helios/bioruby-ngs)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-ngs)

bio-ngs provides a framework for handling NGS data with BioRuby

* URL: [http://github.com/helios/bioruby-ngs](http://github.com/helios/bioruby-ngs)
* Authors: Francesco Strozzi, Raoul J.P. Bonnal

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-ngs|
|downloads|30434|
|version|0.5.2|
|version_created_at|2012-06-06T13:27:18.216Z|
|version_downloads|5551|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/helios/bioruby-ngs"}|
|yanked|false|
|sha|b0f766ed6e464ab32608dc78b0d4fe07aab999238d389d4667a104a862e768a9|
|spec_sha|6c95a928cdda628e2bef1914ab62134e20f57331bd96ea6f6c1c1827df115b42|
|project_uri|https://rubygems.org/gems/bio-ngs|
|gem_uri|https://rubygems.org/gems/bio-ngs-0.5.2.gem|
|dependencies|{"development"=>[{"name"=>"activerecord", "requirements"=>">= 3.0.5"}, {"name"=>"activerecord-jdbcsqlite3-adapter", "requirements"=>">= 0"}, {"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bio-blastxmlparser", "requirements"=>">= 0"}, {"name"=>"bio-samtools", "requirements"=>">= 0.3.2"}, {"name"=>"bundler", "requirements"=>"~> 1.1.0"}, {"name"=>"daemons", "requirements"=>">= 1.1.0"}, {"name"=>"ffi", "requirements"=>">= 1.0.6"}, {"name"=>"jdbc-sqlite3", "requirements"=>">= 0"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.3"}, {"name"=>"json", "requirements"=>">= 0"}, {"name"=>"parallel", "requirements"=>">= 0"}, {"name"=>"progressbar", "requirements"=>">= 0.9.0"}, {"name"=>"rake", "requirements"=>"= 0.9.2.2"}, {"name"=>"rcov", "requirements"=>"~> 0.9.11"}, {"name"=>"rspec", "requirements"=>">= 2.5.0"}, {"name"=>"ruby-ensembl-api", "requirements"=>">= 1.0.1"}, {"name"=>"rubyvis", "requirements"=>">= 0.5.0"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"sqlite3", "requirements"=>">= 0"}, {"name"=>"thor", "requirements"=>"= 0.14.6"}], "runtime"=>[{"name"=>"activerecord", "requirements"=>">= 3.0.5"}, {"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bio-blastxmlparser", "requirements"=>">= 0"}, {"name"=>"bio-samtools", "requirements"=>">= 0.3.2"}, {"name"=>"daemons", "requirements"=>">= 1.1.0"}, {"name"=>"json", "requirements"=>">= 0"}, {"name"=>"parallel", "requirements"=>">= 0"}, {"name"=>"progressbar", "requirements"=>">= 0.9.0"}, {"name"=>"rake", "requirements"=>"= 0.9.2.2"}, {"name"=>"ruby-ensembl-api", "requirements"=>">= 1.0.1"}, {"name"=>"rubyvis", "requirements"=>">= 0.5.0"}, {"name"=>"thor", "requirements"=>"= 0.14.6"}]}|

</details>

## bio-pipengine

[![Gem Version](https://badge.fury.io/rb/bio-pipengine.svg)](https://badge.fury.io/rb/bio-pipengine)
![Year](https://img.shields.io/static/v1?label=date&message=2017-08-28&color=blue)
![Code Size](https://tokei.rs/b1/github/fstrozzi/bioruby-pipengine)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-pipengine)

A pipeline manager

* URL: [http://github.com/fstrozzi/bioruby-pipengine](http://github.com/fstrozzi/bioruby-pipengine)
* Authors: Francesco Strozzi, Raoul Jean Pierre Bonnal

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-pipengine|
|downloads|29428|
|version|0.9.7|
|version_created_at|2017-08-28T08:02:35.924Z|
|version_downloads|2719|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|4d65c7a9a10e9fc7aa979616ab5cf5c9bd03368ca6f013f01403e8cc67170a9f|
|spec_sha|df956021338525f6bd71570939a9a8ca5c7f8edaad7e84c67e7a06f9efc8d0b1|
|project_uri|https://rubygems.org/gems/bio-pipengine|
|gem_uri|https://rubygems.org/gems/bio-pipengine-0.9.7.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-pipengine/0.9.7|
|dependencies|{"development"=>[{"name"=>"jeweler", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"colorize", "requirements"=>">= 0.8.1"}, {"name"=>"trollop", "requirements"=>">= 2.1.2"}]}|

</details>

## bio-velvet

[![Gem Version](https://badge.fury.io/rb/bio-velvet.svg)](https://badge.fury.io/rb/bio-velvet)
![Year](https://img.shields.io/static/v1?label=date&message=2015-01-09&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-velvet)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-velvet)

Parser to work with some file formats used in the velvet DNA assembler

* URL: [http://github.com/wwood/bioruby-velvet](http://github.com/wwood/bioruby-velvet)
* Authors: Ben J Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-velvet|
|downloads|26038|
|version|0.6.0|
|version_created_at|2015-01-09T05:47:28.928Z|
|version_downloads|4150|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|319995af9a2ad99a717ee7b23856fdff2a83da8c0146c5aea1fc5df306c90560|
|spec_sha|80c69284ce70f048c80beb30602ae2775cd990c2b06f2662c4263fc422099014|
|project_uri|https://rubygems.org/gems/bio-velvet|
|gem_uri|https://rubygems.org/gems/bio-velvet-0.6.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-velvet/0.6.0|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>"~> 1.4"}, {"name"=>"bundler", "requirements"=>"~> 1.0"}, {"name"=>"jeweler", "requirements"=>"~> 2.0"}, {"name"=>"pry", "requirements"=>"~> 0.9"}, {"name"=>"rdoc", "requirements"=>"~> 4.1"}, {"name"=>"rspec", "requirements"=>"~> 2.8"}], "runtime"=>[{"name"=>"bio-commandeer", "requirements"=>"~> 0.1"}, {"name"=>"bio-logger", "requirements"=>"~> 1.0"}, {"name"=>"files", "requirements"=>"~> 0.3"}, {"name"=>"hopcsv", "requirements"=>"~> 0.4"}, {"name"=>"parallel", "requirements"=>">= 1.3.3, ~> 1.3"}, {"name"=>"systemu", "requirements"=>"~> 2.6"}]}|

</details>

## bio-bigbio

[![Gem Version](https://badge.fury.io/rb/bio-bigbio.svg)](https://badge.fury.io/rb/bio-bigbio)
![Year](https://img.shields.io/static/v1?label=date&message=2014-05-16&color=blue)
![Code Size](https://tokei.rs/b1/github/pjotrp/bigbio)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-bigbio)

Fasta reader, ORF emitter, sequence translation

* URL: [http://github.com/pjotrp/bigbio](http://github.com/pjotrp/bigbio)
* Authors: Pjotr Prins

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-bigbio|
|downloads|25616|
|version|0.1.6|
|version_created_at|2014-05-16T11:31:16.797Z|
|version_downloads|4405|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|19fdcf1efa3479cf93191a8e703e61e4fc63d46be41ad1eac92d8a89c94ecc11|
|spec_sha|e50251a51f736cea830db6ee28e5f3b455b83cb111f6abfa94741700e1e11874|
|project_uri|https://rubygems.org/gems/bio-bigbio|
|gem_uri|https://rubygems.org/gems/bio-bigbio-0.1.6.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-bigbio/0.1.6|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 0"}, {"name"=>"jeweler", "requirements"=>">= 0"}, {"name"=>"rspec", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"bio-logger", "requirements"=>">= 0"}]}|

</details>

## bio-signalp

[![Gem Version](https://badge.fury.io/rb/bio-signalp.svg)](https://badge.fury.io/rb/bio-signalp)
![Year](https://img.shields.io/static/v1?label=date&message=2014-08-04&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-signalp)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-signalp)

A wrapper for the signal peptide prediction algorith SignalP. Not very well supported, but seems to work for the author, at least.

* URL: [http://github.com/wwood/bioruby-signalp](http://github.com/wwood/bioruby-signalp)
* Authors: Ben J Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-signalp|
|downloads|25002|
|version|0.3.1|
|version_created_at|2014-08-04T23:04:16.371Z|
|version_downloads|4281|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|ac9bb29395b47f98681945b22225b959ca346c3311f73c6b90db7c59798320a0|
|spec_sha|2a508568e8ae6969ce4b53fb3231127933eeb85899f8640ff7395872300cbbc2|
|project_uri|https://rubygems.org/gems/bio-signalp|
|gem_uri|https://rubygems.org/gems/bio-signalp-0.3.1.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-signalp/0.3.1|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 1.0.21, ~> 1.0"}, {"name"=>"jeweler", "requirements"=>"~> 2.0"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rspec", "requirements"=>"~> 2.8"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.1, ~> 1.4"}, {"name"=>"bio-logger", "requirements"=>"~> 1.0"}]}|

</details>

## bio-commandeer

[![Gem Version](https://badge.fury.io/rb/bio-commandeer.svg)](https://badge.fury.io/rb/bio-commandeer)
![Year](https://img.shields.io/static/v1?label=date&message=2017-08-10&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-commandeer)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-commandeer)

A dead simple method of running shell commands from within Ruby, by applying opinion

* URL: [http://github.com/wwood/bioruby-commandeer](http://github.com/wwood/bioruby-commandeer)
* Authors: Ben J. Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-commandeer|
|downloads|23915|
|version|0.4.0|
|version_created_at|2017-08-10T11:59:16.511Z|
|version_downloads|4740|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|d1522ba5744d0213da53834558d41d8cb0e5c598f3768baa688316ea0377174e|
|spec_sha|76b60b77174bdee9abccaee58416fcfc27e94945f768783b3ced0c255a4739c4|
|project_uri|https://rubygems.org/gems/bio-commandeer|
|gem_uri|https://rubygems.org/gems/bio-commandeer-0.4.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-commandeer/0.4.0|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.3"}, {"name"=>"jeweler", "requirements"=>"~> 2.0"}, {"name"=>"rdoc", "requirements"=>"~> 5.0"}, {"name"=>"rspec", "requirements"=>"~> 3.2"}], "runtime"=>[{"name"=>"bio-logger", "requirements"=>"~> 1.0"}, {"name"=>"systemu", "requirements"=>"~> 2.6"}]}|

</details>

## bio-phyta

[![Gem Version](https://badge.fury.io/rb/bio-phyta.svg)](https://badge.fury.io/rb/bio-phyta)
![Year](https://img.shields.io/static/v1?label=date&message=2011-12-07&color=blue)
![Code Size](https://tokei.rs/b1/github/PalMuc/bio-phyta)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-phyta)

Pipeline to remove contaminations from EST libraries

* URL: [https://github.com/PalMuc/bio-phyta](https://github.com/PalMuc/bio-phyta)
* Authors: Philipp Comans

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-phyta|
|downloads|23186|
|version|0.9.4|
|version_created_at|2011-12-07T10:23:30.050Z|
|version_downloads|5515|
|platform|ruby|
|metadata|{"homepage_uri"=>"https://github.com/PalMuc/bio-phyta"}|
|yanked|false|
|sha|9b88babd6666c381c935b69993374054b4799d5dcabf92785e7c25519abee786|
|spec_sha|7f53603355f6087c7ba8b9d4fed4bd1b6ae5b121f25897873f6214a17ce57912|
|project_uri|https://rubygems.org/gems/bio-phyta|
|gem_uri|https://rubygems.org/gems/bio-phyta-0.9.4.gem|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.4"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"fastercsv", "requirements"=>">= 1.5.4"}, {"name"=>"mysql2", "requirements"=>">= 0"}, {"name"=>"nokogiri", "requirements"=>">= 1.5.0"}, {"name"=>"sequel", "requirements"=>">= 3.28.0"}, {"name"=>"trollop", "requirements"=>">= 1.16.2"}]}|

</details>

## bio-polyploid-tools

[![Gem Version](https://badge.fury.io/rb/bio-polyploid-tools.svg)](https://badge.fury.io/rb/bio-polyploid-tools)
![Year](https://img.shields.io/static/v1?label=date&message=2021-05-10&color=blue)
![Code Size](https://tokei.rs/b1/github/tgac/bioruby-polyploid-tools)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-polyploid-tools)

Repository of tools developed at Crop Genetics in JIC to work with polyploid wheat

* URL: [http://github.com/tgac/bioruby-polyploid-tools](http://github.com/tgac/bioruby-polyploid-tools)
* Authors: Ricardo H.  Ramirez-Gonzalez

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-polyploid-tools|
|downloads|119763|
|version|1.2.1|
|version_created_at|2021-05-10T08:28:42.442Z|
|version_downloads|2721|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|4c315f8691200db1e5309fdc5caff9257283bc7b701e8ea54abbb67971351a2e|
|spec_sha|53a621066d01f3b3572e9fcd9b343fc4ada16c0715fea8955c48f428f4e7d3e4|
|project_uri|https://rubygems.org/gems/bio-polyploid-tools|
|gem_uri|https://rubygems.org/gems/bio-polyploid-tools-1.2.1.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-polyploid-tools/1.2.1|
|dependencies|{"development"=>[{"name"=>"juwelier", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 2.10"}, {"name"=>"test-unit", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.5.1"}, {"name"=>"bio-samtools", "requirements"=>">= 2.6.2"}, {"name"=>"descriptive_statistics", "requirements"=>">= 0"}, {"name"=>"systemu", "requirements"=>">= 2.5.2"}]}|

</details>

## bio-sambamba

[![Gem Version](https://badge.fury.io/rb/bio-sambamba.svg)](https://badge.fury.io/rb/bio-sambamba)
![Year](https://img.shields.io/static/v1?label=date&message=2012-08-20&color=blue)
![Code Size](https://tokei.rs/b1/github/lomereiter/bioruby-sambamba)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-sambamba)

New Sambamba library comes with a command-line tool for working with SAM/BAM files. This gem brings some of its functionality to Ruby.

* URL: [http://github.com/lomereiter/bioruby-sambamba](http://github.com/lomereiter/bioruby-sambamba)
* Authors: Artem Tarasov

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-sambamba|
|downloads|21333|
|version|0.0.5|
|version_created_at|2012-08-20T08:38:29.374Z|
|version_downloads|5158|
|platform|ruby|
|metadata|{"wiki_uri"=>"https://github.com/lomereiter/bioruby-sambamba/wiki/Usage", "homepage_uri"=>"http://github.com/lomereiter/bioruby-sambamba", "bug_tracker_uri"=>"", "source_code_uri"=>"", "mailing_list_uri"=>"", "documentation_uri"=>""}|
|yanked|false|
|sha|3d3d150fe791207a788d2bcdbca6e0e9ea66010e8af9fd784fecaa14335429de|
|spec_sha|b741ac0e6ec51972bd9473a65628e221f458ab87b35a63abc0f889a88a7f3573|
|project_uri|https://rubygems.org/gems/bio-sambamba|
|gem_uri|https://rubygems.org/gems/bio-sambamba-0.0.5.gem|
|wiki_uri|https://github.com/lomereiter/bioruby-sambamba/wiki/Usage|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.1.4"}, {"name"=>"cucumber", "requirements"=>">= 0"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.3"}, {"name"=>"rake", "requirements"=>">= 0"}, {"name"=>"rspec", "requirements"=>"~> 2.7.0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"msgpack", "requirements"=>">= 0.4.7"}]}|

</details>

## bio-assembly

[![Gem Version](https://badge.fury.io/rb/bio-assembly.svg)](https://badge.fury.io/rb/bio-assembly)
![Year](https://img.shields.io/static/v1?label=date&message=2011-01-12&color=blue)
![Code Size](https://tokei.rs/b1/github/chmille4/bioruby-assembly)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-assembly)

bioruby plugin to parse, write, and manipulate assembly data

* URL: [http://github.com/chmille4/bioruby-assembly](http://github.com/chmille4/bioruby-assembly)
* Authors: Chase Miller, Francesco Strozzi

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-assembly|
|downloads|19435|
|version|0.1.0|
|version_created_at|2011-01-12T18:14:20.839Z|
|version_downloads|5894|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/chmille4/bioruby-assembly"}|
|yanked|false|
|sha|8d9dabf24c48a4216d6d9bd3b74d742b8e942bb9b0008ccf3263d7b07a48d3d4|
|spec_sha|64fcf8782555fe91dfca1d9b700a655ca20c93bdcbd06e9afae79727146cacd5|
|project_uri|https://rubygems.org/gems/bio-assembly|
|gem_uri|https://rubygems.org/gems/bio-assembly-0.1.0.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.5.2"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}]}|

</details>

## bio-nexml

[![Gem Version](https://badge.fury.io/rb/bio-nexml.svg)](https://badge.fury.io/rb/bio-nexml)
![Year](https://img.shields.io/static/v1?label=date&message=2012-05-09&color=blue)
![Code Size](https://tokei.rs/b1/github/nexml/bio-nexml)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-nexml)

This plugin reads, writes and generates NeXML

* URL: [https://github.com/nexml/bio-nexml](https://github.com/nexml/bio-nexml)
* Authors: Rutger Vos, Anurag Priyam

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-nexml|
|downloads|18753|
|version|1.1.0|
|version_created_at|2012-05-09T06:15:27.128Z|
|version_downloads|5145|
|platform|ruby|
|metadata|{"homepage_uri"=>"https://github.com/nexml/bio-nexml"}|
|yanked|false|
|sha|4483e3e64a0cfee90f1c338d4a540b2b7054e0c4f2a7e5728b0a00b26dd2df80|
|spec_sha|36dc343fcf80cfbab8bfa9a66214cfe4b6f0974f5ea0d0568443c6865f3e26de|
|project_uri|https://rubygems.org/gems/bio-nexml|
|gem_uri|https://rubygems.org/gems/bio-nexml-1.1.0.gem|
|dependencies|{"development"=>[], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"libxml-ruby", "requirements"=>"= 1.1.4"}]}|

</details>

## bio-tabix

[![Gem Version](https://badge.fury.io/rb/bio-tabix.svg)](https://badge.fury.io/rb/bio-tabix)
![Year](https://img.shields.io/static/v1?label=date&message=2013-09-30&color=blue)
![Code Size](https://tokei.rs/b1/github/throwern/bio-tabix)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-tabix)

Tabix file indexing routines from the samtools package http://samtools.sourceforge.net/

* URL: [http://github.com/throwern/bio-tabix](http://github.com/throwern/bio-tabix)
* Authors: throwern

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-tabix|
|downloads|16195|
|version|1.0.1|
|version_created_at|2013-09-30T14:05:37.762Z|
|version_downloads|4616|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|2d948601f6b982933e8d04eb159efe9a38fd0147a304c3a940f6c592c6777bef|
|spec_sha|9908889878177c88c9cba8682bd75ff0de17a7650f996e0954c161b2e3554dc3|
|project_uri|https://rubygems.org/gems/bio-tabix|
|gem_uri|https://rubygems.org/gems/bio-tabix-1.0.1.gem|
|wiki_uri||
|documentation_uri|http://rubydoc.info/github/throwern/bio-tabix/frames|
|mailing_list_uri||
|source_code_uri||
|bug_tracker_uri||
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 0"}, {"name"=>"jeweler", "requirements"=>">= 0"}, {"name"=>"rdoc", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"simplecov", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"ffi", "requirements"=>">= 0"}]}|

</details>

## bio-graphics

[![Gem Version](https://badge.fury.io/rb/bio-graphics.svg)](https://badge.fury.io/rb/bio-graphics)
![Year](https://img.shields.io/static/v1?label=date&message=2009-07-25&color=blue)
![Code Size]()
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-graphics)

Library for visualizing genomic regions

* URL: [http://bio-graphics.rubyforge.org/](http://bio-graphics.rubyforge.org/)
* Authors: Jan Aerts

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-graphics|
|downloads|15780|
|version|1.4|
|version_created_at|2009-07-25T18:31:34.000Z|
|version_downloads|6295|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://bio-graphics.rubyforge.org/"}|
|yanked|false|
|sha|61fffa7c87451771d6f1c3381c5c21d193ec2d3b9cfacce23aeee61537fd03da|
|spec_sha|6cb359523eb07d3326c553447ca87ee0fe22b33a528151a7224d9f75017cf53b|
|project_uri|https://rubygems.org/gems/bio-graphics|
|gem_uri|https://rubygems.org/gems/bio-graphics-1.4.gem|
|dependencies|{"development"=>[], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1"}]}|

</details>

## bio-plasmoap

[![Gem Version](https://badge.fury.io/rb/bio-plasmoap.svg)](https://badge.fury.io/rb/bio-plasmoap)
![Year](https://img.shields.io/static/v1?label=date&message=2016-04-14&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-plasmoap)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-plasmoap)

The PlasmoAP algorithm (Foth and Ralph et. al. 2003) predicts apicoplast transit peptides in Plasmodium falciparum. This is an implementation of that algorithm.

* URL: [http://github.com/wwood/bioruby-plasmoap](http://github.com/wwood/bioruby-plasmoap)
* Authors: Ben J Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-plasmoap|
|downloads|15712|
|version|0.1.1|
|version_created_at|2016-04-14T10:13:52.998Z|
|version_downloads|3798|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|e5ecf8f70c03b27749c4aa87d821ffcd66d08106c644b2781798ce03be89a953|
|spec_sha|5c03fe3465211fc9ddee9ee32cf67379ab87e384a8744f358d4f368fd5b695c1|
|project_uri|https://rubygems.org/gems/bio-plasmoap|
|gem_uri|https://rubygems.org/gems/bio-plasmoap-0.1.1.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-plasmoap/0.1.1|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 1.0.0"}, {"name"=>"jeweler", "requirements"=>">= 1.6.4"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"test-unit", "requirements"=>">= 3.1.8"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"bio-signalp", "requirements"=>">= 0.2.0"}]}|

</details>

## bio-ucsc-api

[![Gem Version](https://badge.fury.io/rb/bio-ucsc-api.svg)](https://badge.fury.io/rb/bio-ucsc-api)
![Year](https://img.shields.io/static/v1?label=date&message=2018-05-11&color=blue)
![Code Size](https://tokei.rs/b1/github/misshie/bioruby-ucsc-api)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-ucsc-api)

Ruby UCSC API: accessing the UCSC Genome Database using Ruby

* URL: [http://github.com/misshie/bioruby-ucsc-api](http://github.com/misshie/bioruby-ucsc-api)
* Authors: Hiroyuki Mishima, Jan Aerts

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-ucsc-api|
|downloads|75546|
|version|0.6.5|
|version_created_at|2018-05-11T08:24:36.148Z|
|version_downloads|2598|
|platform|ruby|
|licenses|["The MIT License"]|
|metadata|{}|
|yanked|false|
|sha|c26294c1a6d1335cd2769761ab5513d49660b25fa2a618a37f40abe47d76d174|
|spec_sha|8f5cc85ee72d405c49a125333d0ad86c31fa60f77cf5801d08dfcdc872e7ddf2|
|project_uri|https://rubygems.org/gems/bio-ucsc-api|
|gem_uri|https://rubygems.org/gems/bio-ucsc-api-0.6.5.gem|
|wiki_uri||
|documentation_uri|https://github.com/misshie/bioruby-ucsc-api|
|mailing_list_uri||
|source_code_uri|http://github.com/misshie/bioruby-ucsc-api|
|bug_tracker_uri|https://github.com/misshie/bioruby-ucsc-api|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.5.1"}, {"name"=>"bundler", "requirements"=>">= 1.15.4"}, {"name"=>"jeweler", "requirements"=>">= 2.3.7"}, {"name"=>"rake", "requirements"=>">= 12.1.0"}, {"name"=>"rdoc", "requirements"=>">= 4.0.1"}, {"name"=>"rspec", "requirements"=>">= 2.14.1"}], "runtime"=>[{"name"=>"activerecord", "requirements"=>"~> 4.2.9"}, {"name"=>"activerecord-deprecated_finders", "requirements"=>"~> 1.0.4"}, {"name"=>"activesupport", "requirements"=>"~> 4.2.9"}, {"name"=>"bio-genomic-interval", "requirements"=>">= 0.1.2"}, {"name"=>"mysql2", "requirements"=>"~> 0.4.9"}, {"name"=>"safe_attributes", "requirements"=>"~> 1.0.10"}]}|

</details>

## bio-octopus

[![Gem Version](https://badge.fury.io/rb/bio-octopus.svg)](https://badge.fury.io/rb/bio-octopus)
![Year](https://img.shields.io/static/v1?label=date&message=2011-05-02&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-octopus)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-octopus)

Running and parsing of the protein transmembrane domain predictor octopus

* URL: [http://github.com/wwood/bioruby-octopus](http://github.com/wwood/bioruby-octopus)
* Authors: Ben J Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-octopus|
|downloads|14935|
|version|0.1.1|
|version_created_at|2011-05-02T01:33:19.844Z|
|version_downloads|5691|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/wwood/bioruby-octopus"}|
|yanked|false|
|sha|27259c24217cc622dabf583d3b6bc46e2e144e51ba971ffaf5d561c279875311|
|spec_sha|5fb2c68e2cef06e2cf951f9091546136e4edb2a173b757b8dd25f441cc663ecf|
|project_uri|https://rubygems.org/gems/bio-octopus|
|gem_uri|https://rubygems.org/gems/bio-octopus-0.1.1.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.5.2"}, {"name"=>"rcov", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"bio-tm_hmm", "requirements"=>">= 0"}, {"name"=>"fastercsv", "requirements"=>">= 0"}, {"name"=>"rio", "requirements"=>">= 0"}]}|

</details>

## bio-krona

[![Gem Version](https://badge.fury.io/rb/bio-krona.svg)](https://badge.fury.io/rb/bio-krona)
![Year](https://img.shields.io/static/v1?label=date&message=2015-12-12&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-krona)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-krona)

This biogem is built around Krona, a flashy way of representing hierarchical data.

* URL: [http://github.com/wwood/bioruby-krona](http://github.com/wwood/bioruby-krona)
* Authors: Ben J. Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-krona|
|downloads|14618|
|version|0.2.0|
|version_created_at|2015-12-12T02:59:51.705Z|
|version_downloads|3505|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|7945bd027e891d51b12c7b22ad3895803d9699b25bca6b678793f3aa06668c06|
|spec_sha|9d9a79e15b28f372b6e809b383d242816a52590f5f3dd120f81df35a636258d6|
|project_uri|https://rubygems.org/gems/bio-krona|
|gem_uri|https://rubygems.org/gems/bio-krona-0.2.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-krona/0.2.0|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>">= 1.8.3"}, {"name"=>"pry", "requirements"=>">= 0"}, {"name"=>"rdoc", "requirements"=>">= 3.12"}, {"name"=>"rspec", "requirements"=>"~> 2.8.0"}, {"name"=>"yard", "requirements"=>">= 0.7"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 0"}]}|

</details>

## bio-bwa

[![Gem Version](https://badge.fury.io/rb/bio-bwa.svg)](https://badge.fury.io/rb/bio-bwa)
![Year](https://img.shields.io/static/v1?label=date&message=2011-04-15&color=blue)
![Code Size](https://tokei.rs/b1/github/fstrozzi/bioruby-bwa)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-bwa)

Ruby binding for BWA mapping software, built using Ruby-FFI library

* URL: [http://github.com/fstrozzi/bioruby-bwa](http://github.com/fstrozzi/bioruby-bwa)
* Authors: Francesco Strozzi

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-bwa|
|downloads|14384|
|version|0.2.2|
|version_created_at|2011-04-15T13:22:40.740Z|
|version_downloads|5784|
|platform|ruby|
|metadata|{"wiki_uri"=>"https://github.com/fstrozzi/bioruby-bwa/wiki", "homepage_uri"=>"http://github.com/fstrozzi/bioruby-bwa", "bug_tracker_uri"=>"https://github.com/fstrozzi/bioruby-bwa/issues", "source_code_uri"=>"https://github.com/fstrozzi/bioruby-bwa", "mailing_list_uri"=>"", "documentation_uri"=>"http://fstrozzi.github.com/bioruby-bwa/"}|
|yanked|false|
|sha|a02efbf0dde45164c47634a736d42c355cdb8f5af101deca19fd70f1281c2101|
|spec_sha|3962154b3362d8ae8232bc53a90114e370d4989e4c1a4c213096c7a1b04ed61e|
|project_uri|https://rubygems.org/gems/bio-bwa|
|gem_uri|https://rubygems.org/gems/bio-bwa-0.2.2.gem|
|wiki_uri|https://github.com/fstrozzi/bioruby-bwa/wiki|
|documentation_uri|http://fstrozzi.github.com/bioruby-bwa/|
|source_code_uri|https://github.com/fstrozzi/bioruby-bwa|
|bug_tracker_uri|https://github.com/fstrozzi/bioruby-bwa/issues|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"ffi", "requirements"=>">= 1.0.5"}, {"name"=>"jeweler", "requirements"=>"~> 1.5.2"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-lazyblastxml

[![Gem Version](https://badge.fury.io/rb/bio-lazyblastxml.svg)](https://badge.fury.io/rb/bio-lazyblastxml)
![Year](https://img.shields.io/static/v1?label=date&message=2011-06-01&color=blue)
![Code Size](https://tokei.rs/b1/github/robsyme/bioruby-lazyblastxml)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-lazyblastxml)

This is very scrappy at the moment, and will need to be seriously cleaned up. It does what I need it to do for now. I'll fix it up in the coming weeks. Promise :)

* URL: [http://github.com/robsyme/bioruby-lazyblastxml](http://github.com/robsyme/bioruby-lazyblastxml)
* Authors: robsyme

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-lazyblastxml|
|downloads|13977|
|version|0.4.0|
|version_created_at|2011-06-01T02:18:05.499Z|
|version_downloads|5480|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/robsyme/bioruby-lazyblastxml"}|
|yanked|false|
|sha|9e574f6e5fe17bba5d3f8e37e9cf2ae3144529f5a8a298a79e631ac982c01e24|
|spec_sha|db9bf66a9b4c38117f0feb7ac3b192fa01e366315c0610825adbf074c7a063ca|
|project_uri|https://rubygems.org/gems/bio-lazyblastxml|
|gem_uri|https://rubygems.org/gems/bio-lazyblastxml-0.4.0.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.0"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"libxml-ruby", "requirements"=>">= 2.0.5"}]}|

</details>

## bio-liftover

[![Gem Version](https://badge.fury.io/rb/bio-liftover.svg)](https://badge.fury.io/rb/bio-liftover)
![Year](https://img.shields.io/static/v1?label=date&message=2014-03-11&color=blue)
![Code Size](https://tokei.rs/b1/github/andreirozanski/bioruby-liftover)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-liftover)

Simple, under development Ruby solution for UCSC LiftOver tool

* URL: [http://github.com/andreirozanski/bioruby-liftover](http://github.com/andreirozanski/bioruby-liftover)
* Authors: Andrei Rozanski

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-liftover|
|downloads|13669|
|version|1.3.0|
|version_created_at|2014-03-11T14:12:55.282Z|
|version_downloads|4216|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|6233217f46b0ef8a9733979954a47879b41023ee53a01305c872250f393ff147|
|spec_sha|d34167190fb4f9a7f797f5baa4e15c81ef8c516ed16df59eec5ae9df9b7aaebf|
|project_uri|https://rubygems.org/gems/bio-liftover|
|gem_uri|https://rubygems.org/gems/bio-liftover-1.3.0.gem|
|wiki_uri|https://github.com/andreirozanski/bioruby-liftover|
|documentation_uri|https://github.com/andreirozanski/bioruby-liftover|
|mailing_list_uri|https://github.com/andreirozanski/bioruby-liftover|
|source_code_uri|https://github.com/andreirozanski/bioruby-liftover|
|bug_tracker_uri|https://github.com/andreirozanski/bioruby-liftover|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>"~> 2.0.1"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"simplecov", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"interval-tree", "requirements"=>">= 0.1.3"}]}|

</details>

## bio-gngm

[![Gem Version](https://badge.fury.io/rb/bio-gngm.svg)](https://badge.fury.io/rb/bio-gngm)
![Year](https://img.shields.io/static/v1?label=date&message=2012-12-06&color=blue)
![Code Size](https://tokei.rs/b1/github/danmaclean/bioruby-gngm)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-gngm)

Identify causative mutations in a model genome from NGS reads using the NGM method.

* URL: [http://github.com/danmaclean/bioruby-gngm](http://github.com/danmaclean/bioruby-gngm)
* Authors: Dan MacLean

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-gngm|
|downloads|13405|
|version|0.2.1|
|version_created_at|2012-12-06T12:07:43.571Z|
|version_downloads|5166|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{"homepage_uri"=>"http://github.com/danmaclean/bioruby-gngm"}|
|yanked|false|
|sha|904ca6413bf375f4ed70b2e6936086e4e4a45d7f420556d0bb1514a377f0d622|
|spec_sha|f623d948aec76d448b5871eaf484b23ee3bb166a5956fed3abfd96ae2b029975|
|project_uri|https://rubygems.org/gems/bio-gngm|
|gem_uri|https://rubygems.org/gems/bio-gngm-0.2.1.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bio-samtools", "requirements"=>">= 0.5.0"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>">= 0"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"rdoc", "requirements"=>">= 0"}, {"name"=>"rinruby", "requirements"=>">= 2.0.2"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bio-samtools", "requirements"=>">= 0.5.0"}, {"name"=>"rinruby", "requirements"=>">= 2.0.2"}]}|

</details>

## bio-chembl

[![Gem Version](https://badge.fury.io/rb/bio-chembl.svg)](https://badge.fury.io/rb/bio-chembl)
![Year](https://img.shields.io/static/v1?label=date&message=2012-07-23&color=blue)
![Code Size](https://tokei.rs/b1/github/nakao/bio-chembl)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-chembl)

BioRuby plugin for ChEMBL REST API Client, Parser and Container classes

* URL: [http://github.com/nakao/bio-chembl](http://github.com/nakao/bio-chembl)
* Authors: Mitsuteru Nakao

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-chembl|
|downloads|12889|
|version|0.1.3|
|version_created_at|2012-07-23T14:33:41.705Z|
|version_downloads|5049|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/nakao/bio-chembl"}|
|yanked|false|
|sha|f9eed2f903e6e4e44e1cfc788df01eb341b8c40ff421aa8caf82fa9ded7cd7e0|
|spec_sha|858d8a98e57ba1b0d9ad4c11ec7d1750fb7084583c99da7cc3f434a85ab8cfca|
|project_uri|https://rubygems.org/gems/bio-chembl|
|gem_uri|https://rubygems.org/gems/bio-chembl-0.1.3.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.0"}, {"name"=>"curb", "requirements"=>">= 0"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.3"}, {"name"=>"nokogiri", "requirements"=>"~> 1.5.5"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-rdf

[![Gem Version](https://badge.fury.io/rb/bio-rdf.svg)](https://badge.fury.io/rb/bio-rdf)
![Year](https://img.shields.io/static/v1?label=date&message=2012-05-23&color=blue)
![Code Size](https://tokei.rs/b1/github/pjotrp/bioruby-rdf)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-rdf)

Store relationships in a triple-store, such as gene enrichment and QTL, and use this information for inference and causality

* URL: [http://github.com/pjotrp/bioruby-rdf](http://github.com/pjotrp/bioruby-rdf)
* Authors: Pjotr Prins

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-rdf|
|downloads|12390|
|version|0.0.2|
|version_created_at|2012-05-23T07:44:24.909Z|
|version_downloads|5423|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/pjotrp/bioruby-rdf"}|
|yanked|false|
|sha|3dc4062d5dcf01889b77f037337c67cb0801225317046379e129e76e2be2179a|
|spec_sha|fb593ed5400c40555abc4f2756a0e8d3f6bed3f3726f8048d8d854c7bacecb90|
|project_uri|https://rubygems.org/gems/bio-rdf|
|gem_uri|https://rubygems.org/gems/bio-rdf-0.0.2.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.0"}, {"name"=>"cucumber", "requirements"=>">= 0"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.3"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rspec", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio-logger", "requirements"=>">= 0"}]}|

</details>

## bio-affy

[![Gem Version](https://badge.fury.io/rb/bio-affy.svg)](https://badge.fury.io/rb/bio-affy)
![Year](https://img.shields.io/static/v1?label=date&message=2011-12-09&color=blue)
![Code Size](https://tokei.rs/b1/github/pjotrp/bioruby-affy)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-affy)

Affymetrix microarray file format parser
  (CEL/CDF) for Ruby. FFI binding to Biolib port of R/Affyio by Benjamin Milo Bolstad

* URL: [http://github.com/pjotrp/bioruby-affy](http://github.com/pjotrp/bioruby-affy)
* Authors: Pjotr Prins

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-affy|
|downloads|12283|
|version|0.5.1|
|version_created_at|2011-12-09T15:48:17.263Z|
|version_downloads|5439|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/pjotrp/bioruby-affy"}|
|yanked|false|
|sha|d95b77ca55ae70accd9aaeacf7883e3396663bc2c0a3c94c9fde1c9dcf2d911b|
|spec_sha|c97bc2be6e40f000e01339848eecef03ebf11bbb73a07f726b5b9f1593964d07|
|project_uri|https://rubygems.org/gems/bio-affy|
|gem_uri|https://rubygems.org/gems/bio-affy-0.5.1.gem|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.0.12"}, {"name"=>"ffi", "requirements"=>">= 1.0.11"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.4"}, {"name"=>"mkrf", "requirements"=>">= 0.2.3"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"rspec", "requirements"=>"~> 2.7.0"}], "runtime"=>[]}|

</details>

## bio-unichem

[![Gem Version](https://badge.fury.io/rb/bio-unichem.svg)](https://badge.fury.io/rb/bio-unichem)
![Year](https://img.shields.io/static/v1?label=date&message=2013-02-09&color=blue)
![Code Size](https://tokei.rs/b1/github/nakao/bioruby-unichem)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-unichem)

BioRuby plugin for UniChem REST Web service

* URL: [http://github.com/nakao/bioruby-unichem](http://github.com/nakao/bioruby-unichem)
* Authors: Mitsuteru Nakao

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-unichem|
|downloads|12088|
|version|0.3.0|
|version_created_at|2013-02-09T14:54:24.582Z|
|version_downloads|4820|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{"homepage_uri"=>"http://github.com/nakao/bioruby-unichem"}|
|yanked|false|
|sha|cc8fb5c73615db2b9b56e415d1e59d705e8aaed62ac7b9d995fdadce2c6a6f10|
|spec_sha|8cb255aa22cee162069e4461fb933bb594911c25b394bb3d883df01d809947b7|
|project_uri|https://rubygems.org/gems/bio-unichem|
|gem_uri|https://rubygems.org/gems/bio-unichem-0.3.0.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>"> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.4"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-synreport

[![Gem Version](https://badge.fury.io/rb/bio-synreport.svg)](https://badge.fury.io/rb/bio-synreport)
![Year](https://img.shields.io/static/v1?label=date&message=2013-05-16&color=blue)
![Code Size](https://tokei.rs/b1/github/danmaclean/bioruby-synreport)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-synreport)

Takes a GFF and genomic sequence file, constructs CDS and when given a position and alternative base will report whether this change is in a coding region and if it results in a synonymous or non-synonymous mutation.

* URL: [http://github.com/danmaclean/bioruby-synreport](http://github.com/danmaclean/bioruby-synreport)
* Authors: Dan MacLean

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-synreport|
|downloads|12002|
|version|0.1.2|
|version_created_at|2013-05-16T20:23:38.911Z|
|version_downloads|4671|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|fc53271e6e043a9c6b8c2d4ed7e463afc57442f37d62e7a999479dc81eb623f6|
|spec_sha|2b2eece996fb4181c7c21724b88e6c330064f0bc4fd684e8e56f5dbbf500f9eb|
|project_uri|https://rubygems.org/gems/bio-synreport|
|gem_uri|https://rubygems.org/gems/bio-synreport-0.1.2.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-synreport/0.1.2|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.4"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-locus

[![Gem Version](https://badge.fury.io/rb/bio-locus.svg)](https://badge.fury.io/rb/bio-locus)
![Year](https://img.shields.io/static/v1?label=date&message=2015-09-14&color=blue)
![Code Size](https://tokei.rs/b1/github/pjotrp/bio-locus)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-locus)

A tool for fast querying and filtering of genome locations in VCF and other formats

* URL: [http://github.com/pjotrp/bio-locus](http://github.com/pjotrp/bio-locus)
* Authors: Pjotr Prins

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-locus|
|downloads|11994|
|version|0.0.7|
|version_created_at|2015-09-14T10:06:07.839Z|
|version_downloads|5833|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|803aa9e6e68b2147df0ec0878fd76ae6ef552d74f093aba1d249ce1a7dcf750b|
|spec_sha|5374ff9a5f316885ae74b5b6c5479ce6fabe6a54c088b735e737273333c1c2c3|
|project_uri|https://rubygems.org/gems/bio-locus|
|gem_uri|https://rubygems.org/gems/bio-locus-0.0.7.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-locus/0.0.7|
|dependencies|{"development"=>[], "runtime"=>[]}|

</details>

## bio-sra

[![Gem Version](https://badge.fury.io/rb/bio-sra.svg)](https://badge.fury.io/rb/bio-sra)
![Year](https://img.shields.io/static/v1?label=date&message=2015-12-21&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-sra)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-sra)

A Sequence Read Archive (SRA) download script and Ruby interface to the SRAdb (SRA metadata) SQLite database.

* URL: [http://github.com/wwood/bioruby-sra](http://github.com/wwood/bioruby-sra)
* Authors: Ben J. Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-sra|
|downloads|11892|
|version|0.3.1|
|version_created_at|2015-12-21T06:44:38.882Z|
|version_downloads|3496|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|6ddc7b16742882bedacf288135bb171ea08c59aa9de9e8e01c38b5565969ca5e|
|spec_sha|c3fda6d4b52af5e5f96076a898e648e9ec9da782002b599fa2640576009c06f5|
|project_uri|https://rubygems.org/gems/bio-sra|
|gem_uri|https://rubygems.org/gems/bio-sra-0.3.1.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-sra/0.3.1|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2, ~> 1.4"}, {"name"=>"bio-commandeer", "requirements"=>"~> 0.1"}, {"name"=>"bundler", "requirements"=>"~> 1.6"}, {"name"=>"jeweler", "requirements"=>">= 2.0.1, ~> 2.0"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>"~> 3.5"}], "runtime"=>[{"name"=>"activerecord", "requirements"=>">= 3"}, {"name"=>"activesupport", "requirements"=>">= 3"}, {"name"=>"bio-logger", "requirements"=>"~> 1.0"}, {"name"=>"sqlite3", "requirements"=>"~> 1.3"}]}|

</details>

## bio-protparam

[![Gem Version](https://badge.fury.io/rb/bio-protparam.svg)](https://badge.fury.io/rb/bio-protparam)
![Year](https://img.shields.io/static/v1?label=date&message=2012-12-27&color=blue)
![Code Size](https://tokei.rs/b1/github/hryk/bioruby-protparam)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-protparam)

Bio::Protparam has same interface and function as Bio::Tools::Protparam class of BioPerl, except that it calculate parameters instead of throwing query to Expasy protparam tool.

* URL: [http://github.com/hryk/bioruby-protparam](http://github.com/hryk/bioruby-protparam)
* Authors: hryk

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-protparam|
|downloads|10541|
|version|0.2.0|
|version_created_at|2012-12-27T04:23:46.323Z|
|version_downloads|6309|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{"wiki_uri"=>"", "homepage_uri"=>"http://github.com/hryk/bioruby-protparam", "bug_tracker_uri"=>"https://github.com/hryk/bioruby-protparam/issues", "source_code_uri"=>"https://github.com/hryk/bioruby-protparam", "mailing_list_uri"=>"", "documentation_uri"=>"http://rdoc.info/github/hryk/bioruby-protparam/"}|
|yanked|false|
|sha|92a08b8c08f843dac4eaa7ecc23ae7b220680050a1f1c7b2699ac7561ffbf028|
|spec_sha|9a9805904c76ae42f3e23376dfb71508da1de70bff1d14e36e1452b4f4ecdd5e|
|project_uri|https://rubygems.org/gems/bio-protparam|
|gem_uri|https://rubygems.org/gems/bio-protparam-0.2.0.gem|
|documentation_uri|http://rdoc.info/github/hryk/bioruby-protparam/|
|source_code_uri|https://github.com/hryk/bioruby-protparam|
|bug_tracker_uri|https://github.com/hryk/bioruby-protparam/issues|
|dependencies|{"development"=>[{"name"=>"jeweler", "requirements"=>"~> 1.8.4"}, {"name"=>"minitest", "requirements"=>">= 0"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}]}|

</details>

## bio-core

[![Gem Version](https://badge.fury.io/rb/bio-core.svg)](https://badge.fury.io/rb/bio-core)
![Year](https://img.shields.io/static/v1?label=date&message=2011-08-30&color=blue)
![Code Size](https://tokei.rs/b1/github/helios/bioruby-core)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-core)

Bioruby core plugins

* URL: [http://github.com/helios/bioruby-core](http://github.com/helios/bioruby-core)
* Authors: Raoul J.P. Bonnal

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-core|
|downloads|9810|
|version|0.0.1|
|version_created_at|2011-08-30T12:30:46.493Z|
|version_downloads|5559|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/helios/bioruby-core"}|
|yanked|false|
|sha|22c49e1abd9ffcdbf2ac5f863c7a80c409148000aca8ea05aa2aa9e2f8ebdba8|
|spec_sha|f7e8c7fca33855ce490a74080ec3a2798dc7aaa4ba058bd1536d2706aa771012|
|project_uri|https://rubygems.org/gems/bio-core|
|gem_uri|https://rubygems.org/gems/bio-core-0.0.1.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.0"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>"= 1.4.2"}, {"name"=>"bio-assembly", "requirements"=>"= 0.1.0"}, {"name"=>"bio-gff3", "requirements"=>"= 0.8.7"}, {"name"=>"bio-logger", "requirements"=>"= 1.0.0"}, {"name"=>"bio-ucsc-api", "requirements"=>"= 0.1.0"}, {"name"=>"intermine-bio", "requirements"=>"= 0.98.1"}, {"name"=>"ruby-ensembl-api", "requirements"=>"= 1.0.1"}]}|

</details>

## bio-mummer

[![Gem Version](https://badge.fury.io/rb/bio-mummer.svg)](https://badge.fury.io/rb/bio-mummer)
![Year](https://img.shields.io/static/v1?label=date&message=2015-03-31&color=blue)
![Code Size](https://tokei.rs/b1/github/robsyme/bioruby-mummer)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-mummer)

Help for working with the output of the .delta files produced by nucmer and promer

* URL: [http://github.com/robsyme/bioruby-mummer](http://github.com/robsyme/bioruby-mummer)
* Authors: robsyme

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-mummer|
|downloads|9337|
|version|0.2.1|
|version_created_at|2015-03-31T03:32:46.915Z|
|version_downloads|3784|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|b2514d066e64132396be74c2d50ca775ca9f68e0a8229c30a825f3327a338445|
|spec_sha|9f4880884473ba98a5fe72eea5fffe44e510f7b510a3dc1a2e2e71fc45ddbb14|
|project_uri|https://rubygems.org/gems/bio-mummer|
|gem_uri|https://rubygems.org/gems/bio-mummer-0.2.1.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-mummer/0.2.1|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>"~> 2.0.0"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"simplecov", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-basespace-sdk

[![Gem Version](https://badge.fury.io/rb/bio-basespace-sdk.svg)](https://badge.fury.io/rb/bio-basespace-sdk)
![Year](https://img.shields.io/static/v1?label=date&message=2013-08-08&color=blue)
![Code Size](https://tokei.rs/b1/github/basespace/basespace-ruby-sdk)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-basespace-sdk)

BaseSpace Ruby SDK is a Ruby based Software Development Kit to be used in the development of Apps and scripts for working with Illumina's BaseSpace cloud-computing solution for next-gen sequencing data analysis.

  The primary purpose of the SDK is to provide an easy-to-use Ruby environment enabling developers to authenticate a user, retrieve data, and upload data/results from their own analysis to BaseSpace.

* URL: [https://github.com/basespace/basespace-ruby-sdk](https://github.com/basespace/basespace-ruby-sdk)
* Authors: Joachim Baran, Raoul Bonnal, Eri Kibukawa, Francesco Strozzi, Toshiaki Katayama

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-basespace-sdk|
|downloads|42394|
|version|0.1.7|
|version_created_at|2013-08-08T11:09:28.252Z|
|version_downloads|28286|
|platform|ruby|
|licenses|["Apache, Version 2.0"]|
|metadata|{}|
|yanked|false|
|sha|d5583fd60134726d96e6569b360f62bc7c20b42d8f789dddfbbed98849cf576b|
|spec_sha|f855c3cca4776630f1cf4f9f979951c55c191d520d6134b8265120abdff9f789|
|project_uri|https://rubygems.org/gems/bio-basespace-sdk|
|gem_uri|https://rubygems.org/gems/bio-basespace-sdk-0.1.7.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-basespace-sdk/0.1.7|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.1.5"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.4"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rspec", "requirements"=>"~> 2.14.1"}], "runtime"=>[{"name"=>"addressable", "requirements"=>">= 2.3.2"}, {"name"=>"getopt", "requirements"=>">= 1.4.1"}]}|

</details>

## bio-gag

[![Gem Version](https://badge.fury.io/rb/bio-gag.svg)](https://badge.fury.io/rb/bio-gag)
![Year](https://img.shields.io/static/v1?label=date&message=2012-11-05&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-gag)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-gag)

bio-gag is a biogem for detecting and correcting a particular type of error that occurs/occurred in particular versions of the IonTorrent DNA sequencing kit. Recent versions of the system don't appear to suffer the same problem

* URL: [http://github.com/wwood/bioruby-gag](http://github.com/wwood/bioruby-gag)
* Authors: Ben J Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-gag|
|downloads|9067|
|version|0.1.0|
|version_created_at|2012-11-05T23:02:37.937Z|
|version_downloads|4975|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/wwood/bioruby-gag"}|
|yanked|false|
|sha|1486d238e7597a109a5a31046ae31d2ba7f1e738857b416003773d47d276c7d0|
|spec_sha|046d6d9f478fbeacf29bf8390f45df6e7c105f901269464936529b413e2cc513|
|project_uri|https://rubygems.org/gems/bio-gag|
|gem_uri|https://rubygems.org/gems/bio-gag-0.1.0.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.3"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio-logger", "requirements"=>">= 1.0.0"}, {"name"=>"bio-pileup_iterator", "requirements"=>">= 0.0.1"}, {"name"=>"progressbar", "requirements"=>">= 0"}]}|

</details>

## bio-gggenome

[![Gem Version](https://badge.fury.io/rb/bio-gggenome.svg)](https://badge.fury.io/rb/bio-gggenome)
![Year](https://img.shields.io/static/v1?label=date&message=2013-08-13&color=blue)
![Code Size](https://tokei.rs/b1/github/nakao/bio-gggenome)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-gggenome)

A Ruby client for GGGenome the Ultrafast sequence search

* URL: [http://github.com/nakao/bio-gggenome](http://github.com/nakao/bio-gggenome)
* Authors: Mitsuteru Nakao

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-gggenome|
|downloads|8047|
|version|0.2.0|
|version_created_at|2013-08-13T14:10:32.689Z|
|version_downloads|4574|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|7b9c78fab936aa1feb258d882c73bf1c3a9bc19182a48d22689861c55df65dc4|
|spec_sha|e1af25db4492ae8eb65b6f91967bc7cd66a5aab7725cceff996c3f0968dd8eb0|
|project_uri|https://rubygems.org/gems/bio-gggenome|
|gem_uri|https://rubygems.org/gems/bio-gggenome-0.2.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-gggenome/0.2.0|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.4"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}], "runtime"=>[]}|

</details>

## bio-shell

[![Gem Version](https://badge.fury.io/rb/bio-shell.svg)](https://badge.fury.io/rb/bio-shell)
![Year](https://img.shields.io/static/v1?label=date&message=2021-11-08&color=blue)
![Code Size](https://tokei.rs/b1/github/bioruby/bioruby-shell)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-shell)

BioRuby Shell is a command line interface on BioRuby open source bioinformatics library. It provides easy-to-use analysis environment for bioinformatics.

* URL: [https://github.com/bioruby/bioruby-shell](https://github.com/bioruby/bioruby-shell)
* Authors: BioRuby project

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-shell|
|downloads|7734|
|version|1.0.1|
|version_created_at|2021-11-08T13:34:38.439Z|
|version_downloads|2413|
|platform|ruby|
|licenses|[]|
|metadata|{}|
|yanked|false|
|sha|0adddaab0e1579940820829b777d0f567859e914c2bb2feca3d6c9b12dfbe390|
|spec_sha|2b38da68c1a79e89e7b63b0d4bf053d0705bf9507e2a63888558dec3bef7a5c4|
|project_uri|https://rubygems.org/gems/bio-shell|
|gem_uri|https://rubygems.org/gems/bio-shell-1.0.1.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-shell/1.0.1|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 2.2.10"}, {"name"=>"rake", "requirements"=>">= 12.3.3"}, {"name"=>"rdoc", "requirements"=>">= 6.3.1"}, {"name"=>"test-unit", "requirements"=>"~> 3"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 2.0.0"}]}|

</details>

## bio-ipcress

[![Gem Version](https://badge.fury.io/rb/bio-ipcress.svg)](https://badge.fury.io/rb/bio-ipcress)
![Year](https://img.shields.io/static/v1?label=date&message=2017-01-09&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-ipcress)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-ipcress)

a programmatic interface to the iPCRess in-silico PCR software. iPCRess is part of the exonerate suite.

* URL: [http://github.com/wwood/bioruby-ipcress](http://github.com/wwood/bioruby-ipcress)
* Authors: Ben J Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-ipcress|
|downloads|7682|
|version|0.1.0|
|version_created_at|2017-01-09T05:09:25.224Z|
|version_downloads|3147|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|43063aaaef86699430de26b9d9e439c43a32a5ee966c9561ccb5a24e097c377d|
|spec_sha|ddf02e3457ada7e4fb7fe3575df258b4027eb66ec7ff0169494d1401ab46f9ea|
|project_uri|https://rubygems.org/gems/bio-ipcress|
|gem_uri|https://rubygems.org/gems/bio-ipcress-0.1.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-ipcress/0.1.0|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>">= 2.3.2, ~> 2.3"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"test-unit", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}]}|

</details>

## bio-repeatmasker

[![Gem Version](https://badge.fury.io/rb/bio-repeatmasker.svg)](https://badge.fury.io/rb/bio-repeatmasker)
![Year](https://img.shields.io/static/v1?label=date&message=2014-01-23&color=blue)
![Code Size](https://tokei.rs/b1/github/andreirozanski/bio-repeatmasker)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-repeatmasker)

A simple gem to parse, filter/search raw Repeat Masker output file from GoldenPath

* URL: [http://rubygems.org/gems/bio-repeatmasker](http://rubygems.org/gems/bio-repeatmasker)
* Authors: Andrei Rozanski

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-repeatmasker|
|downloads|7595|
|version|0.0.3|
|version_created_at|2014-01-23T01:00:56.459Z|
|version_downloads|4311|
|platform|ruby|
|licenses|[]|
|metadata|{}|
|yanked|false|
|sha|da491643978670fa887460ce29e8bab1afd4e36c7f65b6fa0b02a39998c061db|
|spec_sha|d8db223022302a3cc641d0ba0329bb11dbf973bde9c9f81c34aacacee18257cb|
|project_uri|https://rubygems.org/gems/bio-repeatmasker|
|gem_uri|https://rubygems.org/gems/bio-repeatmasker-0.0.3.gem|
|wiki_uri|https://github.com/andreirozanski/bio-repeatmasker|
|documentation_uri|https://github.com/andreirozanski/bio-repeatmasker|
|mailing_list_uri||
|source_code_uri|https://github.com/andreirozanski/bio-repeatmasker|
|bug_tracker_uri|https://github.com/andreirozanski/bio-repeatmasker|
|dependencies|{"development"=>[], "runtime"=>[]}|

</details>

## bio-cigar

[![Gem Version](https://badge.fury.io/rb/bio-cigar.svg)](https://badge.fury.io/rb/bio-cigar)
![Year](https://img.shields.io/static/v1?label=date&message=2014-02-28&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-cigar)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-cigar)

A parser for CIGAR format alignments.

* URL: [http://github.com/wwood/bioruby-cigar](http://github.com/wwood/bioruby-cigar)
* Authors: Ben J Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-cigar|
|downloads|7512|
|version|0.1.0|
|version_created_at|2014-02-28T03:03:24.888Z|
|version_downloads|4349|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|49c930e019d421f9013cbb1809a2663c11f4784733e2da362b19404029b1d83f|
|spec_sha|2791deac9075672dfc505c202980eebaa87c912b4a49da2d2486151c3d55149a|
|project_uri|https://rubygems.org/gems/bio-cigar|
|gem_uri|https://rubygems.org/gems/bio-cigar-0.1.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-cigar/0.1.0|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>">= 0"}, {"name"=>"rdoc", "requirements"=>">= 3.12"}, {"name"=>"rspec", "requirements"=>">= 2.8.0"}], "runtime"=>[{"name"=>"bio-samtools", "requirements"=>">= 0.6.2"}]}|

</details>

## bio-twobit

[![Gem Version](https://badge.fury.io/rb/bio-twobit.svg)](https://badge.fury.io/rb/bio-twobit)
![Year](https://img.shields.io/static/v1?label=date&message=2024-11-19&color=blue)
![Code Size](https://tokei.rs/b1/github/ruby-on-bioc/bio-twobit)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-twobit)

This is a Ruby binding for lib2bit(https://github.com/dpryan79/lib2bit), which provides high-speed access to genomic data in 2bit file format.

* URL: [https://github.com/ruby-on-bioc/bio-twobit](https://github.com/ruby-on-bioc/bio-twobit)
* Authors: kojix2

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-twobit|
|downloads|7277|
|version|0.2.3|
|version_created_at|2024-11-19T03:49:52.264Z|
|version_downloads|211|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|230e25e0d935ab9dc25b8986bd29e662e126bb7176ddde92791d799851991c25|
|spec_sha|f67c293a97d8a5698cdb9ae6691441d59de3f918209f2c7501667b0d5f684e3a|
|project_uri|https://rubygems.org/gems/bio-twobit|
|gem_uri|https://rubygems.org/gems/bio-twobit-0.2.3.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-twobit/0.2.3|
|dependencies|{"development"=>[], "runtime"=>[]}|

</details>

## bio-nwk

[![Gem Version](https://badge.fury.io/rb/bio-nwk.svg)](https://badge.fury.io/rb/bio-nwk)
![Year](https://img.shields.io/static/v1?label=date&message=2022-04-06&color=blue)
![Code Size]()
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-nwk)

Some codes for parsing newick tree

* URL: []()
* Authors: Sishuo Wang

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-nwk|
|downloads|7201|
|version|0.1.4|
|version_created_at|2022-04-06T07:23:50.620Z|
|version_downloads|1273|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|d32005e7e74a43acf247f528bcca2f54cbb74954b7fb1c17eeecfb5bf4d7a7d5|
|spec_sha|a0d7efc33e9c629ccfbf1bfb66cb05ae1344a0e431212e5ce3e62bb9e0014dad|
|project_uri|https://rubygems.org/gems/bio-nwk|
|gem_uri|https://rubygems.org/gems/bio-nwk-0.1.4.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-nwk/0.1.4|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>"~> 1.5"}, {"name"=>"bundler", "requirements"=>"~> 2.0"}, {"name"=>"rake", "requirements"=>"~> 10.0"}, {"name"=>"rspec", "requirements"=>"~> 3.0"}], "runtime"=>[]}|

</details>

## bio-gfastqc

[![Gem Version](https://badge.fury.io/rb/bio-gfastqc.svg)](https://badge.fury.io/rb/bio-gfastqc)
![Year](https://img.shields.io/static/v1?label=date&message=2015-06-26&color=blue)
![Code Size](https://tokei.rs/b1/github/helios/bioruby-gfastqc)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-gfastqc)

Bioinformatics. Aggregate FastQC (quality control for Next Generation Sequencing -NGS-) results from many different samples in a single web page, with charts and tables organized and simplified. The main goal is to speed up the communication process with colleagues (PIs, Biologists, BioInformaticians).

* URL: [http://github.com/helios/bioruby-gfastqc](http://github.com/helios/bioruby-gfastqc)
* Authors: Raoul Jean Pierre Bonnal

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-gfastqc|
|downloads|6548|
|version|0.0.4|
|version_created_at|2015-06-26T10:55:54.307Z|
|version_downloads|3715|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|66fe6425db289603d24533388b8c6734ca4de5a8843dbb817c6bbde4d5f71815|
|spec_sha|9f046fdf4d7be42f707466c3a7fede4755281d51aa052c7f70a7fd1a69daa511|
|project_uri|https://rubygems.org/gems/bio-gfastqc|
|gem_uri|https://rubygems.org/gems/bio-gfastqc-0.0.4.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-gfastqc/0.0.4|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>">= 0"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"simplecov", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"rubyzip", "requirements"=>">= 0"}]}|

</details>

## bio-freebayes

[![Gem Version](https://badge.fury.io/rb/bio-freebayes.svg)](https://badge.fury.io/rb/bio-freebayes)
![Year](https://img.shields.io/static/v1?label=date&message=2015-05-22&color=blue)
![Code Size]()
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-freebayes)

The API for Freebayes version #{Bio::Freebayes::EXTVERSION}

* URL: [http://rubygems.org/gems/bio-freebayes](http://rubygems.org/gems/bio-freebayes)
* Authors: Natapol Pornputtapong

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-freebayes|
|downloads|6464|
|version|0.1.6|
|version_created_at|2015-05-22T19:00:14.317Z|
|version_downloads|3746|
|platform|ruby|
|licenses|[]|
|metadata|{"allowed_push_host"=>"TODO: Set to 'http://mygemserver.com'"}|
|yanked|false|
|sha|2126b998266da18cc7a9b111c907beadc803fa7d3cd99fd38e8273e31f30f146|
|spec_sha|6d344d0e83c86d749439a4463a318446c3d218926d33eb70a65ce74520628553|
|project_uri|https://rubygems.org/gems/bio-freebayes|
|gem_uri|https://rubygems.org/gems/bio-freebayes-0.1.6.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-freebayes/0.1.6|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.9"}, {"name"=>"rake", "requirements"=>"~> 10.0"}], "runtime"=>[{"name"=>"bio-vcf", "requirements"=>"~> 0.8"}, {"name"=>"parallel", "requirements"=>"~> 1.4"}]}|

</details>

## bio-kseq

[![Gem Version](https://badge.fury.io/rb/bio-kseq.svg)](https://badge.fury.io/rb/bio-kseq)
![Year](https://img.shields.io/static/v1?label=date&message=2014-07-16&color=blue)
![Code Size](https://tokei.rs/b1/github/gusevfe/bio-kseq)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-kseq)

A fast FASTA/FASTQ parser based on kseq.h by Heng Li

* URL: [https://github.com/gusevfe/bio-kseq](https://github.com/gusevfe/bio-kseq)
* Authors: Fedor Gusev

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-kseq|
|downloads|6141|
|version|0.0.2|
|version_created_at|2014-07-16T08:21:38.034Z|
|version_downloads|6134|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|3fcfd82492a0c59fd906b6c6251361ea290908386ecdcf0d9365c1698882caf7|
|spec_sha|12b08d733e55f5e99333917075abccb022fabc4add326813c35fe17d2691da71|
|project_uri|https://rubygems.org/gems/bio-kseq|
|gem_uri|https://rubygems.org/gems/bio-kseq-0.0.2.gem|
|wiki_uri||
|documentation_uri|https://www.rubydoc.info/gems/bio-kseq/0.0.2|
|mailing_list_uri||
|source_code_uri|https://github.com/gusevfe/bio-kseq|
|bug_tracker_uri|https://github.com/gusevfe/bio-kseq/issues|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.5"}, {"name"=>"rake", "requirements"=>">= 0"}, {"name"=>"rake-compiler", "requirements"=>">= 0"}, {"name"=>"rspec", "requirements"=>">= 3.0.0"}], "runtime"=>[]}|

</details>

## bio-pangenome

[![Gem Version](https://badge.fury.io/rb/bio-pangenome.svg)](https://badge.fury.io/rb/bio-pangenome)
![Year](https://img.shields.io/static/v1?label=date&message=2020-04-25&color=blue)
![Code Size](https://tokei.rs/b1/github/Uauy-Lab/bioruby-pangenome)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-pangenome)

Tools to find similarity between pangenomes.

* URL: [http://github.com/Uauy-Lab/bioruby-pangenome](http://github.com/Uauy-Lab/bioruby-pangenome)
* Authors: Ricardo H. Ramirez-Gonzalez

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-pangenome|
|downloads|6125|
|version|0.1.3|
|version_created_at|2020-04-25T10:07:12.305Z|
|version_downloads|2278|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|325b2b4fbb37d0709d422a53160cf2ed93a190103ca53773ab2fe92b63d9974d|
|spec_sha|d59b83eede7100a636846fa8037a118babc34193182d6212794a0deca71fd99b|
|project_uri|https://rubygems.org/gems/bio-pangenome|
|gem_uri|https://rubygems.org/gems/bio-pangenome-0.1.3.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-pangenome/0.1.3|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"juwelier", "requirements"=>">= 0"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"simplecov", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 0"}, {"name"=>"bio-blastxmlparser", "requirements"=>">= 0"}, {"name"=>"bio-gff3", "requirements"=>">= 0"}]}|

</details>

## bio-tm_hmm

[![Gem Version](https://badge.fury.io/rb/bio-tm_hmm.svg)](https://badge.fury.io/rb/bio-tm_hmm)
![Year](https://img.shields.io/static/v1?label=date&message=2017-04-06&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-tm_hmm)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-tm_hmm)

A bioruby plugin for interaction with the transmembrane predictor TMHMM

* URL: [http://github.com/wwood/bioruby-tm_hmm](http://github.com/wwood/bioruby-tm_hmm)
* Authors: Ben J. Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-tm_hmm|
|downloads|26051|
|version|0.2.4|
|version_created_at|2017-04-06T07:57:43.742Z|
|version_downloads|2884|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|2b02a4377453acf6acbba02e7149424ede1979f804e1b3294168bc48c6dbc1f0|
|spec_sha|55479df541de471045ae121d94d9e63f68116cc8311b69683e058ab6ebe0188b|
|project_uri|https://rubygems.org/gems/bio-tm_hmm|
|gem_uri|https://rubygems.org/gems/bio-tm_hmm-0.2.4.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-tm_hmm/0.2.4|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>"~> 2.3"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"test-unit", "requirements"=>"~> 3.2"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}]}|

</details>

## bio-dbla-classifier

[![Gem Version](https://badge.fury.io/rb/bio-dbla-classifier.svg)](https://badge.fury.io/rb/bio-dbla-classifier)
![Year](https://img.shields.io/static/v1?label=date&message=2013-02-22&color=blue)
![Code Size](https://tokei.rs/b1/github/georgeG/bioruby-dbla-classifier)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-dbla-classifier)

Methods to classify and manipulate PfEMP1 DBL-alpha sequence tags

* URL: [http://github.com/georgeG/bioruby-dbla-classifier](http://github.com/georgeG/bioruby-dbla-classifier)
* Authors: George Githinji

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-dbla-classifier|
|downloads|25838|
|version|0.7.1|
|version_created_at|2013-02-22T09:40:39.257Z|
|version_downloads|4794|
|platform|ruby|
|licenses|["Ruby"]|
|metadata|{"homepage_uri"=>"http://github.com/georgeG/bioruby-dbla-classifier"}|
|yanked|false|
|sha|be5f1c68997909ec02ade7697585f98c29fb802752c3dbf3f40a75a57034c57e|
|spec_sha|b6593a8c954839981a39e4663fd44c428fd83f01470421c94a5f79ea9393a7b4|
|project_uri|https://rubygems.org/gems/bio-dbla-classifier|
|gem_uri|https://rubygems.org/gems/bio-dbla-classifier-0.7.1.gem|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 1.0.22"}, {"name"=>"jeweler", "requirements"=>">= 1.6.4"}, {"name"=>"rake", "requirements"=>">= 0.9.2.2"}, {"name"=>"rspec", "requirements"=>">= 2.8.0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.3"}]}|

</details>

## bio-executables

[![Gem Version](https://badge.fury.io/rb/bio-executables.svg)](https://badge.fury.io/rb/bio-executables)
![Year](https://img.shields.io/static/v1?label=date&message=2019-06-14&color=blue)
![Code Size](https://tokei.rs/b1/github/bioruby/bioruby-executables)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-executables)

BioRuby executables is a collection of miscellaneous utilities for bioinformatics that were formerly part of the BioRuby core ("bio" gem). Because of reducing complexity, they were moved to this gem.

* URL: [https://github.com/bioruby/bioruby-executables](https://github.com/bioruby/bioruby-executables)
* Authors: BioRuby project

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-executables|
|downloads|5891|
|version|1.0.0|
|version_created_at|2019-06-14T05:36:59.331Z|
|version_downloads|3659|
|platform|ruby|
|licenses|[]|
|metadata|{}|
|yanked|false|
|sha|29f02115bc543d7f39b0c6d4c518719dcae9d112a913c215b988b599a6736b91|
|spec_sha|787f23febc797a1606450490e685bf021395aff15e80d532e67343a5b6c17afc|
|project_uri|https://rubygems.org/gems/bio-executables|
|gem_uri|https://rubygems.org/gems/bio-executables-1.0.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-executables/1.0.0|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.15"}, {"name"=>"rake", "requirements"=>"~> 10.0"}, {"name"=>"rdoc", "requirements"=>"~> 5"}, {"name"=>"test-unit", "requirements"=>"~> 3"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 2.0.0"}, {"name"=>"bio-old-biofetch-emulator", "requirements"=>">= 0.0.1"}]}|

</details>

## bio-sge

[![Gem Version](https://badge.fury.io/rb/bio-sge.svg)](https://badge.fury.io/rb/bio-sge)
![Year](https://img.shields.io/static/v1?label=date&message=2010-12-23&color=blue)
![Code Size](https://tokei.rs/b1/github/ktym/bioruby-sge)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-sge)

Entries in a flatfile will be parased by the BioRuby's Bio::FlatFile.auto module. These entries are used as queries for the Sun Grid Engine (SGE) system. Huge amount of queries are automatically splitted into subdirectories. With a specified command line to be executed, queries are submited to the SGE as an array job.

* URL: [http://github.com/ktym/bioruby-sge](http://github.com/ktym/bioruby-sge)
* Authors: Toshiaki Katayama

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-sge|
|downloads|5825|
|version|0.0.0|
|version_created_at|2010-12-23T23:50:10.549Z|
|version_downloads|5825|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/ktym/bioruby-sge"}|
|yanked|false|
|sha|8478f29051e2d56dcf7c795ec1e1efba8a3840f393e326fdd216accf9a9600c1|
|spec_sha|378c982fab4ae54b9a2e11065eb65a19bc68a3c0327f1575de3d6dff2d051832|
|project_uri|https://rubygems.org/gems/bio-sge|
|gem_uri|https://rubygems.org/gems/bio-sge-0.0.0.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.5.2"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-hello

[![Gem Version](https://badge.fury.io/rb/bio-hello.svg)](https://badge.fury.io/rb/bio-hello)
![Year](https://img.shields.io/static/v1?label=date&message=2010-12-31&color=blue)
![Code Size](https://tokei.rs/b1/github/ktym/bioruby-hello)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-hello)

Encode any message string into a DNA sequence

* URL: [http://github.com/ktym/bioruby-hello](http://github.com/ktym/bioruby-hello)
* Authors: Toshiaki Katayama

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-hello|
|downloads|5740|
|version|0.0.0|
|version_created_at|2010-12-31T04:24:19.169Z|
|version_downloads|5740|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/ktym/bioruby-hello"}|
|yanked|false|
|sha|3d2ad0c17e1a407b57e49d456f02f630ef129bac19d412e565be3279d2a4ca3c|
|spec_sha|59116210631a83ee6c18ae13c82bfd3708b8ea229b9508745fe1f2b48f8e6b13|
|project_uri|https://rubygems.org/gems/bio-hello|
|gem_uri|https://rubygems.org/gems/bio-hello-0.0.0.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.5.2"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-kmer_counter

[![Gem Version](https://badge.fury.io/rb/bio-kmer_counter.svg)](https://badge.fury.io/rb/bio-kmer_counter)
![Year](https://img.shields.io/static/v1?label=date&message=2013-05-30&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-kmer_counter)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-kmer_counter)

A biogem for counting small kmers for fingerprinting nucleotide sequences. See README for details.

* URL: [http://github.com/wwood/bioruby-kmer_counter](http://github.com/wwood/bioruby-kmer_counter)
* Authors: Ben J Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-kmer_counter|
|downloads|24222|
|version|0.1.2|
|version_created_at|2013-05-30T04:06:14.659Z|
|version_downloads|4843|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|fdc8fd8bea74a468545f05d9b12375f65657dffcffd7c1dd25ee2af423b27826|
|spec_sha|8f390f858fa40c570d761d9a8686e101be7757b18131bc18dd80f609153d7179|
|project_uri|https://rubygems.org/gems/bio-kmer_counter|
|gem_uri|https://rubygems.org/gems/bio-kmer_counter-0.1.2.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-kmer_counter/0.1.2|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>">= 1.8.3"}, {"name"=>"rdoc", "requirements"=>">= 3.12"}, {"name"=>"rdoc", "requirements"=>">= 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bio-logger", "requirements"=>">= 1.0.1"}, {"name"=>"parallel", "requirements"=>">= 0.5.17"}, {"name"=>"progressbar", "requirements"=>">= 0.11.0"}]}|

</details>

## bio-biolinux

[![Gem Version](https://badge.fury.io/rb/bio-biolinux.svg)](https://badge.fury.io/rb/bio-biolinux)
![Year](https://img.shields.io/static/v1?label=date&message=2011-09-24&color=blue)
![Code Size](https://tokei.rs/b1/github/helios/bioruby-biolinux)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-biolinux)

Meta package for BioLinux distribution: bio-core, bio-core-ext

* URL: [http://github.com/helios/bioruby-biolinux](http://github.com/helios/bioruby-biolinux)
* Authors: Raoul J.P. Bonnal

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-biolinux|
|downloads|5473|
|version|0.1.0|
|version_created_at|2011-09-24T13:03:12.725Z|
|version_downloads|5473|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/helios/bioruby-biolinux"}|
|yanked|false|
|sha|6bfa59585fd0701cdb311c221ccafbcdfa2bb73bb61757640f6242fe5ebb0d95|
|spec_sha|7d0454cf41d26ca15fc1b359bffa9970e351da092b6dc4e84e8dff40699d0941|
|project_uri|https://rubygems.org/gems/bio-biolinux|
|gem_uri|https://rubygems.org/gems/bio-biolinux-0.1.0.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.4"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio-core", "requirements"=>"= 0.0.1"}, {"name"=>"bio-core-ext", "requirements"=>"= 0.0.1"}]}|

</details>

## bio-ensembl

[![Gem Version](https://badge.fury.io/rb/bio-ensembl.svg)](https://badge.fury.io/rb/bio-ensembl)
![Year](https://img.shields.io/static/v1?label=date&message=2011-12-06&color=blue)
![Code Size](https://tokei.rs/b1/github/fstrozzi/bioruby-ensembl)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-ensembl)

This API provides a complete set of methods and classes to access the Ensembl database using Ruby programming language

* URL: [http://github.com/fstrozzi/bioruby-ensembl](http://github.com/fstrozzi/bioruby-ensembl)
* Authors: Jan Aerts, Francesco Strozzi

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-ensembl|
|downloads|5438|
|version|1.1.0|
|version_created_at|2011-12-06T14:58:11.373Z|
|version_downloads|5438|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/fstrozzi/bioruby-ensembl"}|
|yanked|false|
|sha|1a8037bfc6030278c8be07b9ee51af40e2901d91a16f4cd86298c1e7e4ffeac0|
|spec_sha|81090a9273c1ea3b3dcd77fe9760b5e4ce64cc469c5128297606feb166d90745|
|project_uri|https://rubygems.org/gems/bio-ensembl|
|gem_uri|https://rubygems.org/gems/bio-ensembl-1.1.0.gem|
|dependencies|{"development"=>[{"name"=>"activerecord", "requirements"=>"= 3.0.0"}, {"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.5.2"}, {"name"=>"mysql", "requirements"=>">= 0"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"activerecord", "requirements"=>"= 3.0.0"}, {"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"mysql", "requirements"=>">= 0"}]}|

</details>

## bio-dbsnp

[![Gem Version](https://badge.fury.io/rb/bio-dbsnp.svg)](https://badge.fury.io/rb/bio-dbsnp)
![Year](https://img.shields.io/static/v1?label=date&message=2011-11-03&color=blue)
![Code Size](https://tokei.rs/b1/github/misshie/bioruby-dbsnp)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-dbsnp)

decoding the dbSNP bitfield containg detaild information

* URL: [http://github.com/misshie/bioruby-dbsnp](http://github.com/misshie/bioruby-dbsnp)
* Authors: Hiroyuki Mishima

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-dbsnp|
|downloads|5389|
|version|0.1.1|
|version_created_at|2011-11-03T13:42:06.325Z|
|version_downloads|5389|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/misshie/bioruby-dbsnp"}|
|yanked|false|
|sha|4cefcd706da46d83a743ed9b919726e41fa596a07915350ed7f346ce49bf5cf5|
|spec_sha|0c8a171a5327d7aa436292a99b5ad2c1ec028530414b403bed5de80bf6dff62b|
|project_uri|https://rubygems.org/gems/bio-dbsnp|
|gem_uri|https://rubygems.org/gems/bio-dbsnp-0.1.1.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.4"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"rspec", "requirements"=>"~> 2.3.0"}], "runtime"=>[]}|

</details>

## bio-hydropathy

[![Gem Version](https://badge.fury.io/rb/bio-hydropathy.svg)](https://badge.fury.io/rb/bio-hydropathy)
![Year](https://img.shields.io/static/v1?label=date&message=2012-01-26&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-hydropathy)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-hydropathy)

Hydropathy scale for BioRuby

* URL: [http://github.com/wwood/bioruby-hydropathy](http://github.com/wwood/bioruby-hydropathy)
* Authors: Ben J Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-hydropathy|
|downloads|5367|
|version|0.0.1|
|version_created_at|2012-01-26T06:26:30.290Z|
|version_downloads|5367|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/wwood/bioruby-hydropathy"}|
|yanked|false|
|sha|3534b79494d849022b446336d38cf130dd10bb4285cd1b6880bbb477b4b5c003|
|spec_sha|8cd76f0d927221d756ab58ff06f81d5654906b1ee1e76dae817e7f1fd249badb|
|project_uri|https://rubygems.org/gems/bio-hydropathy|
|gem_uri|https://rubygems.org/gems/bio-hydropathy-0.0.1.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.4"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-phyloxml

[![Gem Version](https://badge.fury.io/rb/bio-phyloxml.svg)](https://badge.fury.io/rb/bio-phyloxml)
![Year](https://img.shields.io/static/v1?label=date&message=2012-05-24&color=blue)
![Code Size](https://tokei.rs/b1/github/csw/bioruby-phyloxml)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-phyloxml)

Provides PhyloXML support for BioRuby.

* URL: [http://github.com/csw/bioruby-phyloxml](http://github.com/csw/bioruby-phyloxml)
* Authors: Diana Jaunzeikare, Clayton Wheeler

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-phyloxml|
|downloads|5339|
|version|0.9.0|
|version_created_at|2012-05-24T17:03:43.643Z|
|version_downloads|5339|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/csw/bioruby-phyloxml"}|
|yanked|false|
|sha|d56ac15a573582782ca25d697a839345c832a201180fc4a89d4cdacdf3662e8d|
|spec_sha|8e42a3bcf740cd90b86770a3c38d056ae4f6cd50516159c828c083f78cae0432|
|project_uri|https://rubygems.org/gems/bio-phyloxml|
|gem_uri|https://rubygems.org/gems/bio-phyloxml-0.9.0.gem|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.1.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.3"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}], "runtime"=>[{"name"=>"bio", "requirements"=>"~> 1.4.2"}, {"name"=>"libxml-ruby", "requirements"=>"~> 2.3.2"}]}|

</details>

## bio-foo

[![Gem Version](https://badge.fury.io/rb/bio-foo.svg)](https://badge.fury.io/rb/bio-foo)
![Year](https://img.shields.io/static/v1?label=date&message=2011-11-01&color=blue)
![Code Size](https://tokei.rs/b1/github/ktym/bioruby-foo)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-foo)

longer description of your gem

* URL: [http://github.com/ktym/bioruby-foo](http://github.com/ktym/bioruby-foo)
* Authors: Toshiaki Katayama

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-foo|
|downloads|5323|
|version|0.0.0|
|version_created_at|2011-11-01T22:31:24.748Z|
|version_downloads|5323|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/ktym/bioruby-foo"}|
|yanked|false|
|sha|7c73173934faff49a0e66373bde467986c5e8cf6544dd0b75b6c68b3c553cac1|
|spec_sha|acfee0618a0cfd308034397dfdca15d9f7a49690aa96be8aa62c7bdda922d696|
|project_uri|https://rubygems.org/gems/bio-foo|
|gem_uri|https://rubygems.org/gems/bio-foo-0.0.0.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.4"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-exportpred

[![Gem Version](https://badge.fury.io/rb/bio-exportpred.svg)](https://badge.fury.io/rb/bio-exportpred)
![Year](https://img.shields.io/static/v1?label=date&message=2012-01-20&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-exportpred)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-exportpred)

Wrapper around the ExportPred algorithm for predicting P. falciparum exported proteins. Requires local install of the program, which is available from http://bioinf.wehi.edu.au/exportpred/

* URL: [http://github.com/wwood/bioruby-exportpred](http://github.com/wwood/bioruby-exportpred)
* Authors: Ben J Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-exportpred|
|downloads|5319|
|version|0.0.1|
|version_created_at|2012-01-20T01:00:12.037Z|
|version_downloads|5319|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/wwood/bioruby-exportpred"}|
|yanked|false|
|sha|49439427fb5c0e33eb43d6e71f067d87bd5693911b8cdbe54fb61969d4050b3d|
|spec_sha|699ce435da6b1050a5c4e5d61b07aad1f0590c8c4c7d3d689d83e701ee4babe9|
|project_uri|https://rubygems.org/gems/bio-exportpred|
|gem_uri|https://rubygems.org/gems/bio-exportpred-0.0.1.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.4"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-incanter

[![Gem Version](https://badge.fury.io/rb/bio-incanter.svg)](https://badge.fury.io/rb/bio-incanter)
![Year](https://img.shields.io/static/v1?label=date&message=2012-08-06&color=blue)
![Code Size](https://tokei.rs/b1/github/cerbero/bio-incanter)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-incanter)

A Jruby wrapper to Incanter lib

* URL: [http://github.com/cerbero/bio-incanter](http://github.com/cerbero/bio-incanter)
* Authors: Matteo Picciolini

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-incanter|
|downloads|5258|
|version|0.1.0|
|version_created_at|2012-08-06T15:08:41.459Z|
|version_downloads|5258|
|platform|ruby|
|metadata|{"wiki_uri"=>"https://github.com/cerbero/bioruby-incanter/wiki", "homepage_uri"=>"http://github.com/cerbero/bio-incanter", "bug_tracker_uri"=>"", "source_code_uri"=>"https://github.com/cerbero/bioruby-incanter", "mailing_list_uri"=>"", "documentation_uri"=>"https://github.com/cerbero/bioruby-incanter/tree/master/doc"}|
|yanked|false|
|sha|81e1a2c905612b52eaf46280ef520cd07eef9352a655caff5621a744065979ed|
|spec_sha|2e518ab4eae49901306515ef870f73970bd830c10ac7cadf25c322f8d9d3c206|
|project_uri|https://rubygems.org/gems/bio-incanter|
|gem_uri|https://rubygems.org/gems/bio-incanter-0.1.0.gem|
|wiki_uri|https://github.com/cerbero/bioruby-incanter/wiki|
|documentation_uri|https://github.com/cerbero/bioruby-incanter/tree/master/doc|
|source_code_uri|https://github.com/cerbero/bioruby-incanter|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"< 1.1.5"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.4"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rspec", "requirements"=>"~> 2.8.0"}], "runtime"=>[{"name"=>"jruby-openssl", "requirements"=>">= 0"}]}|

</details>

## bio-gemma-wrapper

[![Gem Version](https://badge.fury.io/rb/bio-gemma-wrapper.svg)](https://badge.fury.io/rb/bio-gemma-wrapper)
![Year](https://img.shields.io/static/v1?label=date&message=2024-07-06&color=blue)
![Code Size](https://tokei.rs/b1/github/genetics-statistics/gemma-wrapper)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-gemma-wrapper)

GEMMA wrapper adds LOCO and permutation support. Also runs in parallel and caches K between runs with LOCO support

* URL: [https://github.com/genetics-statistics/gemma-wrapper](https://github.com/genetics-statistics/gemma-wrapper)
* Authors: Pjotr Prins

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-gemma-wrapper|
|downloads|21934|
|version|0.99.7|
|version_created_at|2024-07-06T10:28:49.963Z|
|version_downloads|294|
|platform|ruby|
|licenses|["GPL3"]|
|metadata|{}|
|yanked|false|
|sha|44ce59070c0507e803dce7aa51356c7b2701c317cc5208f69f428e973b980689|
|spec_sha|fcb359ab711f7acdea025a831a3fbac1406a608f3bf082d25b1c8d28099cf050|
|project_uri|https://rubygems.org/gems/bio-gemma-wrapper|
|gem_uri|https://rubygems.org/gems/bio-gemma-wrapper-0.99.7.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-gemma-wrapper/0.99.7|
|dependencies|{"development"=>[], "runtime"=>[]}|

</details>

## bioruby-bio

[![Gem Version](https://badge.fury.io/rb/bioruby-bio.svg)](https://badge.fury.io/rb/bioruby-bio)
![Year](https://img.shields.io/static/v1?label=date&message=2014-08-11&color=blue)
![Code Size]()
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bioruby-bio)

BioRuby is a library for bioinformatics (biology + information science).

* URL: [http://bioruby.org/](http://bioruby.org/)
* Authors: BioRuby project

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bioruby-bio|
|downloads|19680|
|version|1.3.1.5000|
|version_created_at|2014-08-11T07:28:26.580Z|
|version_downloads|4181|
|platform|ruby|
|licenses|[]|
|metadata|{}|
|yanked|false|
|sha|6f8bf3defd75fd58f6943073f42216a841d2fa05090174e5a7428e2491dc326e|
|spec_sha|440cc305a4716750f2b7556755873e3463c4f0061ed7cce1067cc283968819f0|
|project_uri|https://rubygems.org/gems/bioruby-bio|
|gem_uri|https://rubygems.org/gems/bioruby-bio-1.3.1.5000.gem|
|documentation_uri|https://www.rubydoc.info/gems/bioruby-bio/1.3.1.5000|
|dependencies|{"development"=>[], "runtime"=>[]}|

</details>

## bio-fasta_retrieve

[![Gem Version](https://badge.fury.io/rb/bio-fasta_retrieve.svg)](https://badge.fury.io/rb/bio-fasta_retrieve)
![Year](https://img.shields.io/static/v1?label=date&message=2015-08-27&color=blue)
![Code Size](https://tokei.rs/b1/github/andreirozanski/bio-fasta_retrieve)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-fasta_retrieve)

Gem that helps to retrieve DNA sequence from UCSC

* URL: [http://rubygems.org/gems/bio-fasta_retrieve.rb](http://rubygems.org/gems/bio-fasta_retrieve.rb)
* Authors: Andrei Rozanski

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-fasta_retrieve|
|downloads|21186|
|version|0.0.7|
|version_created_at|2015-08-27T10:51:39.816Z|
|version_downloads|3632|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|c1b648653f1820e044a5637c1af6af469b002a035bb677776d47d1ccc67ead04|
|spec_sha|3684db0902863e3e8e3ba7dd012b5e0ee21d55da944c4a8d8ad41937573151fb|
|project_uri|https://rubygems.org/gems/bio-fasta_retrieve|
|gem_uri|https://rubygems.org/gems/bio-fasta_retrieve-0.0.7.gem|
|wiki_uri|https://github.com/andreirozanski/bio-fasta_retrieve|
|documentation_uri|https://github.com/andreirozanski/bio-fasta_retrieve|
|mailing_list_uri|https://github.com/andreirozanski/bio-fasta_retrieve|
|source_code_uri|https://github.com/andreirozanski/bio-fasta_retrieve|
|bug_tracker_uri|https://github.com/andreirozanski/bio-fasta_retrieve|
|dependencies|{"development"=>[], "runtime"=>[{"name"=>"bio", "requirements"=>"~> 1.5"}, {"name"=>"nokogiri", "requirements"=>">= 1.6.6.2, ~> 1.6"}]}|

</details>

## bio-stockholm

[![Gem Version](https://badge.fury.io/rb/bio-stockholm.svg)](https://badge.fury.io/rb/bio-stockholm)
![Year](https://img.shields.io/static/v1?label=date&message=2013-06-19&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-stockholm)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-stockholm)

Parses stockholm sequence alignment format

* URL: [http://github.com/wwood/bioruby-stockholm](http://github.com/wwood/bioruby-stockholm)
* Authors: Ben J. Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-stockholm|
|downloads|4811|
|version|0.0.1|
|version_created_at|2013-06-19T03:43:25.646Z|
|version_downloads|4811|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|c6407901f14090d5fcaf805a2f02d399755bfb106debbc908247438fec49d8e7|
|spec_sha|30e3f99633e24439b66599a23d3d91add7dfa63c83a38a6c6a73912c867ec05b|
|project_uri|https://rubygems.org/gems/bio-stockholm|
|gem_uri|https://rubygems.org/gems/bio-stockholm-0.0.1.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-stockholm/0.0.1|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.4"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rspec", "requirements"=>"~> 2.8.0"}], "runtime"=>[]}|

</details>

## bio-raxml

[![Gem Version](https://badge.fury.io/rb/bio-raxml.svg)](https://badge.fury.io/rb/bio-raxml)
![Year](https://img.shields.io/static/v1?label=date&message=2013-07-16&color=blue)
![Code Size](https://tokei.rs/b1/github/fizquierdo/bioruby-raxml)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-raxml)

Just a test

* URL: [http://github.com/fizquierdo/bioruby-raxml](http://github.com/fizquierdo/bioruby-raxml)
* Authors: Fernando Izquierdo

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-raxml|
|downloads|4652|
|version|0.1.0|
|version_created_at|2013-07-16T13:26:18.440Z|
|version_downloads|4652|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|6d57e168d15bf1265bffb37a0b0ffd5ade3f1acc3b2641fd6b4930ee36f3224d|
|spec_sha|e6e7f214d71952d0a2972fb11c0e4ecd9b72bcb766429c7c62e57148e1809276|
|project_uri|https://rubygems.org/gems/bio-raxml|
|gem_uri|https://rubygems.org/gems/bio-raxml-0.1.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-raxml/0.1.0|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>"~> 1.3.5"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.4"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-isoelectric_point

[![Gem Version](https://badge.fury.io/rb/bio-isoelectric_point.svg)](https://badge.fury.io/rb/bio-isoelectric_point)
![Year](https://img.shields.io/static/v1?label=date&message=2012-03-01&color=blue)
![Code Size](https://tokei.rs/b1/github/georgeG/bioruby-isoelectric_point)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-isoelectric_point)

A bioruby plugin for calculating the isoelectric point of a protein 

* URL: [http://github.com/georgeG/bioruby-isoelectric_point](http://github.com/georgeG/bioruby-isoelectric_point)
* Authors: George Githinji, Pascal Betz

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-isoelectric_point|
|downloads|19204|
|version|0.1.3|
|version_created_at|2012-03-01T06:16:49.922Z|
|version_downloads|5952|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/georgeG/bioruby-isoelectric_point"}|
|yanked|false|
|sha|744f197fb06803c78d41dc6dbab1aec14070d059ab6dff0c778b2d32a9875659|
|spec_sha|c3fd36909dfd493da387f33e1dd140930d3de76ba519e7618e30719fbcc6f1eb|
|project_uri|https://rubygems.org/gems/bio-isoelectric_point|
|gem_uri|https://rubygems.org/gems/bio-isoelectric_point-0.1.3.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>"~> 1.0.22"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.3"}, {"name"=>"shoulda", "requirements"=>"~> 2.11.3"}], "runtime"=>[]}|

</details>

## bio-velvet_underground

[![Gem Version](https://badge.fury.io/rb/bio-velvet_underground.svg)](https://badge.fury.io/rb/bio-velvet_underground)
![Year](https://img.shields.io/static/v1?label=date&message=2016-08-01&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-velvet_underground)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-velvet_underground)

Bindings to some internals of the velvet assembler.

* URL: [http://github.com/wwood/bioruby-velvet_underground](http://github.com/wwood/bioruby-velvet_underground)
* Authors: Ben Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-velvet_underground|
|downloads|18867|
|version|0.3.1|
|version_created_at|2016-08-01T11:19:19.014Z|
|version_downloads|3204|
|platform|ruby|
|licenses|["GPL"]|
|metadata|{}|
|yanked|false|
|sha|1fd95b6a66e8308041d8a68c5094961d5b01fb7e3892a1d80de08464cdc1d7f1|
|spec_sha|5902d3cfc33c512ec7d1f206aa5df48186a8a051b093826d96aaca1cd5d5e56d|
|project_uri|https://rubygems.org/gems/bio-velvet_underground|
|gem_uri|https://rubygems.org/gems/bio-velvet_underground-0.3.1.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-velvet_underground/0.3.1|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>"~> 2.0"}, {"name"=>"pry", "requirements"=>"> 0"}, {"name"=>"rspec", "requirements"=>"~> 2.8"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"simplecov", "requirements"=>">= 0"}, {"name"=>"yard", "requirements"=>"~> 0.7"}], "runtime"=>[{"name"=>"bio-logger", "requirements"=>"~> 1.0"}, {"name"=>"bio-velvet", "requirements"=>"~> 0.6"}, {"name"=>"ffi", "requirements"=>"~> 1.9"}]}|

</details>

## bio-exominer

[![Gem Version](https://badge.fury.io/rb/bio-exominer.svg)](https://badge.fury.io/rb/bio-exominer)
![Year](https://img.shields.io/static/v1?label=date&message=2014-03-01&color=blue)
![Code Size](https://tokei.rs/b1/github/pjotrp/bioruby-exominer)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-exominer)

Parse publications for gene names in a fuzzy fashion

* URL: [http://github.com/pjotrp/bioruby-exominer](http://github.com/pjotrp/bioruby-exominer)
* Authors: Pjotr Prins

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-exominer|
|downloads|4269|
|version|0.0.3|
|version_created_at|2014-03-01T11:36:57.822Z|
|version_downloads|4269|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|bb144499e6843d15879f6d54e2f07d4bfdb6ef32c9366ec476feaecb37316878|
|spec_sha|cfe736e43f2f93a1c1a308cf83b68f6ec6240df3de8ffc104f0d21e5cad99a1f|
|project_uri|https://rubygems.org/gems/bio-exominer|
|gem_uri|https://rubygems.org/gems/bio-exominer-0.0.3.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-exominer/0.0.3|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 0"}, {"name"=>"cucumber", "requirements"=>">= 0"}, {"name"=>"jeweler", "requirements"=>"~> 2.0.0"}, {"name"=>"minitest", "requirements"=>"~> 5.0.7"}, {"name"=>"rdoc", "requirements"=>">= 0"}, {"name"=>"rspec", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"msgpack", "requirements"=>">= 0"}]}|

</details>

## bio-faidx

[![Gem Version](https://badge.fury.io/rb/bio-faidx.svg)](https://badge.fury.io/rb/bio-faidx)
![Year](https://img.shields.io/static/v1?label=date&message=2014-10-07&color=blue)
![Code Size](https://tokei.rs/b1/github/homonecloco/bioruby-faidx)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-faidx)

ools to work with fasta files, indexed with samtools. The initial releases depend on bio-samtools, but that will change

* URL: [http://github.com/homonecloco/bioruby-faidx](http://github.com/homonecloco/bioruby-faidx)
* Authors: homonecloco

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-faidx|
|downloads|4107|
|version|0.0.1|
|version_created_at|2014-10-07T14:56:44.542Z|
|version_downloads|4107|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|047dfca5dad4b2caa4f74963bb8e43188f9f0f84a6d38df2ccb78ab48a6455ba|
|spec_sha|2b15ed04c54aa9ca9acdb2d258f17e59b1719fef20169c535ab9cd6cefd402c9|
|project_uri|https://rubygems.org/gems/bio-faidx|
|gem_uri|https://rubygems.org/gems/bio-faidx-0.0.1.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-faidx/0.0.1|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>"~> 2.0.1"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"simplecov", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-genomic-interval

[![Gem Version](https://badge.fury.io/rb/bio-genomic-interval.svg)](https://badge.fury.io/rb/bio-genomic-interval)
![Year](https://img.shields.io/static/v1?label=date&message=2011-04-20&color=blue)
![Code Size](https://tokei.rs/b1/github/misshie/bioruby-genomic-interval)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-genomic-interval)

a BioRuby plugin: handling genomic intervals,such as "chr1:123-456", and overlap state between two intervals 

* URL: [http://github.com/misshie/bioruby-genomic-interval](http://github.com/misshie/bioruby-genomic-interval)
* Authors: Hiroyuki Mishima

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-genomic-interval|
|downloads|16252|
|version|0.1.2|
|version_created_at|2011-04-20T06:56:14.307Z|
|version_downloads|7697|
|platform|ruby|
|metadata|{"wiki_uri"=>"http://bioruby.open-bio.org/wiki/Plugins", "homepage_uri"=>"http://github.com/misshie/bioruby-genomic-interval", "bug_tracker_uri"=>"", "source_code_uri"=>"https://github.com/misshie/bioruby-genomic-interval", "mailing_list_uri"=>"http://lists.open-bio.org/mailman/listinfo/bioruby", "documentation_uri"=>"https://github.com/misshie/bioruby-genomic-interval"}|
|yanked|false|
|sha|deb647f1e0fc960f3f417ee8f70d33a60ee033de0d14d0c34c700afc17392dc4|
|spec_sha|823554acbfd8793d038d1b8790711f20bcd282503c078afe4fbe6d388562272e|
|project_uri|https://rubygems.org/gems/bio-genomic-interval|
|gem_uri|https://rubygems.org/gems/bio-genomic-interval-0.1.2.gem|
|wiki_uri|http://bioruby.open-bio.org/wiki/Plugins|
|documentation_uri|https://github.com/misshie/bioruby-genomic-interval|
|mailing_list_uri|http://lists.open-bio.org/mailman/listinfo/bioruby|
|source_code_uri|https://github.com/misshie/bioruby-genomic-interval|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.5.2"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"rspec", "requirements"=>"~> 2.3.0"}], "runtime"=>[]}|

</details>

## bio-picard

[![Gem Version](https://badge.fury.io/rb/bio-picard.svg)](https://badge.fury.io/rb/bio-picard)
![Year](https://img.shields.io/static/v1?label=date&message=2015-05-21&color=blue)
![Code Size]()
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-picard)

API of Picard tool

* URL: [http://rubygems.org/gems/bio-picard](http://rubygems.org/gems/bio-picard)
* Authors: Natapol Pornputtapong

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-picard|
|downloads|3936|
|version|1.131.5|
|version_created_at|2015-05-21T05:19:08.303Z|
|version_downloads|3760|
|platform|ruby|
|licenses|["GPLv3"]|
|metadata|{"allowed_push_host"=>"TODO: Set to 'http://mygemserver.com'"}|
|yanked|false|
|sha|5c773ba5829bd0db9ebd15b3b5494d38c4e538a76bce62058410fc2523cc0d27|
|spec_sha|74cec612271617e2108a94781e9a7a1d36f76ad87c1d027b5eee73c669bf2bb7|
|project_uri|https://rubygems.org/gems/bio-picard|
|gem_uri|https://rubygems.org/gems/bio-picard-1.131.5.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-picard/1.131.5|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.9"}, {"name"=>"rake", "requirements"=>"~> 10.0"}], "runtime"=>[]}|

</details>

## bio-kallisto

[![Gem Version](https://badge.fury.io/rb/bio-kallisto.svg)](https://badge.fury.io/rb/bio-kallisto)
![Year](https://img.shields.io/static/v1?label=date&message=2015-10-05&color=blue)
![Code Size](https://tokei.rs/b1/github/homonecloco/bioruby-kallisto)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-kallisto)

Basic wrapper for the Kallistio Mapper

* URL: [http://github.com/homonecloco/bioruby-kallisto](http://github.com/homonecloco/bioruby-kallisto)
* Authors: homonecloco

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-kallisto|
|downloads|3755|
|version|0.1.0|
|version_created_at|2015-10-05T21:50:10.764Z|
|version_downloads|3755|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|57f668128b29f38ff47e976776c6edb544c0095482adcf7aec9fbd7c41a52b6c|
|spec_sha|41595726fb8f9e0d5ade26ee1b4615c19e40bcb8e399122084d155c8325198fb|
|project_uri|https://rubygems.org/gems/bio-kallisto|
|gem_uri|https://rubygems.org/gems/bio-kallisto-0.1.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-kallisto/0.1.0|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>">= 2.0.1"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"simplecov", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"systemu", "requirements"=>">= 2.5.2"}]}|

</details>

## bio-statsample-timeseries

[![Gem Version](https://badge.fury.io/rb/bio-statsample-timeseries.svg)](https://badge.fury.io/rb/bio-statsample-timeseries)
![Year](https://img.shields.io/static/v1?label=date&message=2013-09-23&color=blue)
![Code Size](https://tokei.rs/b1/github/AnkurGel/bioruby-statsample-timeseries)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-statsample-timeseries)

Statsample-timeseries is an extension to Statsample. It incorporates helpful timeseries functions and modules like ARMA, ARIMA, acf, pacf, lags etc.

* URL: [http://github.com/AnkurGel/bioruby-statsample-timeseries](http://github.com/AnkurGel/bioruby-statsample-timeseries)
* Authors: Ankur Goel, Claudio Bustos

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-statsample-timeseries|
|downloads|14869|
|version|0.2.1|
|version_created_at|2013-09-23T06:06:28.580Z|
|version_downloads|4613|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|a26a9af1d7bb775f98800ab688e26a09972428604fee63d11dcdec8157b8e3ea|
|spec_sha|4ad33257e79a13f79c980480d83b7dad39c67575617b2280265d95a2a6c9fad2|
|project_uri|https://rubygems.org/gems/bio-statsample-timeseries|
|gem_uri|https://rubygems.org/gems/bio-statsample-timeseries-0.2.1.gem|
|wiki_uri||
|documentation_uri|https://www.rubydoc.info/gems/bio-statsample-timeseries/0.2.1|
|mailing_list_uri||
|source_code_uri|https://github.com/AnkurGel/bioruby-statsample-timeseries|
|bug_tracker_uri|https://github.com/AnkurGel/bioruby-statsample-timeseries/issues|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>"~> 1.3.5"}, {"name"=>"cucumber", "requirements"=>">= 0"}, {"name"=>"gsl", "requirements"=>">= 0"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.4"}, {"name"=>"minitest", "requirements"=>"~> 4.7.5"}, {"name"=>"mocha", "requirements"=>"~> 0.14.0"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"activesupport", "requirements"=>"= 3.2.10"}, {"name"=>"statsample", "requirements"=>"= 1.2.0"}]}|

</details>

## ngoto-bio

[![Gem Version](https://badge.fury.io/rb/ngoto-bio.svg)](https://badge.fury.io/rb/ngoto-bio)
![Year](https://img.shields.io/static/v1?label=date&message=2014-08-10&color=blue)
![Code Size]()
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/ngoto-bio)

BioRuby is a library for bioinformatics (biology + information science).

* URL: [http://bioruby.org/](http://bioruby.org/)
* Authors: BioRuby project

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|ngoto-bio|
|downloads|13233|
|version|1.3.1.5000|
|version_created_at|2014-08-10T23:36:57.916Z|
|version_downloads|3446|
|platform|ruby|
|licenses|[]|
|metadata|{}|
|yanked|false|
|sha|a13d7d3b16b794f65f9c397d34a4e7ac13b9de613d9eccc0781639ec179470b1|
|spec_sha|36ce770df90985daccb9680271eb35ffa0c31047d22f81c409a9d6829dea63b6|
|project_uri|https://rubygems.org/gems/ngoto-bio|
|gem_uri|https://rubygems.org/gems/ngoto-bio-1.3.1.5000.gem|
|documentation_uri|https://www.rubydoc.info/gems/ngoto-bio/1.3.1.5000|
|dependencies|{"development"=>[], "runtime"=>[]}|

</details>

## bio-jaspar

[![Gem Version](https://badge.fury.io/rb/bio-jaspar.svg)](https://badge.fury.io/rb/bio-jaspar)
![Year](https://img.shields.io/static/v1?label=date&message=2015-09-30&color=blue)
![Code Size](https://tokei.rs/b1/github/wassermanlab/jaspar-bioruby)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-jaspar)

Basic tools for parsing, searching, and comparing JASPAR motifs; Based on Bio.motifs module in Biopython

* URL: [http://github.com/wassermanlab/jaspar-bioruby](http://github.com/wassermanlab/jaspar-bioruby)
* Authors: Jessica Lee, Wasserman Lab

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-jaspar|
|downloads|3669|
|version|0.1.0|
|version_created_at|2015-09-30T22:07:21.452Z|
|version_downloads|3669|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|5442d3b50b93f4ed198305c5a8c2c45dff6d59af8abe581e49d1fc9cf03fe3e9|
|spec_sha|7388297268799487cc9c546215998e0410aa39f7f7b802b8136d272ac74e38cb|
|project_uri|https://rubygems.org/gems/bio-jaspar|
|gem_uri|https://rubygems.org/gems/bio-jaspar-0.1.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-jaspar/0.1.0|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>"~> 2.0.1"}, {"name"=>"rake", "requirements"=>"~> 0.9.3"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"test-unit", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"mysql2", "requirements"=>"~> 0.3.19"}]}|

</details>

## bio-conduit

[![Gem Version](https://badge.fury.io/rb/bio-conduit.svg)](https://badge.fury.io/rb/bio-conduit)
![Year](https://img.shields.io/static/v1?label=date&message=2015-09-04&color=blue)
![Code Size]()
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-conduit)

A pipeline creator for bioruby

* URL: [http://rubygems.org/gems/bio-conduit](http://rubygems.org/gems/bio-conduit)
* Authors: Natapol Pornputtapong

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-conduit|
|downloads|3646|
|version|1.0.0|
|version_created_at|2015-09-04T21:01:27.480Z|
|version_downloads|3646|
|platform|ruby|
|licenses|["GPL"]|
|metadata|{}|
|yanked|false|
|sha|9e5abceceb9bb9c33efd64bca2be4c2a4211e8573092aa3ad37793d7878d4829|
|spec_sha|7a9336d394d1cb701baeafb9eb681d5a5a85463cbbb0b33feea4a3a304021f81|
|project_uri|https://rubygems.org/gems/bio-conduit|
|gem_uri|https://rubygems.org/gems/bio-conduit-1.0.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-conduit/1.0.0|
|dependencies|{"development"=>[], "runtime"=>[]}|

</details>

## bio-pileup_iterator

[![Gem Version](https://badge.fury.io/rb/bio-pileup_iterator.svg)](https://badge.fury.io/rb/bio-pileup_iterator)
![Year](https://img.shields.io/static/v1?label=date&message=2012-11-19&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-pileup_iterator)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-pileup_iterator)

Iterate through a samtools pileup file

* URL: [http://github.com/wwood/bioruby-pileup_iterator](http://github.com/wwood/bioruby-pileup_iterator)
* Authors: Ben J. Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-pileup_iterator|
|downloads|13895|
|version|0.0.4|
|version_created_at|2012-11-19T02:56:51.131Z|
|version_downloads|5189|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{"homepage_uri"=>"http://github.com/wwood/bioruby-pileup_iterator"}|
|yanked|false|
|sha|5bbaa3a1aa4d603e359ca318e0556d88d00d1219d3ce1eb26edc4ca8ee5251fe|
|spec_sha|261dbef3506beb7ebb3cc8077b6a95310a5cedfdb821a40b72035138b5f1a2d7|
|project_uri|https://rubygems.org/gems/bio-pileup_iterator|
|gem_uri|https://rubygems.org/gems/bio-pileup_iterator-0.0.4.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.3"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bio-logger", "requirements"=>">= 1.0.0"}, {"name"=>"bio-samtools", "requirements"=>">= 0.5.3"}]}|

</details>

## bio-plates

[![Gem Version](https://badge.fury.io/rb/bio-plates.svg)](https://badge.fury.io/rb/bio-plates)
![Year](https://img.shields.io/static/v1?label=date&message=2016-01-29&color=blue)
![Code Size](https://tokei.rs/b1/github/stveep/bioruby-plates)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-plates)

Methods for handling multiwell plate annotations, includes ranges and quadrants

* URL: [http://github.com/stveep/bioruby-plates](http://github.com/stveep/bioruby-plates)
* Authors: Steve Pettitt

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-plates|
|downloads|3456|
|version|0.2.0|
|version_created_at|2016-01-29T11:59:43.877Z|
|version_downloads|3456|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|9b57b1fabcbea3877305393dce7ba91215460e8b96fe42f2448d28eae4d4106f|
|spec_sha|9153c28bdfca009ba77f536256f569d1d471f827b618bf2ddc61287ea9cf8440|
|project_uri|https://rubygems.org/gems/bio-plates|
|gem_uri|https://rubygems.org/gems/bio-plates-0.2.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-plates/0.2.0|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 0"}, {"name"=>"fakefs", "requirements"=>"~> 0.6"}, {"name"=>"jeweler", "requirements"=>"~> 2.0"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rspec", "requirements"=>"~> 3.3"}, {"name"=>"simplecov", "requirements"=>"~> 0.10"}], "runtime"=>[{"name"=>"bio", "requirements"=>"~> 1.4"}, {"name"=>"thor", "requirements"=>">= 0"}]}|

</details>

## bio-cgranges

[![Gem Version](https://badge.fury.io/rb/bio-cgranges.svg)](https://badge.fury.io/rb/bio-cgranges)
![Year](https://img.shields.io/static/v1?label=date&message=2024-09-19&color=blue)
![Code Size](https://tokei.rs/b1/github/kojix2/bio-cgranges)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-cgranges)

Genomic interval overlap queries

* URL: [https://github.com/kojix2/bio-cgranges](https://github.com/kojix2/bio-cgranges)
* Authors: kojix2

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-cgranges|
|downloads|3354|
|version|0.0.4|
|version_created_at|2024-09-19T06:13:06.640Z|
|version_downloads|254|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|1de9485f098c6acc2ee3908f6bd66061362cc3bcd5991791bc6b852d9f122279|
|spec_sha|e3ba579277b5425bb8137201adb726e42f052e10606c3117cae0a6b13269adcb|
|project_uri|https://rubygems.org/gems/bio-cgranges|
|gem_uri|https://rubygems.org/gems/bio-cgranges-0.0.4.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-cgranges/0.0.4|
|dependencies|{"development"=>[], "runtime"=>[]}|

</details>

## bio-gff3-pltools

[![Gem Version](https://badge.fury.io/rb/bio-gff3-pltools.svg)](https://badge.fury.io/rb/bio-gff3-pltools)
![Year](https://img.shields.io/static/v1?label=date&message=2012-08-07&color=blue)
![Code Size](https://tokei.rs/b1/github/gff3-pltools/)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-gff3-pltools)

Ruby wrapper for the gff3-pltools.

* URL: [http://mamarjan.github.com/gff3-pltools/](http://mamarjan.github.com/gff3-pltools/)
* Authors: Marjan Povolni

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-gff3-pltools|
|downloads|13081|
|version|0.3.0|
|version_created_at|2012-08-07T13:09:30.199Z|
|version_downloads|5094|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://mamarjan.github.com/gff3-pltools/"}|
|yanked|false|
|sha|06e6a16047a54c6c73c2344e49736716460ba93bd0b0aa5fbf5947b8ac070a4a|
|spec_sha|26538152dce220f04d80d2e9434f28666f56cdd5734262391d515533dac4cfe8|
|project_uri|https://rubygems.org/gems/bio-gff3-pltools|
|gem_uri|https://rubygems.org/gems/bio-gff3-pltools-0.3.0.gem|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.1.3"}, {"name"=>"cucumber", "requirements"=>">= 0"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.3"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"redcarpet", "requirements"=>">= 0"}, {"name"=>"rspec", "requirements"=>"~> 2.8.0"}, {"name"=>"yard", "requirements"=>"~> 0.7"}], "runtime"=>[]}|

</details>

## bio-ucsc-util

[![Gem Version](https://badge.fury.io/rb/bio-ucsc-util.svg)](https://badge.fury.io/rb/bio-ucsc-util)
![Year](https://img.shields.io/static/v1?label=date&message=2012-10-02&color=blue)
![Code Size](https://tokei.rs/b1/github/throwern/bio-ucsc-util)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-ucsc-util)

Ruby FFI binding of the ucsc kent src. Implemented utilities include: bigWigInfo, bigWigSummary and wigToBigWig

* URL: [http://github.com/throwern/bio-ucsc-util](http://github.com/throwern/bio-ucsc-util)
* Authors: throwern

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-ucsc-util|
|downloads|12809|
|version|0.1.2|
|version_created_at|2012-10-02T16:33:16.336Z|
|version_downloads|5033|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/throwern/bio-ucsc-util"}|
|yanked|false|
|sha|30c8ef8dec1325488c8138d313591746338175c348bf42d0262ac066bd849045|
|spec_sha|5ea018f94c6cfdb7fc8bcf7c1f8a317046c96a625578680b3c4be9104626b663|
|project_uri|https://rubygems.org/gems/bio-ucsc-util|
|gem_uri|https://rubygems.org/gems/bio-ucsc-util-0.1.2.gem|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.3"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}], "runtime"=>[{"name"=>"ffi", "requirements"=>">= 0"}]}|

</details>

## bio-biosql

[![Gem Version](https://badge.fury.io/rb/bio-biosql.svg)](https://badge.fury.io/rb/bio-biosql)
![Year](https://img.shields.io/static/v1?label=date&message=2016-06-17&color=blue)
![Code Size](https://tokei.rs/b1/github/bioruby/bioruby-biosql)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-biosql)

Provides BioSQL support for BioRuby.

* URL: [http://github.com/bioruby/bioruby-biosql](http://github.com/bioruby/bioruby-biosql)
* Authors: BioRuby project

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-biosql|
|downloads|3103|
|version|0.0.0|
|version_created_at|2016-06-17T03:39:50.984Z|
|version_downloads|3103|
|platform|ruby|
|licenses|["Ruby"]|
|metadata|{}|
|yanked|false|
|sha|30a3449afc8b2d7d14c1b6f35d54389bc90224b62a20858a230e2e4e39b86cef|
|spec_sha|2ee0c8386725adaf1e055961669145f269a24e415b0d21dd9c8f39bd1bbb42b6|
|project_uri|https://rubygems.org/gems/bio-biosql|
|gem_uri|https://rubygems.org/gems/bio-biosql-0.0.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-biosql/0.0.0|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.7"}, {"name"=>"rake", "requirements"=>"~> 10.0"}, {"name"=>"rdoc", "requirements"=>"~> 4"}, {"name"=>"test-unit", "requirements"=>"~> 3"}], "runtime"=>[{"name"=>"activerecord", "requirements"=>"~> 3.0.10"}, {"name"=>"bio", "requirements"=>"~> 1.5.0"}, {"name"=>"composite_primary_keys", "requirements"=>"~> 3.1.10"}]}|

</details>

## bio-iqtree

[![Gem Version](https://badge.fury.io/rb/bio-iqtree.svg)](https://badge.fury.io/rb/bio-iqtree)
![Year](https://img.shields.io/static/v1?label=date&message=2019-08-20&color=blue)
![Code Size]()
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-iqtree)

Phylogenetic analysis using IQ-Tree via BioRuby

* URL: []()
* Authors: evolbeginner

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-iqtree|
|downloads|2936|
|version|0.1.2|
|version_created_at|2019-08-20T02:13:44.060Z|
|version_downloads|2819|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|66f0353ebf7f403aed1105d9cf7722e1a3b0955825c0a710295162fc37d23397|
|spec_sha|8beec44f91971d7bb0d3ab8928e5436ba044ef6c8bc7f0ab97c082ee67eda7e8|
|project_uri|https://rubygems.org/gems/bio-iqtree|
|gem_uri|https://rubygems.org/gems/bio-iqtree-0.1.2.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-iqtree/0.1.2|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>"~> 1.5"}, {"name"=>"bundler", "requirements"=>"~> 2.0"}, {"name"=>"rake", "requirements"=>"~> 10.0"}, {"name"=>"rspec", "requirements"=>"~> 3.0"}], "runtime"=>[]}|

</details>

## muscle_bio

[![Gem Version](https://badge.fury.io/rb/muscle_bio.svg)](https://badge.fury.io/rb/muscle_bio)
![Year](https://img.shields.io/static/v1?label=date&message=2022-02-04&color=blue)
![Code Size]()
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/muscle_bio)

Multiple Sequence Alignment for Ruby scripts.  Muscle found at https://www.ebi.ac.uk/Tools/msa/muscle/.  For instructions on how to use, see https://github.com/clarkmu/muscle_bio

* URL: [http://swanstrom.web.unc.edu/](http://swanstrom.web.unc.edu/)
* Authors: Michael Clark

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|muscle_bio|
|downloads|9775|
|version|0.5.0|
|version_created_at|2022-02-04T20:00:54.375Z|
|version_downloads|1591|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|99b6cdeb4e6a11b83f6c74f0dde6f451eafc5a8d40b3292206b7b718656e5162|
|spec_sha|f32f7c1dc81277518572843d22748c162ac1f368e632b27b304894097a640f1b|
|project_uri|https://rubygems.org/gems/muscle_bio|
|gem_uri|https://rubygems.org/gems/muscle_bio-0.5.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/muscle_bio/0.5.0|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 2.0"}, {"name"=>"rake", "requirements"=>"~> 13.0"}, {"name"=>"rspec", "requirements"=>"~> 3.0"}], "runtime"=>[]}|

</details>

## bio-biostars-analytics

[![Gem Version](https://badge.fury.io/rb/bio-biostars-analytics.svg)](https://badge.fury.io/rb/bio-biostars-analytics)
![Year](https://img.shields.io/static/v1?label=date&message=2014-04-08&color=blue)
![Code Size](https://tokei.rs/b1/github/joejimbo/bioruby-biostars-analytics)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-biostars-analytics)

Ruby script for data-mining biostars.org using web-crawling techniques as well as utilizing the Biostars RESTful API. Statistical analysis requires R (http://www.r-project.org).

* URL: [http://github.com/joejimbo/bioruby-biostars-analytics](http://github.com/joejimbo/bioruby-biostars-analytics)
* Authors: Joachim Baran

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-biostars-analytics|
|downloads|10495|
|version|0.2.0|
|version_created_at|2014-04-08T03:47:37.252Z|
|version_downloads|4257|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|29221bef2457adf1ea672aafc072fee194ff56a2b23a1319d21ddf4b2afe9171|
|spec_sha|83288b3c2f30ddf4f6f78a776e51f4c9e4724557051f55619bd8adab13a29be4|
|project_uri|https://rubygems.org/gems/bio-biostars-analytics|
|gem_uri|https://rubygems.org/gems/bio-biostars-analytics-0.2.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-biostars-analytics/0.2.0|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"chronic", "requirements"=>"~> 0.10.2"}, {"name"=>"hpricot", "requirements"=>"~> 0.8.6"}, {"name"=>"jeweler", "requirements"=>"~> 2.0.1"}, {"name"=>"json", "requirements"=>"~> 1.8.0"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-kb-illumina

[![Gem Version](https://badge.fury.io/rb/bio-kb-illumina.svg)](https://badge.fury.io/rb/bio-kb-illumina)
![Year](https://img.shields.io/static/v1?label=date&message=2011-01-11&color=blue)
![Code Size](https://tokei.rs/b1/github/helios/bioruby-kb-illumina)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-kb-illumina)

Provide access to the IlluminaHumanAnnotations. Database comes form Bioconductor's package
     lumiHumanIDMapping

* URL: [http://github.com/helios/bioruby-kb-illumina](http://github.com/helios/bioruby-kb-illumina)
* Authors: Raoul J.P. Bonnal

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-kb-illumina|
|downloads|10314|
|version|0.1.0|
|version_created_at|2011-01-11T12:21:53.806Z|
|version_downloads|5694|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/helios/bioruby-kb-illumina"}|
|yanked|false|
|sha|443c9c2bc501c4de4e2cad7a1f2f4ff138ee3763d6857e37bffb7e69d4ad119a|
|spec_sha|05121c9e813800036c0677521da8972dff1da7ba8d9f3ab9edf7e758a1442762|
|project_uri|https://rubygems.org/gems/bio-kb-illumina|
|gem_uri|https://rubygems.org/gems/bio-kb-illumina-0.1.0.gem|
|dependencies|{"development"=>[{"name"=>"activerecord", "requirements"=>">= 3.0.3"}, {"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.5.1"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-qtlHD

[![Gem Version](https://badge.fury.io/rb/bio-qtlHD.svg)](https://badge.fury.io/rb/bio-qtlHD)
![Year](https://img.shields.io/static/v1?label=date&message=2011-11-16&color=blue)
![Code Size](https://tokei.rs/b1/github/pjotrp/bioruby-qtlHD)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-qtlHD)

Ruby tools and scripts for QTL mapping, part of the qtlHD project

* URL: [http://github.com/pjotrp/bioruby-qtlHD](http://github.com/pjotrp/bioruby-qtlHD)
* Authors: Pjotr Prins, Danny Arends, Karl Broman

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-qtlHD|
|downloads|2695|
|version|0.0.1.alpha.1|
|version_created_at|2011-11-16T12:48:33.862Z|
|version_downloads|2695|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/pjotrp/bioruby-qtlHD"}|
|yanked|false|
|sha|13c26344c063e8def83b41a6cfc3bd13cc2afad110cb0fc387eed303adda1a13|
|spec_sha|40373e310f45945f9766af3e934866a47f616682977c16f8f0bd129a496bb864|
|project_uri|https://rubygems.org/gems/bio-qtlHD|
|gem_uri|https://rubygems.org/gems/bio-qtlHD-0.0.1.alpha.1.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.4"}, {"name"=>"rcov", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-primer3

[![Gem Version](https://badge.fury.io/rb/bio-primer3.svg)](https://badge.fury.io/rb/bio-primer3)
![Year](https://img.shields.io/static/v1?label=date&message=2014-03-18&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-primer3)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-primer3)

Primer3-related Ruby code - wrappers, Boulder IO, etc.

* URL: [http://github.com/wwood/bioruby-primer3](http://github.com/wwood/bioruby-primer3)
* Authors: Ben J Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-primer3|
|downloads|2559|
|version|0.0.1.pre|
|version_created_at|2014-03-18T05:24:43.657Z|
|version_downloads|2559|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|c53cc716c84e267660d55991cf3254012900f2f8e6431a165a86f986c6d89dae|
|spec_sha|92b67289f0978a1506878cce055fb02350e44981db0d30c3a069fecc0fa128ab|
|project_uri|https://rubygems.org/gems/bio-primer3|
|gem_uri|https://rubygems.org/gems/bio-primer3-0.0.1.pre.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-primer3/0.0.1.pre|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.3"}, {"name"=>"jeweler", "requirements"=>"~> 2.0"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rspec", "requirements"=>"~> 2.8"}, {"name"=>"yard", "requirements"=>"~> 0.7"}], "runtime"=>[{"name"=>"bio", "requirements"=>"~> 1.4"}, {"name"=>"bio-commandeer", "requirements"=>"~> 0.1"}, {"name"=>"bio-logger", "requirements"=>"~> 1.0"}]}|

</details>

## bio-cnls_screenscraper

[![Gem Version](https://badge.fury.io/rb/bio-cnls_screenscraper.svg)](https://badge.fury.io/rb/bio-cnls_screenscraper)
![Year](https://img.shields.io/static/v1?label=date&message=2012-05-06&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-cnls_screenscraper)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-cnls_screenscraper)

Programmatic interface to the cNLS nuclear localisation signal prediction software

* URL: [http://github.com/wwood/bioruby-cnls_screenscraper](http://github.com/wwood/bioruby-cnls_screenscraper)
* Authors: Ben J. Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-cnls_screenscraper|
|downloads|9553|
|version|0.3.0|
|version_created_at|2012-05-06T03:59:07.038Z|
|version_downloads|5214|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/wwood/bioruby-cnls_screenscraper"}|
|yanked|false|
|sha|89bad4f77e94e30da20f430c879bdedddbfcc5cb238c3c3b726773dcaee2fd88|
|spec_sha|3546afe7352be96b00b34a337b0ee3eb8bd4247bd097213180ed9eac1dea9309|
|project_uri|https://rubygems.org/gems/bio-cnls_screenscraper|
|gem_uri|https://rubygems.org/gems/bio-cnls_screenscraper-0.3.0.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.3"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-bigwig

[![Gem Version](https://badge.fury.io/rb/bio-bigwig.svg)](https://badge.fury.io/rb/bio-bigwig)
![Year](https://img.shields.io/static/v1?label=date&message=2024-09-30&color=blue)
![Code Size](https://tokei.rs/b1/github/kojix2/bw-ruby)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-bigwig)

This is a Ruby binding for libBigWig (https://github.com/dpryan79/libBigWig), which provides high-speed access to bigWig or bigBed files.

* URL: [https://github.com/kojix2/bw-ruby](https://github.com/kojix2/bw-ruby)
* Authors: kojix2

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-bigwig|
|downloads|2482|
|version|0.0.6|
|version_created_at|2024-09-30T10:38:42.570Z|
|version_downloads|228|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|7caf66cc773f5937780acf2b313ba83612075fee8ca2fad0d2f0641030293ece|
|spec_sha|a3699aad1947f395c6bf1e8952795262354da1116bc4a92209d939926bff9cab|
|project_uri|https://rubygems.org/gems/bio-bigwig|
|gem_uri|https://rubygems.org/gems/bio-bigwig-0.0.6.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-bigwig/0.0.6|
|dependencies|{"development"=>[], "runtime"=>[]}|

</details>

## bio-hmmer_model

[![Gem Version](https://badge.fury.io/rb/bio-hmmer_model.svg)](https://badge.fury.io/rb/bio-hmmer_model)
![Year](https://img.shields.io/static/v1?label=date&message=2012-05-25&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-hmmer_model)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-hmmer_model)

Parse PFAM HMM definition files so that the models can be accessible programmatically

* URL: [http://github.com/wwood/bioruby-hmmer_model](http://github.com/wwood/bioruby-hmmer_model)
* Authors: Ben J. Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-hmmer_model|
|downloads|9124|
|version|0.0.2|
|version_created_at|2012-05-25T00:11:35.380Z|
|version_downloads|5105|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/wwood/bioruby-hmmer_model"}|
|yanked|false|
|sha|860549f3db3717ade58766475b1429f348b7c2b3b097f786151b67d1cbe56b7e|
|spec_sha|e261176933db9682d096b561e1ca017fd9c8f47aa75da9285cabdd1b807fd52f|
|project_uri|https://rubygems.org/gems/bio-hmmer_model|
|gem_uri|https://rubygems.org/gems/bio-hmmer_model-0.0.2.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>"> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.3"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-hmmer3_report

[![Gem Version](https://badge.fury.io/rb/bio-hmmer3_report.svg)](https://badge.fury.io/rb/bio-hmmer3_report)
![Year](https://img.shields.io/static/v1?label=date&message=2015-08-06&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-hmmer3_report)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-hmmer3_report)

Enables programmatic parsing of HMMER version 3 reports

* URL: [http://github.com/wwood/bioruby-hmmer3_report](http://github.com/wwood/bioruby-hmmer3_report)
* Authors: Christian Zmasek, Ben J Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-hmmer3_report|
|downloads|8284|
|version|0.1.0|
|version_created_at|2015-08-06T01:25:37.875Z|
|version_downloads|4093|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|7fe415b44ed45eaa899a6b9a0caf70f6e3fe20170b281e20e23892838068fd9e|
|spec_sha|61a75e4ba99ba14c2459c05b8811001cd2100dc25941b17d0ef30e30a7378ac2|
|project_uri|https://rubygems.org/gems/bio-hmmer3_report|
|gem_uri|https://rubygems.org/gems/bio-hmmer3_report-0.1.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-hmmer3_report/0.1.0|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2, ~> 1.4"}, {"name"=>"bundler", "requirements"=>">= 1.6.2, ~> 1.6"}, {"name"=>"jeweler", "requirements"=>">= 1.8.3, ~> 1.8"}, {"name"=>"minitest", "requirements"=>">= 4.7.5, ~> 4.7"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 3.5.0, ~> 3.5"}], "runtime"=>[{"name"=>"bio-logger", "requirements"=>"~> 1.0"}]}|

</details>

## bio-polymarker_db_batch

[![Gem Version](https://badge.fury.io/rb/bio-polymarker_db_batch.svg)](https://badge.fury.io/rb/bio-polymarker_db_batch)
![Year](https://img.shields.io/static/v1?label=date&message=2015-03-05&color=blue)
![Code Size](https://tokei.rs/b1/github/homonecloco/bioruby-polymarker_db_batch)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-polymarker_db_batch)

Server for polymarker. To be used with cron

* URL: [http://github.com/homonecloco/bioruby-polymarker_db_batch](http://github.com/homonecloco/bioruby-polymarker_db_batch)
* Authors: Ricardo H.  Ramirez-Gonzalez

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-polymarker_db_batch|
|downloads|27588|
|version|0.5.1|
|version_created_at|2015-03-05T21:21:12.632Z|
|version_downloads|3736|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|fa66abdd7eb0b559b2576a6bc4abd4165f4b3f28c0cea1ab393aef854ef66dd6|
|spec_sha|4dd51e02b355dedbb8c734add0f914035837c4c1b37b969aa69ecb400f881c15|
|project_uri|https://rubygems.org/gems/bio-polymarker_db_batch|
|gem_uri|https://rubygems.org/gems/bio-polymarker_db_batch-0.5.1.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-polymarker_db_batch/0.5.1|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>"~> 2"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"simplecov", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio-polyploid-tools", "requirements"=>"= 0.5.1"}, {"name"=>"mysql", "requirements"=>">= 0"}]}|

</details>

## bio-ensembl-rest

[![Gem Version](https://badge.fury.io/rb/bio-ensembl-rest.svg)](https://badge.fury.io/rb/bio-ensembl-rest)
![Year](https://img.shields.io/static/v1?label=date&message=2013-09-27&color=blue)
![Code Size](https://tokei.rs/b1/github/ALTree/bio-ensembl-rest)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-ensembl-rest)

A Ruby library for the RESTful Ensembl API.

* URL: [http://github.com/ALTree/bio-ensembl-rest](http://github.com/ALTree/bio-ensembl-rest)
* Authors: Alberto Donizetti

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-ensembl-rest|
|downloads|8006|
|version|0.2.0|
|version_created_at|2013-09-27T16:10:22.754Z|
|version_downloads|4543|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|8425c66120938186c35c5fbb9a57dec4081eb9e7d1393a914faab9056ae573d9|
|spec_sha|b48e9146688a0db67e8569704a2050e1af1213182c720a7a39bb37a47e5e4f38|
|project_uri|https://rubygems.org/gems/bio-ensembl-rest|
|gem_uri|https://rubygems.org/gems/bio-ensembl-rest-0.2.0.gem|
|wiki_uri|https://github.com/ALTree/bio-ensembl-rest/wiki|
|documentation_uri|https://github.com/ALTree/bio-ensembl-rest/wiki|
|mailing_list_uri||
|source_code_uri|https://github.com/ALTree/bio-ensembl-rest|
|bug_tracker_uri|https://github.com/ALTree/bio-ensembl-rest/issues|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.3.5"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.4"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.3"}, {"name"=>"libxml-ruby", "requirements"=>">= 2"}]}|

</details>

## bio-statsample-glm

[![Gem Version](https://badge.fury.io/rb/bio-statsample-glm.svg)](https://badge.fury.io/rb/bio-statsample-glm)
![Year](https://img.shields.io/static/v1?label=date&message=2013-09-23&color=blue)
![Code Size](https://tokei.rs/b1/github/AnkurGel/bioruby-statsample-glm)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-statsample-glm)

Statsample-GLM is an extension to Statsample, an advance statistics suite in Ruby. This gem includes modules for Regression techniques such as Poisson Regression, Logistic Regression and Exponential Regression

* URL: [http://github.com/AnkurGel/bioruby-statsample-glm](http://github.com/AnkurGel/bioruby-statsample-glm)
* Authors: Ankur Goel

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-statsample-glm|
|downloads|7927|
|version|0.1.1|
|version_created_at|2013-09-23T12:15:20.766Z|
|version_downloads|4552|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|aebf30edd9bf6b0da842b4b10e00cc69a8aaf4f3f4265d0f42f8d734ec674fa0|
|spec_sha|bc0243eec9cc9ec64760e06fb9ea5a6491efd5e1508929e90f9a8ac7c11d8b1f|
|project_uri|https://rubygems.org/gems/bio-statsample-glm|
|gem_uri|https://rubygems.org/gems/bio-statsample-glm-0.1.1.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-statsample-glm/0.1.1|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>"~> 1.3.5"}, {"name"=>"cucumber", "requirements"=>">= 0"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.4"}, {"name"=>"minitest", "requirements"=>"~> 4.7.5"}, {"name"=>"mocha", "requirements"=>"~> 0.14.0"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"activesupport", "requirements"=>"= 3.2.10"}, {"name"=>"statsample", "requirements"=>">= 1.2.0"}]}|

</details>

## bio-samtools-server

[![Gem Version](https://badge.fury.io/rb/bio-samtools-server.svg)](https://badge.fury.io/rb/bio-samtools-server)
![Year](https://img.shields.io/static/v1?label=date&message=2014-05-30&color=blue)
![Code Size](https://tokei.rs/b1/github/homonecloco/bioruby-samtools-server)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-samtools-server)

A minimal web service on the top of sinatra to query bam files

* URL: [http://github.com/homonecloco/bioruby-samtools-server](http://github.com/homonecloco/bioruby-samtools-server)
* Authors: homonecloco

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-samtools-server|
|downloads|7387|
|version|0.1.2|
|version_created_at|2014-05-30T16:58:53.191Z|
|version_downloads|4236|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|c5a2414400b2cb8b959dcbed18ba9912b00346c543b68d83c16108a7b9872339|
|spec_sha|7c3f2b0286477c718eb6fac2080b867eee5f0cdd942dbd5fa48938011a5a0e27|
|project_uri|https://rubygems.org/gems/bio-samtools-server|
|gem_uri|https://rubygems.org/gems/bio-samtools-server-0.1.2.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-samtools-server/0.1.2|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>"~> 2"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"simplecov", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio-samtools", "requirements"=>">= 2.0.3"}, {"name"=>"sinatra", "requirements"=>">= 1.4.5"}]}|

</details>

## bio-restriction_enzyme

[![Gem Version](https://badge.fury.io/rb/bio-restriction_enzyme.svg)](https://badge.fury.io/rb/bio-restriction_enzyme)
![Year](https://img.shields.io/static/v1?label=date&message=2011-09-10&color=blue)
![Code Size](https://tokei.rs/b1/github/trevor/bioruby-restriction_enzyme)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-restriction_enzyme)

Digests DNA based on restriction enzyme cut patterns

* URL: [http://github.com/trevor/bioruby-restriction_enzyme](http://github.com/trevor/bioruby-restriction_enzyme)
* Authors: Trevor Wennblom

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-restriction_enzyme|
|downloads|5722|
|version|1.0.0|
|version_created_at|2011-09-10T04:45:36.977Z|
|version_downloads|5722|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/trevor/bioruby-restriction_enzyme"}|
|yanked|false|
|sha|f73f3f32b345ac41e143d1be86571f51b1a0b867c7f8a989bcb37d5aab563003|
|spec_sha|8578c9473a2e278d19193f6eac903f3b3df224291eaed6d48fc6dcee42305209|
|project_uri|https://rubygems.org/gems/bio-restriction_enzyme|
|gem_uri|https://rubygems.org/gems/bio-restriction_enzyme-1.0.0.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.4"}, {"name"=>"rcov", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio-data-restriction_enzyme", "requirements"=>">= 0"}]}|

</details>

## intermine-bio

[![Gem Version](https://badge.fury.io/rb/intermine-bio.svg)](https://badge.fury.io/rb/intermine-bio)
![Year](https://img.shields.io/static/v1?label=date&message=2011-08-02&color=blue)
![Code Size]()
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/intermine-bio)

= Biological Extensions to the InterMine Webservice Client Library 

This library is a set of extensions to the InterMine Webservices client, 
providing access for data in biological formats. It directly extends the 
InterMine classes, providing extra methods to the Query class.

== Example

Get all sequences for proteins on "h", "r", "eve", "bib" and "zen":

    require "rubygems"
    require "intermine/service"
    require "intermine/bio"

    s = Service.new("www.flymine.org/query")

    puts s.query("Gene").select("proteins").where(:symbol => %w{h r eve bib zen}).fasta

Process the locations of these genes one at a time:

    s.query.select("Gene").where(:symbol => %w{h r eve bib zen}).bed do |line|
      process line
    end

== Who is this for?

InterMine data warehouses are typically constructed to hold
Biological data, and as this library facilitates programmatic
access to these data, this install is primarily aimed at 
bioinformaticians. In particular, users of the following services
may find it especially useful:
 * FlyMine (http://www.flymine.org/query)
 * YeastMine (http://yeastmine.yeastgenome.org/yeastmine)
 * RatMine (http://ratmine.mcw.edu/ratmine)
 * modMine (http://intermine.modencode.org/release-23)
 * metabolicMine (http://www.metabolicmine.org/beta)

These extensions are aimed at bioinformaticians looking to integrate
these sources of data into other workflows.

For details on constructing queries, see the intermine documentation.

== Support

Support is available on our development mailing list: dev@intermine.org

== License

This code is Open Source under the LGPL. Source code for all InterMine code
can be checked out from svn://subversion.flymine.org/flymine


* URL: [http://www.intermine.org](http://www.intermine.org)
* Authors: Alex Kalderimis

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|intermine-bio|
|downloads|5105|
|version|0.98.1|
|version_created_at|2011-08-02T13:19:44.373Z|
|version_downloads|5105|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://www.intermine.org"}|
|yanked|false|
|sha|c9a215bd92a7f52d538e5e26a6726b9b6ae81a86cd3a498b8f147af47fb3bc33|
|spec_sha|a6d7d4cf5ab6f939d28f62d24e21396c7b98cd1e8130c495362743ef7428655d|
|project_uri|https://rubygems.org/gems/intermine-bio|
|gem_uri|https://rubygems.org/gems/intermine-bio-0.98.1.gem|
|dependencies|{"development"=>[], "runtime"=>[{"name"=>"intermine", "requirements"=>">= 0"}]}|

</details>

## bio-core-ext

[![Gem Version](https://badge.fury.io/rb/bio-core-ext.svg)](https://badge.fury.io/rb/bio-core-ext)
![Year](https://img.shields.io/static/v1?label=date&message=2011-08-30&color=blue)
![Code Size](https://tokei.rs/b1/github/helios/bioruby-core-ext)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-core-ext)

BioRuby plugins with dependecies from external library or tools

* URL: [http://github.com/helios/bioruby-core-ext](http://github.com/helios/bioruby-core-ext)
* Authors: Raoul J.P. Bonnal, Pjotr Prins

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-core-ext|
|downloads|5473|
|version|0.0.1|
|version_created_at|2011-08-30T12:33:48.972Z|
|version_downloads|5473|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/helios/bioruby-core-ext"}|
|yanked|false|
|sha|3090fb5771ba36309c18582f9637034ca64503b78ff9254f5fb4f00f6c4a2e99|
|spec_sha|908a2a01d135228e18926064de2b2017ce39cacfde37eb097fb19de144e0efc4|
|project_uri|https://rubygems.org/gems/bio-core-ext|
|gem_uri|https://rubygems.org/gems/bio-core-ext-0.0.1.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.0"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio-blastxmlparser", "requirements"=>"= 1.0.1"}, {"name"=>"bio-bwa", "requirements"=>"= 0.2.2"}, {"name"=>"bio-samtools", "requirements"=>"= 0.2.4"}, {"name"=>"bio-sge", "requirements"=>"= 0.0.0"}]}|

</details>

## bio-aliphatic_index

[![Gem Version](https://badge.fury.io/rb/bio-aliphatic_index.svg)](https://badge.fury.io/rb/bio-aliphatic_index)
![Year](https://img.shields.io/static/v1?label=date&message=2012-01-26&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-aliphatic_index)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-aliphatic_index)

A simple biogem that allows computation of the aliphatic index of a protein

* URL: [http://github.com/wwood/bioruby-aliphatic_index](http://github.com/wwood/bioruby-aliphatic_index)
* Authors: Ben J Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-aliphatic_index|
|downloads|5347|
|version|0.0.1|
|version_created_at|2012-01-26T05:24:19.857Z|
|version_downloads|5347|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/wwood/bioruby-aliphatic_index"}|
|yanked|false|
|sha|2b6402750b1d649e11efa6b760d728d080432f4207a4c7c8172443bc81e3b494|
|spec_sha|fc3ddcca78cfd89efe862e75cc0916db07550a9ddda0318f73368f762bc9dd74|
|project_uri|https://rubygems.org/gems/bio-aliphatic_index|
|gem_uri|https://rubygems.org/gems/bio-aliphatic_index-0.0.1.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.4"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-cd-hit-report

[![Gem Version](https://badge.fury.io/rb/bio-cd-hit-report.svg)](https://badge.fury.io/rb/bio-cd-hit-report)
![Year](https://img.shields.io/static/v1?label=date&message=2013-04-26&color=blue)
![Code Size](https://tokei.rs/b1/github/georgeG/bioruby-cd-hit-report)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-cd-hit-report)

A Ruby library for reading CD-HIT cluster reports

* URL: [http://github.com/georgeG/bioruby-cd-hit-report](http://github.com/georgeG/bioruby-cd-hit-report)
* Authors: George Githinji

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-cd-hit-report|
|downloads|16706|
|version|0.1.0|
|version_created_at|2013-04-26T19:38:24.419Z|
|version_downloads|5107|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{"homepage_uri"=>"http://github.com/georgeG/bioruby-cd-hit-report"}|
|yanked|false|
|sha|2e518468a6de52991ec81288b9cefd6ed83a6e76104669c185d633bd2a074282|
|spec_sha|ae2e4e4b1c86a5696bf7f881183aaba08e8bf811fc95a979c6df5c1af48e3576|
|project_uri|https://rubygems.org/gems/bio-cd-hit-report|
|gem_uri|https://rubygems.org/gems/bio-cd-hit-report-0.1.0.gem|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 1.3.1"}, {"name"=>"jeweler", "requirements"=>">= 0"}, {"name"=>"minitest", "requirements"=>">= 0"}, {"name"=>"rdoc", "requirements"=>">= 0"}, {"name"=>"rdoc", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>"= 1.4.3"}]}|

</details>

## bio-img_metadata

[![Gem Version](https://badge.fury.io/rb/bio-img_metadata.svg)](https://badge.fury.io/rb/bio-img_metadata)
![Year](https://img.shields.io/static/v1?label=date&message=2013-01-11&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-img_metadata)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-img_metadata)

Reads metadata from Integrated Microbial Genomes (IMG) metadata files into a programmaticly useful state.

* URL: [http://github.com/wwood/bioruby-img_metadata](http://github.com/wwood/bioruby-img_metadata)
* Authors: Ben J. Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-img_metadata|
|downloads|4937|
|version|0.0.1|
|version_created_at|2013-01-11T01:46:42.293Z|
|version_downloads|4937|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{"homepage_uri"=>"http://github.com/wwood/bioruby-img_metadata"}|
|yanked|false|
|sha|4a7252f3f1c56aa24337aaf8b92511c009baaa14aa42a3023a5749243e14f13c|
|spec_sha|828cfe504bc5938ba1ddcbe68a79f49924e40776be28f55794ca24b895c35112|
|project_uri|https://rubygems.org/gems/bio-img_metadata|
|gem_uri|https://rubygems.org/gems/bio-img_metadata-0.0.1.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>"~> 1.8.4"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rspec", "requirements"=>"~> 2.8.0"}], "runtime"=>[]}|

</details>

## bio-blat-tools

[![Gem Version](https://badge.fury.io/rb/bio-blat-tools.svg)](https://badge.fury.io/rb/bio-blat-tools)
![Year](https://img.shields.io/static/v1?label=date&message=2014-10-03&color=blue)
![Code Size](https://tokei.rs/b1/github/homonecloco/bioruby-blat-tools)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-blat-tools)

Tools to manipulate blat files from the command line. Uses the Bio::Blat object from BioRuby, but the parser has been modified to avoid reading the whole psl file, it yields just the current entry and it is up to the consumer to decide to store it or not."
  s.email = "ricardo.ramirez-gonzalez@tgac.ac.uk"

* URL: [http://github.com/homonecloco/bioruby-blat-tools](http://github.com/homonecloco/bioruby-blat-tools)
* Authors: homonecloco

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-blat-tools|
|downloads|4054|
|version|0.1.0|
|version_created_at|2014-10-03T16:59:09.962Z|
|version_downloads|4054|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|b859f95aa64ce69f1f10278f46b4d623c2d5be69b7804c3b3bc4c778475b9733|
|spec_sha|78cc1230500dcea937805914297c6a48fafbbdfe25ea4507ade7c1f6df4abe8c|
|project_uri|https://rubygems.org/gems/bio-blat-tools|
|gem_uri|https://rubygems.org/gems/bio-blat-tools-0.1.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-blat-tools/0.1.0|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.3"}, {"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>"~> 2.0.1"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"simplecov", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-location-rdfize

[![Gem Version](https://badge.fury.io/rb/bio-location-rdfize.svg)](https://badge.fury.io/rb/bio-location-rdfize)
![Year](https://img.shields.io/static/v1?label=date&message=2014-11-10&color=blue)
![Code Size](https://tokei.rs/b1/github/ngoto/bioruby-location-rdfize)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-location-rdfize)

This is a BioRuby plugin providing FALDO location RDF output method for Bio::Location objects. A method Bio::Location#rdfize is added.

* URL: [http://github.com/ngoto/bioruby-location-rdfize](http://github.com/ngoto/bioruby-location-rdfize)
* Authors: Joachim Baran, Naohisa Goto

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-location-rdfize|
|downloads|4038|
|version|0.0.0|
|version_created_at|2014-11-10T08:22:26.563Z|
|version_downloads|4038|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|a48ac28545b9d1335806a49038fe384c9a4df1a10b5a53eeca682cfb7337518d|
|spec_sha|998cb936d9b1f054c6144e2bccc519965aa26fedb467cac6254e859c6da0b4fe|
|project_uri|https://rubygems.org/gems/bio-location-rdfize|
|gem_uri|https://rubygems.org/gems/bio-location-rdfize-0.0.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-location-rdfize/0.0.0|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>">= 0"}, {"name"=>"rake", "requirements"=>">= 0"}, {"name"=>"rdoc", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}]}|

</details>

## bio-sparql-client

[![Gem Version](https://badge.fury.io/rb/bio-sparql-client.svg)](https://badge.fury.io/rb/bio-sparql-client)
![Year](https://img.shields.io/static/v1?label=date&message=2016-02-19&color=blue)
![Code Size]()
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-sparql-client)

A ruby wrapper for bio SPARQL service

* URL: [http://rubygems.org/gems/bio-sparql-client](http://rubygems.org/gems/bio-sparql-client)
* Authors: Natapol Pornputtapong

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-sparql-client|
|downloads|3897|
|version|0.0.1|
|version_created_at|2016-02-19T23:12:34.800Z|
|version_downloads|3897|
|platform|ruby|
|licenses|["GPL-3.0"]|
|metadata|{}|
|yanked|false|
|sha|dfa5c8a7d3d4b17c9d6a283ac6b0d366e96c1daf6380f0df6fa93076dfc075bd|
|spec_sha|5bbe743e92277988f5996ac6167e7f8708d01b2f62ab60e3f2f392fa8bfe58c1|
|project_uri|https://rubygems.org/gems/bio-sparql-client|
|gem_uri|https://rubygems.org/gems/bio-sparql-client-0.0.1.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-sparql-client/0.0.1|
|dependencies|{"development"=>[], "runtime"=>[{"name"=>"sparql-client", "requirements"=>"~> 1.1"}]}|

</details>

## jandot-bio

[![Gem Version](https://badge.fury.io/rb/jandot-bio.svg)](https://badge.fury.io/rb/jandot-bio)
![Year](https://img.shields.io/static/v1?label=date&message=2014-08-11&color=blue)
![Code Size](https://tokei.rs/b1/github/jandot/bioruby)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/jandot-bio)

BioRuby is a library for bioinformatics (biology + information science).

* URL: [http://github.com/jandot/bioruby](http://github.com/jandot/bioruby)
* Authors: Jan Aerts

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|jandot-bio|
|downloads|3570|
|version|1.2.1|
|version_created_at|2014-08-11T03:22:52.150Z|
|version_downloads|3570|
|platform|ruby|
|licenses|[]|
|metadata|{}|
|yanked|false|
|sha|592fdc936d86b6b51da49d39d39e9863a7b08f7e64747d7e97b8d9c3ae54df1a|
|spec_sha|7de87443a329c0c4d3242a9583ffecd169006d40fb4107992c30bba340a3cd64|
|project_uri|https://rubygems.org/gems/jandot-bio|
|gem_uri|https://rubygems.org/gems/jandot-bio-1.2.1.gem|
|documentation_uri|https://www.rubydoc.info/gems/jandot-bio/1.2.1|
|dependencies|{"development"=>[], "runtime"=>[]}|

</details>

## bio-sam-mutation

[![Gem Version](https://badge.fury.io/rb/bio-sam-mutation.svg)](https://badge.fury.io/rb/bio-sam-mutation)
![Year](https://img.shields.io/static/v1?label=date&message=2016-02-08&color=blue)
![Code Size](https://tokei.rs/b1/github/stveep/bioruby-sam-mutation)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-sam-mutation)

Simple classes for parsing SAM, CIGAR and MD:Z strings, including slices. Methods for calling mutations in HGVS format and looking up consequences using Ensembl VEP REST API. Developed for calling mutations at an expected position in an alignment - e.g. Amplicon sequencing of CRISPR-induced mutations.

* URL: [http://github.com/stveep/bioruby-sam-mutation](http://github.com/stveep/bioruby-sam-mutation)
* Authors: Stephen Pettitt

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-sam-mutation|
|downloads|3398|
|version|0.4.1|
|version_created_at|2016-02-08T09:24:49.557Z|
|version_downloads|3398|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|e2b7031dc32fa5bfd535c47be11e71fc8e4310052b991fdbba9c5edea7da8b29|
|spec_sha|4d53310f347a74c86266d510cb97267c0da046536e046e2d8d48966475674253|
|project_uri|https://rubygems.org/gems/bio-sam-mutation|
|gem_uri|https://rubygems.org/gems/bio-sam-mutation-0.4.1.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-sam-mutation/0.4.1|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 0"}, {"name"=>"jeweler", "requirements"=>"~> 2.0"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"simplecov", "requirements"=>">= 0"}, {"name"=>"test-unit", "requirements"=>"~> 3.0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bio-ensembl-rest", "requirements"=>"= 0.2.0"}, {"name"=>"bio-samtools", "requirements"=>"~> 2.3.4"}, {"name"=>"oj", "requirements"=>"~> 2.14"}, {"name"=>"rake", "requirements"=>"~> 0.9"}, {"name"=>"trollop", "requirements"=>">= 0"}]}|

</details>

## bio-wolf_psort_wrapper

[![Gem Version](https://badge.fury.io/rb/bio-wolf_psort_wrapper.svg)](https://badge.fury.io/rb/bio-wolf_psort_wrapper)
![Year](https://img.shields.io/static/v1?label=date&message=2012-01-20&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-wolf_psort_wrapper)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-wolf_psort_wrapper)

Enables the localisation predictor WoLF PSORT to be run locally. The algorithm is available at http://wolfpsort.org/

* URL: [http://github.com/wwood/bioruby-wolf_psort_wrapper](http://github.com/wwood/bioruby-wolf_psort_wrapper)
* Authors: Ben J Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-wolf_psort_wrapper|
|downloads|9599|
|version|0.0.3|
|version_created_at|2012-01-20T11:17:05.885Z|
|version_downloads|5376|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/wwood/bioruby-wolf_psort_wrapper"}|
|yanked|false|
|sha|ee1144bda0f73158336222f07f9dae32856ced05345d267d4891f41e55017c4a|
|spec_sha|80698070c715eaa6f7b720cfa05454130a4cc0b973ef20605d6a721253eff585|
|project_uri|https://rubygems.org/gems/bio-wolf_psort_wrapper|
|gem_uri|https://rubygems.org/gems/bio-wolf_psort_wrapper-0.0.3.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.4"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-blast-xmlparser

[![Gem Version](https://badge.fury.io/rb/bio-blast-xmlparser.svg)](https://badge.fury.io/rb/bio-blast-xmlparser)
![Year](https://img.shields.io/static/v1?label=date&message=2019-06-14&color=blue)
![Code Size](https://tokei.rs/b1/github/bioruby/bioruby-blast-xmlparser)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-blast-xmlparser)

BioRuby BLAST XMLParser plugin: a plugin of BioRuby BLAST XML result parser by using the XMLParser extension. This was formerly a part of the BioRuby core ("bio" gem) and split to this gem because of the external XMLParser dependency.

* URL: [https://github.com/bioruby/bioruby-blast-xmlparser](https://github.com/bioruby/bioruby-blast-xmlparser)
* Authors: BioRuby project

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-blast-xmlparser|
|downloads|3024|
|version|0.0.0|
|version_created_at|2019-06-14T04:19:00.189Z|
|version_downloads|3024|
|platform|ruby|
|licenses|[]|
|metadata|{}|
|yanked|false|
|sha|854ffaab93955fd32fbc41b1bdd294922d306f2c8f775a4ff5ddf065afc2ec07|
|spec_sha|8366bdc0eb6988b28d3aeba4a150802fac164d99934b8a65ba5c3b04a8c12bb0|
|project_uri|https://rubygems.org/gems/bio-blast-xmlparser|
|gem_uri|https://rubygems.org/gems/bio-blast-xmlparser-0.0.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-blast-xmlparser/0.0.0|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 1.15"}, {"name"=>"rake", "requirements"=>">= 10.0"}, {"name"=>"rdoc", "requirements"=>">= 5"}, {"name"=>"test-unit", "requirements"=>"~> 3"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 2.0.0"}, {"name"=>"xmlparser", "requirements"=>"~> 0.7.2"}]}|

</details>

## bio-old-biofetch-emulator

[![Gem Version](https://badge.fury.io/rb/bio-old-biofetch-emulator.svg)](https://badge.fury.io/rb/bio-old-biofetch-emulator)
![Year](https://img.shields.io/static/v1?label=date&message=2016-06-13&color=blue)
![Code Size](https://tokei.rs/b1/github/ngoto/bioruby-old-biofetch-emulator)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-old-biofetch-emulator)

Emulator that emulates Bio::Fetch object in BioRuby as if old BioRuby BioFetch server were still alive. It overrides methods and objects in Bio::Fetch, and if the old BioRuby BioFetch server's URL is given, it intercepts all requests and converts them into existing web services such as TogoWS, KEGG REST API, NCBI E-Utilities, and GenomeNet(genome.jp).

* URL: [http://github.com/ngoto/bioruby-old-biofetch-emulator](http://github.com/ngoto/bioruby-old-biofetch-emulator)
* Authors: Naohisa Goto

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-old-biofetch-emulator|
|downloads|8053|
|version|1.0.0|
|version_created_at|2016-06-13T11:44:15.157Z|
|version_downloads|4810|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|2cebae441cd8d680cbf76df5301b7d45a3d086974cdc016b9148a15dfb8b6d8b|
|spec_sha|c6f6a35e20c00c9e897b86d608cca8695407a1d1bc430509ad2918d4fc58fa6e|
|project_uri|https://rubygems.org/gems/bio-old-biofetch-emulator|
|gem_uri|https://rubygems.org/gems/bio-old-biofetch-emulator-1.0.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-old-biofetch-emulator/1.0.0|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>">= 1.0.21"}, {"name"=>"jeweler", "requirements"=>">= 0"}, {"name"=>"rake", "requirements"=>">= 0"}, {"name"=>"rdoc", "requirements"=>">= 0"}, {"name"=>"test-unit", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}]}|

</details>

## bio-express_beta_diversity

[![Gem Version](https://badge.fury.io/rb/bio-express_beta_diversity.svg)](https://badge.fury.io/rb/bio-express_beta_diversity)
![Year](https://img.shields.io/static/v1?label=date&message=2014-01-21&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-express_beta_diversity)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-express_beta_diversity)

Interface for express beta diversity file formats

* URL: [http://github.com/wwood/bioruby-express_beta_diversity](http://github.com/wwood/bioruby-express_beta_diversity)
* Authors: Ben J. Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-express_beta_diversity|
|downloads|7679|
|version|0.1.0|
|version_created_at|2014-01-21T05:33:41.597Z|
|version_downloads|4359|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|18e1cf7c26e4d23a40550c38aa0f56986abe1cb71bc774ed0d21f53990396dd3|
|spec_sha|b1afbd4461b8ade272f3382a490168e8c7f3f73d08517732c3088da8182d6906|
|project_uri|https://rubygems.org/gems/bio-express_beta_diversity|
|gem_uri|https://rubygems.org/gems/bio-express_beta_diversity-0.1.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-express_beta_diversity/0.1.0|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>"~> 1.4"}, {"name"=>"bundler", "requirements"=>"~> 1.3"}, {"name"=>"jeweler", "requirements"=>"~> 2.0"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"rspec", "requirements"=>"~> 2.8"}], "runtime"=>[{"name"=>"bio-logger", "requirements"=>"~> 1.0"}]}|

</details>

## lorem_ipsum_bio

[![Gem Version](https://badge.fury.io/rb/lorem_ipsum_bio.svg)](https://badge.fury.io/rb/lorem_ipsum_bio)
![Year](https://img.shields.io/static/v1?label=date&message=2017-04-29&color=blue)
![Code Size](https://tokei.rs/b1/github/pjanek/lorem_ipsum_bio)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/lorem_ipsum_bio)

Lorem Ipsum Bio is a toolkit for generating varied random biological sequences (especially usable as a control group in analysis)

* URL: [https://github.com/pjanek/lorem_ipsum_bio](https://github.com/pjanek/lorem_ipsum_bio)
* Authors: Pawe Jankowski

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|lorem_ipsum_bio|
|downloads|11166|
|version|0.1.0|
|version_created_at|2017-04-29T14:03:27.459Z|
|version_downloads|2494|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|af3da433aabd069dae00109cdf8ac97730acce19bf32234aa2f85d9dada132fa|
|spec_sha|25fa7d029b2817654ef63857e236c7b4f44ced3fd19f174a57b5002fc62051c3|
|project_uri|https://rubygems.org/gems/lorem_ipsum_bio|
|gem_uri|https://rubygems.org/gems/lorem_ipsum_bio-0.1.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/lorem_ipsum_bio/0.1.0|
|dependencies|{"development"=>[], "runtime"=>[]}|

</details>

## bio-data-restriction_enzyme

[![Gem Version](https://badge.fury.io/rb/bio-data-restriction_enzyme.svg)](https://badge.fury.io/rb/bio-data-restriction_enzyme)
![Year](https://img.shields.io/static/v1?label=date&message=2011-09-10&color=blue)
![Code Size](https://tokei.rs/b1/github/trevor/bioruby-data-restriction_enzyme)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-data-restriction_enzyme)

Restriction Enzyme Dataset from REBASE in yaml form

* URL: [http://github.com/trevor/bioruby-data-restriction_enzyme](http://github.com/trevor/bioruby-data-restriction_enzyme)
* Authors: Sir Richard J. Roberts, Trevor Wennblom

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-data-restriction_enzyme|
|downloads|5457|
|version|1.109.0|
|version_created_at|2011-09-10T04:31:54.523Z|
|version_downloads|5457|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/trevor/bioruby-data-restriction_enzyme"}|
|yanked|false|
|sha|0b4698b20a2c082ad1137a2d491d52d9541ea14b0a4e9bdb52158d96a511181e|
|spec_sha|ab988ba9cb5bad9d6902ffc6706a6b4b62aa7f4f6ebe780cd2730bb4909b4fed|
|project_uri|https://rubygems.org/gems/bio-data-restriction_enzyme|
|gem_uri|https://rubygems.org/gems/bio-data-restriction_enzyme-1.109.0.gem|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}, {"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.6.4"}, {"name"=>"rcov", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bio-blast_seq_extract

[![Gem Version](https://badge.fury.io/rb/bio-blast_seq_extract.svg)](https://badge.fury.io/rb/bio-blast_seq_extract)
![Year](https://img.shields.io/static/v1?label=date&message=2019-12-20&color=blue)
![Code Size]()
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-blast_seq_extract)

Extract the sequence of the subjects' region based on blast search.

* URL: []()
* Authors: Sishuo Wang

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-blast_seq_extract|
|downloads|5219|
|version|0.2.5|
|version_created_at|2019-12-20T01:42:41.066Z|
|version_downloads|2527|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{"allowed_push_host"=>"https://rubygems.org"}|
|yanked|false|
|sha|bd2e8acf736b66471edde2437def59a7242b65fbac79a7ad98371aeb32f471c8|
|spec_sha|95da0eccbe03ca56035e9fda637122560fc282e82909ce3e197a06f95e29a5b7|
|project_uri|https://rubygems.org/gems/bio-blast_seq_extract|
|gem_uri|https://rubygems.org/gems/bio-blast_seq_extract-0.2.5.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-blast_seq_extract/0.2.5|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>"~> 1.5"}, {"name"=>"bundler", "requirements"=>"~> 2.0"}, {"name"=>"rake", "requirements"=>"~> 10.0"}, {"name"=>"rspec", "requirements"=>"~> 3.0"}], "runtime"=>[]}|

</details>

## bio-polymarker

[![Gem Version](https://badge.fury.io/rb/bio-polymarker.svg)](https://badge.fury.io/rb/bio-polymarker)
![Year](https://img.shields.io/static/v1?label=date&message=2024-09-26&color=blue)
![Code Size]()
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-polymarker)

Repository of tools developed at Crop Genetics in JIC to work with polyploid wheat

* URL: []()
* Authors: Rob Ellis

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-polymarker|
|downloads|437|
|version|1.3.3|
|version_created_at|2024-09-26T15:50:49.438Z|
|version_downloads|247|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|8a987819e88ec3ddcefb35250325e4cab43b702d4cf2918cf93b0cea1123cb9a|
|spec_sha|043692a259a3c90ba9818a675d4b85702870d30e801bcbbc26bec47cc54fedb1|
|project_uri|https://rubygems.org/gems/bio-polymarker|
|gem_uri|https://rubygems.org/gems/bio-polymarker-1.3.3.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-polymarker/1.3.3|
|dependencies|{"development"=>[{"name"=>"juwelier", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 2.10"}, {"name"=>"sorted_set", "requirements"=>">= 0"}, {"name"=>"test-unit", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.5.1"}, {"name"=>"bio-samtools-wrapper", "requirements"=>">= 2.7.0"}, {"name"=>"descriptive_statistics", "requirements"=>">= 0"}, {"name"=>"systemu", "requirements"=>">= 2.5.2"}]}|

</details>

## bio-emboss_six_frame_nucleotide_sequences

[![Gem Version](https://badge.fury.io/rb/bio-emboss_six_frame_nucleotide_sequences.svg)](https://badge.fury.io/rb/bio-emboss_six_frame_nucleotide_sequences)
![Year](https://img.shields.io/static/v1?label=date&message=2011-04-03&color=blue)
![Code Size](https://tokei.rs/b1/github/wwood/bioruby-emboss_six_frame_nucleotide_sequences)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-emboss_six_frame_nucleotide_sequences)

a method to get the nucleotide sequence of translations done by the EMBOSS bioinformatics package program transeq.

* URL: [http://github.com/wwood/bioruby-emboss_six_frame_nucleotide_sequences](http://github.com/wwood/bioruby-emboss_six_frame_nucleotide_sequences)
* Authors: Ben J Woodcroft

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-emboss_six_frame_nucleotide_sequences|
|downloads|5731|
|version|0.1.0|
|version_created_at|2011-04-03T09:39:04.117Z|
|version_downloads|5731|
|platform|ruby|
|metadata|{"homepage_uri"=>"http://github.com/wwood/bioruby-emboss_six_frame_nucleotide_sequences"}|
|yanked|false|
|sha|8dc290e6c37f6e6eeba2433910d54c811bc513ba10304ca26b74a1c650588c02|
|spec_sha|56822b3165b2a281630266170a43070dc948f2be121e3831b7046ff21dea10d4|
|project_uri|https://rubygems.org/gems/bio-emboss_six_frame_nucleotide_sequences|
|gem_uri|https://rubygems.org/gems/bio-emboss_six_frame_nucleotide_sequences-0.1.0.gem|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.0.0"}, {"name"=>"jeweler", "requirements"=>"~> 1.5.2"}, {"name"=>"rcov", "requirements"=>">= 0"}, {"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.1"}]}|

</details>

## bio-samtools-wrapper

[![Gem Version](https://badge.fury.io/rb/bio-samtools-wrapper.svg)](https://badge.fury.io/rb/bio-samtools-wrapper)
![Year](https://img.shields.io/static/v1?label=date&message=2024-09-25&color=blue)
![Code Size]()
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bio-samtools-wrapper)

Wrapper of samtools for ruby. 

  This project was born from the need to add support of BAM files to 
  the gee_fu genome browser (http://github.com/danmaclean/gee_fu).

* URL: []()
* Authors: Rob Ellis

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bio-samtools-wrapper|
|downloads|275|
|version|2.7.0|
|version_created_at|2024-09-25T14:01:29.076Z|
|version_downloads|275|
|platform|ruby|
|licenses|["GPL-3.0"]|
|metadata|{}|
|yanked|false|
|sha|0886ea063b62e680fdbd0bef2e1e7b15da4eee80df881a487881dbb7a49e3532|
|spec_sha|ebb19a56d7d6ad4a481a7ef7073aad5039b374a4bb5531808859753ca97982d6|
|project_uri|https://rubygems.org/gems/bio-samtools-wrapper|
|gem_uri|https://rubygems.org/gems/bio-samtools-wrapper-2.7.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bio-samtools-wrapper/2.7.0|
|dependencies|{"development"=>[{"name"=>"juwelier", "requirements"=>">= 0"}, {"name"=>"rack", "requirements"=>"= 1.6.4"}, {"name"=>"shoulda", "requirements"=>"= 2.10"}, {"name"=>"test-unit", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bio-svgenes", "requirements"=>">= 0.4.1"}]}|

</details>

## linkhub-jekyll-theme

[![Gem Version](https://badge.fury.io/rb/linkhub-jekyll-theme.svg)](https://badge.fury.io/rb/linkhub-jekyll-theme)
![Year](https://img.shields.io/static/v1?label=date&message=2024-05-22&color=blue)
![Code Size](https://tokei.rs/b1/github/digitalmalayali/linkhub-jekyll-theme)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/linkhub-jekyll-theme)

A minimal and super-lightweight free Jekyll theme to create a link-in-bio website like Linktree.

* URL: [https://digitalmalayali.github.io/linkhub-jekyll-theme](https://digitalmalayali.github.io/linkhub-jekyll-theme)
* Authors: Digital Malayali

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|linkhub-jekyll-theme|
|downloads|2280|
|version|0.1.6|
|version_created_at|2024-05-22T21:47:05.862Z|
|version_downloads|352|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{"funding_uri"=>"https://liberapay.com/zcraber", "changelog_uri"=>"https://github.com/digitalmalayali/linkhub-jekyll-theme/blob/main/CHANGELOG.md", "bug_tracker_uri"=>"https://github.com/digitalmalayali/linkhub-jekyll-theme/issues", "source_code_uri"=>"https://github.com/digitalmalayali/linkhub-jekyll-theme", "documentation_uri"=>"https://github.com/digitalmalayali/linkhub-jekyll-theme#readme"}|
|yanked|false|
|sha|0fed4d4255ebc7c342a3b2269681742917f2b2fe4c217cfc3875d55dad359921|
|spec_sha|12663515f1b95974b9babf7e0af535fc09f2b9c1647b34438b15b2f55e0bbc6c|
|project_uri|https://rubygems.org/gems/linkhub-jekyll-theme|
|gem_uri|https://rubygems.org/gems/linkhub-jekyll-theme-0.1.6.gem|
|documentation_uri|https://github.com/digitalmalayali/linkhub-jekyll-theme#readme|
|source_code_uri|https://github.com/digitalmalayali/linkhub-jekyll-theme|
|bug_tracker_uri|https://github.com/digitalmalayali/linkhub-jekyll-theme/issues|
|changelog_uri|https://github.com/digitalmalayali/linkhub-jekyll-theme/blob/main/CHANGELOG.md|
|funding_uri|https://liberapay.com/zcraber|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 2.4"}], "runtime"=>[{"name"=>"jekyll", "requirements"=>"~> 4.3"}]}|

</details>

## geni-svgenes

[![Gem Version](https://badge.fury.io/rb/geni-svgenes.svg)](https://badge.fury.io/rb/geni-svgenes)
![Year](https://img.shields.io/static/v1?label=date&message=2017-01-30&color=blue)
![Code Size](https://tokei.rs/b1/github/southpawlife/geni-svgenes)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/geni-svgenes)

Modified version of the bio-svgenes gem, for use in https://geni-act.org education website.

* URL: [http://github.com/southpawlife/geni-svgenes](http://github.com/southpawlife/geni-svgenes)
* Authors: Kristopher Williams

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|geni-svgenes|
|downloads|3043|
|version|1.1.0|
|version_created_at|2017-01-30T00:03:05.367Z|
|version_downloads|3016|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{}|
|yanked|false|
|sha|34f65f06b33d0451e4e1dff25645d18b14f0f17a80799581462e8714f2a543b9|
|spec_sha|d7ba83d406c9f55371c7ace571dec8935f922904935f859f81cbc1715382838c|
|project_uri|https://rubygems.org/gems/geni-svgenes|
|gem_uri|https://rubygems.org/gems/geni-svgenes-1.1.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/geni-svgenes/1.1.0|
|dependencies|{"development"=>[{"name"=>"bio", "requirements"=>">= 1.4.2"}, {"name"=>"bundler", "requirements"=>"~> 1.0"}, {"name"=>"json", "requirements"=>">= 1.7"}, {"name"=>"juwelier", "requirements"=>"~> 2.1.0"}, {"name"=>"rdoc", "requirements"=>"~> 3.12"}, {"name"=>"shoulda", "requirements"=>">= 0"}, {"name"=>"simplecov", "requirements"=>">= 0"}], "runtime"=>[]}|

</details>

## bioruby-phyloxml

[![Gem Version](https://badge.fury.io/rb/bioruby-phyloxml.svg)](https://badge.fury.io/rb/bioruby-phyloxml)
![Year](https://img.shields.io/static/v1?label=date&message=2016-06-15&color=blue)
![Code Size](https://tokei.rs/b1/github/bioruby/bioruby-phyloxml)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/bioruby-phyloxml)

Provides PhyloXML support for BioRuby. This bioruby-phyloxml gem replaces old unmaintained bio-phyloxml gem.

* URL: [http://github.com/bioruby/bioruby-phyloxml](http://github.com/bioruby/bioruby-phyloxml)
* Authors: Diana Jaunzeikare, Clayton Wheeler, BioRuby project

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|bioruby-phyloxml|
|downloads|4109|
|version|1.0.0|
|version_created_at|2016-06-15T15:02:23.934Z|
|version_downloads|4109|
|platform|ruby|
|licenses|["Ruby"]|
|metadata|{}|
|yanked|false|
|sha|00d87cba440c80812ad94ae6af455e18df52a39216126ddad66e385b4c80fc46|
|spec_sha|98ad91ad42ef13f51da32337e54825526bc33fb316ec398bed5e8c86f7e71fe6|
|project_uri|https://rubygems.org/gems/bioruby-phyloxml|
|gem_uri|https://rubygems.org/gems/bioruby-phyloxml-1.0.0.gem|
|documentation_uri|https://www.rubydoc.info/gems/bioruby-phyloxml/1.0.0|
|dependencies|{"development"=>[{"name"=>"bundler", "requirements"=>"~> 1.10"}, {"name"=>"rake", "requirements"=>"~> 10.0"}, {"name"=>"rdoc", "requirements"=>"~> 4"}], "runtime"=>[{"name"=>"bio", "requirements"=>">= 1.5.0"}, {"name"=>"libxml-ruby", "requirements"=>"~> 2.8"}]}|

</details>

## eutils

[![Gem Version](https://badge.fury.io/rb/eutils.svg)](https://badge.fury.io/rb/eutils)
![Year](https://img.shields.io/static/v1?label=date&message=2013-06-27&color=blue)
![Code Size](https://tokei.rs/b1/github/seouri/eutils)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/eutils)

Lightweight Ruby API for NCBI Eutils. This gem only provides API for NCBI Eutils. If you need full access to other NCBI databases, try BioRuby (bio gem) instead.

* URL: [http://github.com/seouri/eutils](http://github.com/seouri/eutils)
* Authors: Joon Lee, aka seouri

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|eutils|
|downloads|19975|
|version|0.2.1|
|version_created_at|2013-06-27T20:08:20.771Z|
|version_downloads|4321|
|platform|ruby|
|licenses|[]|
|metadata|{}|
|yanked|false|
|sha|ed151b01ecc2f9c7e321afab0a7b4c228c3b3eb7c736a98bc57b2e0b80c92ca3|
|spec_sha|44d1403ee10831ff8774be5528801af738f42d34224f0e406834baf87fbdf33e|
|project_uri|https://rubygems.org/gems/eutils|
|gem_uri|https://rubygems.org/gems/eutils-0.2.1.gem|
|documentation_uri|https://www.rubydoc.info/gems/eutils/0.2.1|
|dependencies|{"development"=>[{"name"=>"shoulda", "requirements"=>">= 0"}], "runtime"=>[{"name"=>"activesupport", "requirements"=>">= 3.0.0"}]}|

</details>

## htslib

[![Gem Version](https://badge.fury.io/rb/htslib.svg)](https://badge.fury.io/rb/htslib)
![Year](https://img.shields.io/static/v1?label=date&message=2024-09-13&color=blue)
![Code Size](https://tokei.rs/b1/github/kojix2/ruby-htslib)
[![RubyDoc](https://img.shields.io/badge/rubydoc-blue.svg)](https://rubydoc.info/gems/htslib)

HTSlib bindings for Ruby

* URL: [https://github.com/kojix2/ruby-htslib](https://github.com/kojix2/ruby-htslib)
* Authors: kojix2

<details>
<summary><b>table</b></summary>

|key|value|
|---|-----|
|name|htslib|
|downloads|19019|
|version|0.2.9|
|version_created_at|2024-09-13T04:41:49.861Z|
|version_downloads|239|
|platform|ruby|
|licenses|["MIT"]|
|metadata|{"msys2_mingw_dependencies"=>"htslib"}|
|yanked|false|
|sha|aaf5a125d2c71e3c57bee935fce1f29342ae01be05afc9be78382edb4264e738|
|spec_sha|db32e2ae309ffc8dc075c3502f9e623fe1d8e9ac7a0b63b01e67cfbf1f0735c2|
|project_uri|https://rubygems.org/gems/htslib|
|gem_uri|https://rubygems.org/gems/htslib-0.2.9.gem|
|documentation_uri|https://www.rubydoc.info/gems/htslib/0.2.9|
|dependencies|{"development"=>[], "runtime"=>[{"name"=>"ffi", "requirements"=>">= 0"}, {"name"=>"ffi-bitfield", "requirements"=>">= 0"}, {"name"=>"pkg-config", "requirements"=>">= 0"}]}|

</details>

