# Bio Gems

## bio

|key|value|
|---|-----|
|info|BioRuby is a library for bioinformatics (biology + information science).
|url|http://bioruby.org/|
|authors|BioRuby project|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|http://bioruby.org/rdoc/|
|metadata|{}|
|bug_tracker_uri|https://github.com/bioruby/bioruby/issues|
|project_uri|https://rubygems.org/gems/bio|
|version|2.0.3|
|sha|b931ed0c94498bacb8c7dbd87c680672dbbae059299a3be1ee8c9f2b56b5816f|
|platform|ruby|
|source_code_uri|https://github.com/bioruby/bioruby/|
|licenses|["Ruby"]|
|gem_uri|https://rubygems.org/gems/bio-2.0.3.gem|
|downloads|406832|
|mailing_list_uri|http://lists.open-bio.org/mailman/listinfo/bioruby|
|name|bio|
|wiki_uri|http://bioruby.open-bio.org/wiki/|
|version_downloads|17429|

</details>

## bio-samtools

|key|value|
|---|-----|
|info|Binder of samtools for ruby, on the top of FFI. 

  This project was born from the need to add support of BAM files to 
  the gee_fu genome browser (http://github.com/danmaclean/gee_fu).
|url|http://github.com/helios/bioruby-samtools|
|authors|Ricardo Ramirez-Gonzalez, Dan MacLean, Raoul J.P. Bonnal|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-samtools/2.6.2|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-samtools|
|version|2.6.2|
|sha|0643ef0f131b29ab786280e68fc639e0db18689ffec77f655e3ab4b5a49420ad|
|platform|ruby|
|licenses|["GPL-3.0"]|
|gem_uri|https://rubygems.org/gems/bio-samtools-2.6.2.gem|
|downloads|100095|
|name|bio-samtools|
|version_downloads|2805|

</details>

## bio-gem

|key|value|
|---|-----|
|info|Biogem is a software generator for those bioinformaticans who want to start coding an application or a library for using/extending BioRuby core library and sharing it through rubygems.org .
  The basic idea is to simplify and promote a modular approach to bioinformatics software development
|url|http://github.com/helios/bioruby-gem|
|authors|Raoul J.P. Bonnal, Pjotr Prins|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-gem/1.3.6|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-gem|
|version|1.3.6|
|sha|a96253804a8ac3a9bce64f560bac5a41a1492d83bf33e73ad72050aa8223649d|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-gem-1.3.6.gem|
|downloads|81754|
|name|bio-gem|
|version_downloads|3428|

</details>

## bio-gadget

|key|value|
|---|-----|
|info|Gadgets for bioinformatics
|url|https://github.com/shka/ruby-bio-gadget|
|authors|Shintaro Katayama|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-gadget/0.5.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-gadget|
|version|0.5.0|
|sha|0dfe50b252972194227b888ef6b3a4af3bccef986feeb90970b7abf40c2dd08e|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-gadget-0.5.0.gem|
|downloads|67716|
|name|bio-gadget|
|version_downloads|1935|

</details>

## bio-svgenes

|key|value|
|---|-----|
|info|This bio-gem facilitates the creation of pretty, publication quality SVG images from feature data.
|url|http://github.com/danmaclean/bioruby-svgenes|
|authors|Dan MacLean|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-svgenes/0.4.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-svgenes|
|version|0.4.1|
|sha|86e17e70fe5a5ae0c28896b0897eb5d7815ab74f7f045ebffaa05ad8d2c19eda|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-svgenes-0.4.1.gem|
|downloads|50548|
|name|bio-svgenes|
|version_downloads|11742|

</details>

## bio-maf

|key|value|
|---|-----|
|info|Multiple Alignment Format parser for BioRuby.
|url|http://github.com/csw/bioruby-maf|
|authors|Clayton Wheeler|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-maf/1.0.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-maf|
|version|1.0.1|
|sha|b0568a723f04c62211c0eda85baad5a314333ae06d866cd8f4f4b42cf9016f26|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-maf-1.0.1.gem|
|downloads|43293|
|name|bio-maf|
|version_downloads|4266|

</details>

## bio-fastqc

|key|value|
|---|-----|
|info|ruby parser for FastQC, a quality control software for high-throughput sequencing data.
|url|http://github.com/inutano/bioruby-fastqc|
|authors|Tazro Inutano Ohta|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-fastqc/0.10.8|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-fastqc|
|version|0.10.8|
|sha|fee9b5551f7fe8c0bd3598a23c7b963f55807f0ee3839fb9388d68557aad8d4e|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-fastqc-0.10.8.gem|
|downloads|42656|
|name|bio-fastqc|
|version_downloads|1486|

</details>

## bio-blastxmlparser

|key|value|
|---|-----|
|info|Fast big data BLAST XML parser and library; this libxml2 based version is 50x faster than BioRuby and comes with a nice CLI
|url|http://github.com/pjotrp/blastxmlparser|
|authors|Pjotr Prins|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-blastxmlparser/2.0.4|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-blastxmlparser|
|version|2.0.4|
|sha|2299c27e40c4853f361643d6611c4dfa85ec9ed2bb52672a664fb6ca9ec7c4f1|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-blastxmlparser-2.0.4.gem|
|downloads|41649|
|name|bio-blastxmlparser|
|version_downloads|5319|

</details>

## bio-rocker

|key|value|
|---|-----|
|info|Detecting and quantifying functional genes in short-read metagenomic datasets
|url|http://enve-omics.ce.gatech.edu/rocker|
|authors|Luis (Coto) Orellana, Luis M. Rodriguez-R|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-rocker/1.5.2|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-rocker|
|version|1.5.2|
|sha|b500c32eaed5f6fabf64cb384d18ef5a7f03e17e016d46a6b5c8c56ac118119d|
|platform|ruby|
|licenses|["Artistic-2.0"]|
|gem_uri|https://rubygems.org/gems/bio-rocker-1.5.2.gem|
|downloads|40534|
|name|bio-rocker|
|version_downloads|1942|

</details>

## bio-grid

|key|value|
|---|-----|
|info|A BioGem to submit jobs on a queue system
|url|http://github.com/fstrozzi/bioruby-grid|
|authors|Francesco Strozzi|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-grid/0.3.3|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-grid|
|version|0.3.3|
|sha|e4f9fb68abf87ba7f057ce036bfe591d539af6bb28c0cf10f560b2513c867da1|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-grid-0.3.3.gem|
|downloads|38603|
|name|bio-grid|
|version_downloads|3912|

</details>

## bio-gff3

|key|value|
|---|-----|
|info|GFF3 (genome browser) information and digest mRNA and CDS sequences.
Options for low memory use and caching of records.
Support for external FASTA files.

|url|https://github.com/pjotrp/bioruby-gff3-plugin|
|authors|Pjotr Prins|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-gff3/0.9.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-gff3|
|version|0.9.1|
|sha|392e6a9f4ef3f60fc3fa16ee9a9a5679b12f05a32f1d12f83dd4dd4fa6a2846d|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-gff3-0.9.1.gem|
|downloads|38208|
|name|bio-gff3|
|version_downloads|4367|

</details>

## bio-bgzf

|key|value|
|---|-----|
|info|This library provides support for BGZF (Blocked GZip Format) in Ruby. BGZF, originally defined as part of the SAM/BAM specification, is used to compress record-oriented data in a way that facilitates random access, unlike plain gzip. BGZF is principally used for bioinformatics data but would be useful in other contexts as well.
|url|http://github.com/csw/bioruby-bgzf|
|authors|Artem Tarasov, Clayton Wheeler|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-bgzf/0.2.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-bgzf|
|version|0.2.1|
|sha|29556916ce8a4b69d7bd69db7748df04d4c3c1ea94285f70160e4f545608812c|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-bgzf-0.2.1.gem|
|downloads|32711|
|name|bio-bgzf|
|version_downloads|23046|

</details>

## bio-band

|key|value|
|---|-----|
|info|Data mining and machine learning algorithms for JRuby 
|url|http://github.com/arrigonialberto86/bioruby-band|
|authors|arrigonialberto86|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-band/0.1.11|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-band|
|version|0.1.11|
|sha|8d3da73dd15a41cdc4c53731eba742b15eb6925d66d8ab18be500b8e7f41a08d|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-band-0.1.11.gem|
|downloads|30752|
|name|bio-band|
|version_downloads|3584|

</details>

## bio-logger

|key|value|
|---|-----|
|info|Log4r wrapper for BioRuby
|url|https://github.com/pjotrp/bioruby-logger-plugin|
|authors|Pjotr Prins|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-logger/1.0.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-logger|
|version|1.0.1|
|sha|4834e677a7350a276486128f587f90d3d59f59b0d06e7abfffb84d8bbca3fe0a|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-logger-1.0.1.gem|
|downloads|30402|
|name|bio-logger|
|version_downloads|14433|

</details>

## bio-alignment

|key|value|
|---|-----|
|info|Support for multiple sequence alignments (MSA)
|url|http://github.com/pjotrp/bioruby-alignment|
|authors|Pjotr Prins|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-alignment/0.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-alignment|
|version|0.1.0|
|sha|5b0ebd3ae6e28c74454ed7f72102a27403fbecbf81c180a46febf82d8039d6dd|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-alignment-0.1.0.gem|
|downloads|29557|
|name|bio-alignment|
|version_downloads|1758|

</details>

## bio-vcf

|key|value|
|---|-----|
|info|Smart lazy multi-threaded parser for VCF format with useful filtering and output rewriting (JSON, RDF etc.)
|url|http://github.com/vcflib/bio-vcf|
|authors|Pjotr Prins|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-vcf/0.9.5|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-vcf|
|version|0.9.5|
|sha|cfbc2d226f09df69cbe063b863d4a3c902f41576de9bd9af1ee1239dec2d9cbe|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-vcf-0.9.5.gem|
|downloads|29376|
|name|bio-vcf|
|version_downloads|1209|

</details>

## bio-table

|key|value|
|---|-----|
|info|Functions and tools for tranforming and changing tab delimited and comma separated table files - useful for Excel sheets and SQL/RDF output
|url|http://github.com/pjotrp/bioruby-table|
|authors|Pjotr Prins|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-table/1.0.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-table|
|version|1.0.0|
|sha|d9d85c99d3afa788c81c4f48f5782e93846c0f39c20beb5c0ea44d328ea697ca|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-table-1.0.0.gem|
|downloads|28881|
|name|bio-table|
|version_downloads|3787|

</details>

## bio-publisci

|key|value|
|---|-----|
|info|A toolkit for publishing scientific results and datasets using RDF, OWL, and related technologies 
|url|http://github.com/wstrinz/bioruby-publisci|
|authors|Will Strinz|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-publisci/0.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-publisci|
|version|0.1.0|
|sha|f202b23f14c56ab3b3446b9610e03429c30562ff370007ba260b147f122c2da8|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-publisci-0.1.0.gem|
|downloads|25839|
|name|bio-publisci|
|version_downloads|3582|

</details>

## bio-faster

|key|value|
|---|-----|
|info|A fast parser for FastQ files
|url|http://github.com/fstrozzi/bioruby-faster|
|authors|Francesco Strozzi|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-faster/0.4.5|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-faster|
|version|0.4.5|
|sha|a3fa411c33e019428c79faa5cf97f1c5ca5eb2797174cad985944f8d8fdeb97a|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-faster-0.4.5.gem|
|downloads|25286|
|name|bio-faster|
|version_downloads|4684|

</details>

## bio-ngs

|key|value|
|---|-----|
|info|bio-ngs provides a framework for handling NGS data with BioRuby
|url|http://github.com/helios/bioruby-ngs|
|authors|Francesco Strozzi, Raoul J.P. Bonnal|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-ngs/0.5.2|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-ngs|
|version|0.5.2|
|sha|b0f766ed6e464ab32608dc78b0d4fe07aab999238d389d4667a104a862e768a9|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-ngs-0.5.2.gem|
|downloads|23629|
|name|bio-ngs|
|version_downloads|4546|

</details>

## bio-bigbio

|key|value|
|---|-----|
|info|Fasta reader, ORF emitter, sequence translation
|url|http://github.com/pjotrp/bigbio|
|authors|Pjotr Prins|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-bigbio/0.1.6|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-bigbio|
|version|0.1.6|
|sha|19fdcf1efa3479cf93191a8e703e61e4fc63d46be41ad1eac92d8a89c94ecc11|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-bigbio-0.1.6.gem|
|downloads|21016|
|name|bio-bigbio|
|version_downloads|3341|

</details>

## bio-pipengine

|key|value|
|---|-----|
|info|A pipeline manager
|url|http://github.com/fstrozzi/bioruby-pipengine|
|authors|Francesco Strozzi, Raoul Jean Pierre Bonnal|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-pipengine/0.9.7|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-pipengine|
|version|0.9.7|
|sha|4d65c7a9a10e9fc7aa979616ab5cf5c9bd03368ca6f013f01403e8cc67170a9f|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-pipengine-0.9.7.gem|
|downloads|20776|
|name|bio-pipengine|
|version_downloads|1739|

</details>

## bio-signalp

|key|value|
|---|-----|
|info|A wrapper for the signal peptide prediction algorith SignalP. Not very well supported, but seems to work for the author, at least.
|url|http://github.com/wwood/bioruby-signalp|
|authors|Ben J Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-signalp/0.3.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-signalp|
|version|0.3.1|
|sha|ac9bb29395b47f98681945b22225b959ca346c3311f73c6b90db7c59798320a0|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-signalp-0.3.1.gem|
|downloads|20474|
|name|bio-signalp|
|version_downloads|3247|

</details>

## bio-velvet

|key|value|
|---|-----|
|info|Parser to work with some file formats used in the velvet DNA assembler
|url|http://github.com/wwood/bioruby-velvet|
|authors|Ben J Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-velvet/0.6.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-velvet|
|version|0.6.0|
|sha|319995af9a2ad99a717ee7b23856fdff2a83da8c0146c5aea1fc5df306c90560|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-velvet-0.6.0.gem|
|downloads|20070|
|name|bio-velvet|
|version_downloads|3073|

</details>

## bio-phyta

|key|value|
|---|-----|
|info|Pipeline to remove contaminations from EST libraries
|url|https://github.com/PalMuc/bio-phyta|
|authors|Philipp Comans|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-phyta/0.9.4|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-phyta|
|version|0.9.4|
|sha|9b88babd6666c381c935b69993374054b4799d5dcabf92785e7c25519abee786|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-phyta-0.9.4.gem|
|downloads|19355|
|name|bio-phyta|
|version_downloads|4461|

</details>

## bio-sambamba

|key|value|
|---|-----|
|info|New Sambamba library comes with a command-line tool for working with SAM/BAM files. This gem brings some of its functionality to Ruby.
|url|http://github.com/lomereiter/bioruby-sambamba|
|authors|Artem Tarasov|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-sambamba/0.0.5|
|metadata|{}|
|bug_tracker_uri||
|project_uri|https://rubygems.org/gems/bio-sambamba|
|version|0.0.5|
|sha|3d3d150fe791207a788d2bcdbca6e0e9ea66010e8af9fd784fecaa14335429de|
|platform|ruby|
|source_code_uri||
|gem_uri|https://rubygems.org/gems/bio-sambamba-0.0.5.gem|
|downloads|17534|
|mailing_list_uri||
|name|bio-sambamba|
|wiki_uri|https://github.com/lomereiter/bioruby-sambamba/wiki/Usage|
|version_downloads|4111|

</details>

## bio-polyploid-tools

|key|value|
|---|-----|
|info|Repository of tools developed at Crop Genetics in JIC to work with polyploid wheat
|url|http://github.com/tgac/bioruby-polyploid-tools|
|authors|Ricardo H.  Ramirez-Gonzalez|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-polyploid-tools/1.2.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-polyploid-tools|
|version|1.2.1|
|sha|4c315f8691200db1e5309fdc5caff9257283bc7b701e8ea54abbb67971351a2e|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-polyploid-tools-1.2.1.gem|
|downloads|84764|
|name|bio-polyploid-tools|
|version_downloads|1546|

</details>

## bio-commandeer

|key|value|
|---|-----|
|info|A dead simple method of running shell commands from within Ruby, by applying opinion
|url|http://github.com/wwood/bioruby-commandeer|
|authors|Ben J. Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-commandeer/0.4.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-commandeer|
|version|0.4.0|
|sha|d1522ba5744d0213da53834558d41d8cb0e5c598f3768baa688316ea0377174e|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-commandeer-0.4.0.gem|
|downloads|16537|
|name|bio-commandeer|
|version_downloads|2293|

</details>

## bio-assembly

|key|value|
|---|-----|
|info|bioruby plugin to parse, write, and manipulate assembly data
|url|http://github.com/chmille4/bioruby-assembly|
|authors|Chase Miller, Francesco Strozzi|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-assembly/0.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-assembly|
|version|0.1.0|
|sha|8d9dabf24c48a4216d6d9bd3b74d742b8e942bb9b0008ccf3263d7b07a48d3d4|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-assembly-0.1.0.gem|
|downloads|16327|
|name|bio-assembly|
|version_downloads|4878|

</details>

## bio-nexml

|key|value|
|---|-----|
|info|This plugin reads, writes and generates NeXML
|url|https://github.com/nexml/bio-nexml|
|authors|Rutger Vos, Anurag Priyam|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-nexml/1.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-nexml|
|version|1.1.0|
|sha|4483e3e64a0cfee90f1c338d4a540b2b7054e0c4f2a7e5728b0a00b26dd2df80|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-nexml-1.1.0.gem|
|downloads|15680|
|name|bio-nexml|
|version_downloads|4158|

</details>

## bio-graphics

|key|value|
|---|-----|
|info|Library for visualizing genomic regions
|url|http://bio-graphics.rubyforge.org/|
|authors|Jan Aerts|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-graphics/1.4|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-graphics|
|version|1.4|
|sha|61fffa7c87451771d6f1c3381c5c21d193ec2d3b9cfacce23aeee61537fd03da|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-graphics-1.4.gem|
|downloads|13337|
|name|bio-graphics|
|version_downloads|5241|

</details>

## bio-tabix

|key|value|
|---|-----|
|info|Tabix file indexing routines from the samtools package http://samtools.sourceforge.net/
|url|http://github.com/throwern/bio-tabix|
|authors|throwern|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|http://rubydoc.info/github/throwern/bio-tabix/frames|
|metadata|{}|
|bug_tracker_uri||
|project_uri|https://rubygems.org/gems/bio-tabix|
|version|1.0.1|
|sha|2d948601f6b982933e8d04eb159efe9a38fd0147a304c3a940f6c592c6777bef|
|platform|ruby|
|source_code_uri||
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-tabix-1.0.1.gem|
|downloads|13076|
|mailing_list_uri||
|name|bio-tabix|
|wiki_uri||
|version_downloads|3560|

</details>

## bio-ucsc-api

|key|value|
|---|-----|
|info|Ruby UCSC API: accessing the UCSC Genome Database using Ruby
|url|http://github.com/misshie/bioruby-ucsc-api|
|authors|Hiroyuki Mishima, Jan Aerts|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://github.com/misshie/bioruby-ucsc-api|
|metadata|{}|
|bug_tracker_uri|https://github.com/misshie/bioruby-ucsc-api|
|project_uri|https://rubygems.org/gems/bio-ucsc-api|
|version|0.6.5|
|sha|c26294c1a6d1335cd2769761ab5513d49660b25fa2a618a37f40abe47d76d174|
|platform|ruby|
|source_code_uri|http://github.com/misshie/bioruby-ucsc-api|
|licenses|["The MIT License"]|
|gem_uri|https://rubygems.org/gems/bio-ucsc-api-0.6.5.gem|
|downloads|61410|
|mailing_list_uri||
|name|bio-ucsc-api|
|wiki_uri||
|version_downloads|1577|

</details>

## bio-octopus

|key|value|
|---|-----|
|info|Running and parsing of the protein transmembrane domain predictor octopus
|url|http://github.com/wwood/bioruby-octopus|
|authors|Ben J Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-octopus/0.1.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-octopus|
|version|0.1.1|
|sha|27259c24217cc622dabf583d3b6bc46e2e144e51ba971ffaf5d561c279875311|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-octopus-0.1.1.gem|
|downloads|12486|
|name|bio-octopus|
|version_downloads|4626|

</details>

## bio-plasmoap

|key|value|
|---|-----|
|info|The PlasmoAP algorithm (Foth and Ralph et. al. 2003) predicts apicoplast transit peptides in Plasmodium falciparum. This is an implementation of that algorithm.
|url|http://github.com/wwood/bioruby-plasmoap|
|authors|Ben J Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-plasmoap/0.1.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-plasmoap|
|version|0.1.1|
|sha|e5ecf8f70c03b27749c4aa87d821ffcd66d08106c644b2781798ce03be89a953|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-plasmoap-0.1.1.gem|
|downloads|12379|
|name|bio-plasmoap|
|version_downloads|2762|

</details>

## bio-bwa

|key|value|
|---|-----|
|info|Ruby binding for BWA mapping software, built using Ruby-FFI library
|url|http://github.com/fstrozzi/bioruby-bwa|
|authors|Francesco Strozzi|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|http://fstrozzi.github.com/bioruby-bwa/|
|metadata|{}|
|bug_tracker_uri|https://github.com/fstrozzi/bioruby-bwa/issues|
|project_uri|https://rubygems.org/gems/bio-bwa|
|version|0.2.2|
|sha|a02efbf0dde45164c47634a736d42c355cdb8f5af101deca19fd70f1281c2101|
|platform|ruby|
|source_code_uri|https://github.com/fstrozzi/bioruby-bwa|
|gem_uri|https://rubygems.org/gems/bio-bwa-0.2.2.gem|
|downloads|11915|
|mailing_list_uri||
|name|bio-bwa|
|wiki_uri|https://github.com/fstrozzi/bioruby-bwa/wiki|
|version_downloads|4694|

</details>

## bio-lazyblastxml

|key|value|
|---|-----|
|info|This is very scrappy at the moment, and will need to be seriously cleaned up. It does what I need it to do for now. I'll fix it up in the coming weeks. Promise :)
|url|http://github.com/robsyme/bioruby-lazyblastxml|
|authors|robsyme|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-lazyblastxml/0.4.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-lazyblastxml|
|version|0.4.0|
|sha|9e574f6e5fe17bba5d3f8e37e9cf2ae3144529f5a8a298a79e631ac982c01e24|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-lazyblastxml-0.4.0.gem|
|downloads|11538|
|name|bio-lazyblastxml|
|version_downloads|4430|

</details>

## bio-krona

|key|value|
|---|-----|
|info|This biogem is built around Krona, a flashy way of representing hierarchical data.
|url|http://github.com/wwood/bioruby-krona|
|authors|Ben J. Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-krona/0.2.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-krona|
|version|0.2.0|
|sha|7945bd027e891d51b12c7b22ad3895803d9699b25bca6b678793f3aa06668c06|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-krona-0.2.0.gem|
|downloads|11473|
|name|bio-krona|
|version_downloads|2449|

</details>

## bio-gngm

|key|value|
|---|-----|
|info|Identify causative mutations in a model genome from NGS reads using the NGM method.
|url|http://github.com/danmaclean/bioruby-gngm|
|authors|Dan MacLean|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-gngm/0.2.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-gngm|
|version|0.2.1|
|sha|904ca6413bf375f4ed70b2e6936086e4e4a45d7f420556d0bb1514a377f0d622|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-gngm-0.2.1.gem|
|downloads|10868|
|name|bio-gngm|
|version_downloads|4114|

</details>

## bio-liftover

|key|value|
|---|-----|
|info|Simple, under development Ruby solution for UCSC LiftOver tool
|url|http://github.com/andreirozanski/bioruby-liftover|
|authors|Andrei Rozanski|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://github.com/andreirozanski/bioruby-liftover|
|metadata|{}|
|bug_tracker_uri|https://github.com/andreirozanski/bioruby-liftover|
|project_uri|https://rubygems.org/gems/bio-liftover|
|version|1.3.0|
|sha|6233217f46b0ef8a9733979954a47879b41023ee53a01305c872250f393ff147|
|platform|ruby|
|source_code_uri|https://github.com/andreirozanski/bioruby-liftover|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-liftover-1.3.0.gem|
|downloads|10543|
|mailing_list_uri|https://github.com/andreirozanski/bioruby-liftover|
|name|bio-liftover|
|wiki_uri|https://github.com/andreirozanski/bioruby-liftover|
|version_downloads|3161|

</details>

## bio-chembl

|key|value|
|---|-----|
|info|BioRuby plugin for ChEMBL REST API Client, Parser and Container classes
|url|http://github.com/nakao/bio-chembl|
|authors|Mitsuteru Nakao|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-chembl/0.1.3|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-chembl|
|version|0.1.3|
|sha|f9eed2f903e6e4e44e1cfc788df01eb341b8c40ff421aa8caf82fa9ded7cd7e0|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-chembl-0.1.3.gem|
|downloads|10511|
|name|bio-chembl|
|version_downloads|4054|

</details>

## bio-rdf

|key|value|
|---|-----|
|info|Store relationships in a triple-store, such as gene enrichment and QTL, and use this information for inference and causality
|url|http://github.com/pjotrp/bioruby-rdf|
|authors|Pjotr Prins|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-rdf/0.0.2|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-rdf|
|version|0.0.2|
|sha|3dc4062d5dcf01889b77f037337c67cb0801225317046379e129e76e2be2179a|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-rdf-0.0.2.gem|
|downloads|9955|
|name|bio-rdf|
|version_downloads|4334|

</details>

## bio-affy

|key|value|
|---|-----|
|info|Affymetrix microarray file format parser
  (CEL/CDF) for Ruby. FFI binding to Biolib port of R/Affyio by Benjamin Milo Bolstad
|url|http://github.com/pjotrp/bioruby-affy|
|authors|Pjotr Prins|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-affy/0.5.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-affy|
|version|0.5.1|
|sha|d95b77ca55ae70accd9aaeacf7883e3396663bc2c0a3c94c9fde1c9dcf2d911b|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-affy-0.5.1.gem|
|downloads|9898|
|name|bio-affy|
|version_downloads|4386|

</details>

## bio-unichem

|key|value|
|---|-----|
|info|BioRuby plugin for UniChem REST Web service
|url|http://github.com/nakao/bioruby-unichem|
|authors|Mitsuteru Nakao|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-unichem/0.3.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-unichem|
|version|0.3.0|
|sha|cc8fb5c73615db2b9b56e415d1e59d705e8aaed62ac7b9d995fdadce2c6a6f10|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-unichem-0.3.0.gem|
|downloads|9664|
|name|bio-unichem|
|version_downloads|3768|

</details>

## bio-synreport

|key|value|
|---|-----|
|info|Takes a GFF and genomic sequence file, constructs CDS and when given a position and alternative base will report whether this change is in a coding region and if it results in a synonymous or non-synonymous mutation.
|url|http://github.com/danmaclean/bioruby-synreport|
|authors|Dan MacLean|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-synreport/0.1.2|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-synreport|
|version|0.1.2|
|sha|fc53271e6e043a9c6b8c2d4ed7e463afc57442f37d62e7a999479dc81eb623f6|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-synreport-0.1.2.gem|
|downloads|9576|
|name|bio-synreport|
|version_downloads|3627|

</details>

## bio-sra

|key|value|
|---|-----|
|info|A Sequence Read Archive (SRA) download script and Ruby interface to the SRAdb (SRA metadata) SQLite database.
|url|http://github.com/wwood/bioruby-sra|
|authors|Ben J. Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-sra/0.3.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-sra|
|version|0.3.1|
|sha|6ddc7b16742882bedacf288135bb171ea08c59aa9de9e8e01c38b5565969ca5e|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-sra-0.3.1.gem|
|downloads|8763|
|name|bio-sra|
|version_downloads|2437|

</details>

## bio-protparam

|key|value|
|---|-----|
|info|Bio::Protparam has same interface and function as Bio::Tools::Protparam class of BioPerl, except that it calculate parameters instead of throwing query to Expasy protparam tool.
|url|http://github.com/hryk/bioruby-protparam|
|authors|hryk|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|http://rdoc.info/github/hryk/bioruby-protparam/|
|metadata|{}|
|bug_tracker_uri|https://github.com/hryk/bioruby-protparam/issues|
|project_uri|https://rubygems.org/gems/bio-protparam|
|version|0.2.0|
|sha|92a08b8c08f843dac4eaa7ecc23ae7b220680050a1f1c7b2699ac7561ffbf028|
|platform|ruby|
|source_code_uri|https://github.com/hryk/bioruby-protparam|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-protparam-0.2.0.gem|
|downloads|8662|
|mailing_list_uri||
|name|bio-protparam|
|wiki_uri||
|version_downloads|5125|

</details>

## bio-locus

|key|value|
|---|-----|
|info|A tool for fast querying and filtering of genome locations in VCF and other formats
|url|http://github.com/pjotrp/bio-locus|
|authors|Pjotr Prins|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-locus/0.0.7|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-locus|
|version|0.0.7|
|sha|803aa9e6e68b2147df0ec0878fd76ae6ef552d74f093aba1d249ce1a7dcf750b|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-locus-0.0.7.gem|
|downloads|8133|
|name|bio-locus|
|version_downloads|3365|

</details>

## bio-core

|key|value|
|---|-----|
|info|Bioruby core plugins
|url|http://github.com/helios/bioruby-core|
|authors|Raoul J.P. Bonnal|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-core/0.0.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-core|
|version|0.0.1|
|sha|22c49e1abd9ffcdbf2ac5f863c7a80c409148000aca8ea05aa2aa9e2f8ebdba8|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-core-0.0.1.gem|
|downloads|8057|
|name|bio-core|
|version_downloads|4498|

</details>

## bio-basespace-sdk

|key|value|
|---|-----|
|info|BaseSpace Ruby SDK is a Ruby based Software Development Kit to be used in the development of Apps and scripts for working with Illumina's BaseSpace cloud-computing solution for next-gen sequencing data analysis.

  The primary purpose of the SDK is to provide an easy-to-use Ruby environment enabling developers to authenticate a user, retrieve data, and upload data/results from their own analysis to BaseSpace.
|url|https://github.com/basespace/basespace-ruby-sdk|
|authors|Joachim Baran, Raoul Bonnal, Eri Kibukawa, Francesco Strozzi, Toshiaki Katayama|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-basespace-sdk/0.1.7|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-basespace-sdk|
|version|0.1.7|
|sha|d5583fd60134726d96e6569b360f62bc7c20b42d8f789dddfbbed98849cf576b|
|platform|ruby|
|licenses|["Apache, Version 2.0"]|
|gem_uri|https://rubygems.org/gems/bio-basespace-sdk-0.1.7.gem|
|downloads|35443|
|name|bio-basespace-sdk|
|version_downloads|24257|

</details>

## bio-gag

|key|value|
|---|-----|
|info|bio-gag is a biogem for detecting and correcting a particular type of error that occurs/occurred in particular versions of the IonTorrent DNA sequencing kit. Recent versions of the system don't appear to suffer the same problem
|url|http://github.com/wwood/bioruby-gag|
|authors|Ben J Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-gag/0.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-gag|
|version|0.1.0|
|sha|1486d238e7597a109a5a31046ae31d2ba7f1e738857b416003773d47d276c7d0|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-gag-0.1.0.gem|
|downloads|7308|
|name|bio-gag|
|version_downloads|3919|

</details>

## bio-mummer

|key|value|
|---|-----|
|info|Help for working with the output of the .delta files produced by nucmer and promer
|url|http://github.com/robsyme/bioruby-mummer|
|authors|robsyme|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-mummer/0.2.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-mummer|
|version|0.2.1|
|sha|b2514d066e64132396be74c2d50ca775ca9f68e0a8229c30a825f3327a338445|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-mummer-0.2.1.gem|
|downloads|6936|
|name|bio-mummer|
|version_downloads|2748|

</details>

## bio-gggenome

|key|value|
|---|-----|
|info|A Ruby client for GGGenome the Ultrafast sequence search
|url|http://github.com/nakao/bio-gggenome|
|authors|Mitsuteru Nakao|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-gggenome/0.2.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-gggenome|
|version|0.2.0|
|sha|7b9c78fab936aa1feb258d882c73bf1c3a9bc19182a48d22689861c55df65dc4|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-gggenome-0.2.0.gem|
|downloads|6322|
|name|bio-gggenome|
|version_downloads|3540|

</details>

## bio-ipcress

|key|value|
|---|-----|
|info|a programmatic interface to the iPCRess in-silico PCR software. iPCRess is part of the exonerate suite.
|url|http://github.com/wwood/bioruby-ipcress|
|authors|Ben J Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-ipcress/0.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-ipcress|
|version|0.1.0|
|sha|43063aaaef86699430de26b9d9e439c43a32a5ee966c9561ccb5a24e097c377d|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-ipcress-0.1.0.gem|
|downloads|5899|
|name|bio-ipcress|
|version_downloads|2139|

</details>

## bio-repeatmasker

|key|value|
|---|-----|
|info|A simple gem to parse, filter/search raw Repeat Masker output file from GoldenPath
|url|http://rubygems.org/gems/bio-repeatmasker|
|authors|Andrei Rozanski|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://github.com/andreirozanski/bio-repeatmasker|
|metadata|{}|
|bug_tracker_uri|https://github.com/andreirozanski/bio-repeatmasker|
|project_uri|https://rubygems.org/gems/bio-repeatmasker|
|version|0.0.3|
|sha|da491643978670fa887460ce29e8bab1afd4e36c7f65b6fa0b02a39998c061db|
|platform|ruby|
|source_code_uri|https://github.com/andreirozanski/bio-repeatmasker|
|licenses|[]|
|gem_uri|https://rubygems.org/gems/bio-repeatmasker-0.0.3.gem|
|downloads|5871|
|mailing_list_uri||
|name|bio-repeatmasker|
|wiki_uri|https://github.com/andreirozanski/bio-repeatmasker|
|version_downloads|3273|

</details>

## bio-cigar

|key|value|
|---|-----|
|info|A parser for CIGAR format alignments.
|url|http://github.com/wwood/bioruby-cigar|
|authors|Ben J Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-cigar/0.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-cigar|
|version|0.1.0|
|sha|49c930e019d421f9013cbb1809a2663c11f4784733e2da362b19404029b1d83f|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-cigar-0.1.0.gem|
|downloads|5757|
|name|bio-cigar|
|version_downloads|3289|

</details>

## bio-dbla-classifier

|key|value|
|---|-----|
|info|Methods to classify and manipulate PfEMP1 DBL-alpha sequence tags
|url|http://github.com/georgeG/bioruby-dbla-classifier|
|authors|George Githinji|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-dbla-classifier/0.7.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-dbla-classifier|
|version|0.7.1|
|sha|be5f1c68997909ec02ade7697585f98c29fb802752c3dbf3f40a75a57034c57e|
|platform|ruby|
|licenses|["Ruby"]|
|gem_uri|https://rubygems.org/gems/bio-dbla-classifier-0.7.1.gem|
|downloads|21323|
|name|bio-dbla-classifier|
|version_downloads|3726|

</details>

## bio-tm_hmm

|key|value|
|---|-----|
|info|A bioruby plugin for interaction with the transmembrane predictor TMHMM
|url|http://github.com/wwood/bioruby-tm_hmm|
|authors|Ben J. Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-tm_hmm/0.2.4|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-tm_hmm|
|version|0.2.4|
|sha|2b02a4377453acf6acbba02e7149424ede1979f804e1b3294168bc48c6dbc1f0|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-tm_hmm-0.2.4.gem|
|downloads|20797|
|name|bio-tm_hmm|
|version_downloads|1884|

</details>

## bio-shell

|key|value|
|---|-----|
|info|BioRuby Shell is a command line interface on BioRuby open source bioinformatics library. It provides easy-to-use analysis environment for bioinformatics.
|url|https://github.com/bioruby/bioruby-shell|
|authors|BioRuby project|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-shell/1.0.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-shell|
|version|1.0.1|
|sha|0adddaab0e1579940820829b777d0f567859e914c2bb2feca3d6c9b12dfbe390|
|platform|ruby|
|licenses|[]|
|gem_uri|https://rubygems.org/gems/bio-shell-1.0.1.gem|
|downloads|4917|
|name|bio-shell|
|version_downloads|991|

</details>

## bio-gfastqc

|key|value|
|---|-----|
|info|Bioinformatics. Aggregate FastQC (quality control for Next Generation Sequencing -NGS-) results from many different samples in a single web page, with charts and tables organized and simplified. The main goal is to speed up the communication process with colleagues (PIs, Biologists, BioInformaticians).
|url|http://github.com/helios/bioruby-gfastqc|
|authors|Raoul Jean Pierre Bonnal|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-gfastqc/0.0.4|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-gfastqc|
|version|0.0.4|
|sha|66fe6425db289603d24533388b8c6734ca4de5a8843dbb817c6bbde4d5f71815|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-gfastqc-0.0.4.gem|
|downloads|4803|
|name|bio-gfastqc|
|version_downloads|2663|

</details>

## bio-sge

|key|value|
|---|-----|
|info|Entries in a flatfile will be parased by the BioRuby's Bio::FlatFile.auto module. These entries are used as queries for the Sun Grid Engine (SGE) system. Huge amount of queries are automatically splitted into subdirectories. With a specified command line to be executed, queries are submited to the SGE as an array job.
|url|http://github.com/ktym/bioruby-sge|
|authors|Toshiaki Katayama|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-sge/0.0.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-sge|
|version|0.0.0|
|sha|8478f29051e2d56dcf7c795ec1e1efba8a3840f393e326fdd216accf9a9600c1|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-sge-0.0.0.gem|
|downloads|4762|
|name|bio-sge|
|version_downloads|4762|

</details>

## bio-kmer_counter

|key|value|
|---|-----|
|info|A biogem for counting small kmers for fingerprinting nucleotide sequences. See README for details.
|url|http://github.com/wwood/bioruby-kmer_counter|
|authors|Ben J Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-kmer_counter/0.1.2|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-kmer_counter|
|version|0.1.2|
|sha|fdc8fd8bea74a468545f05d9b12375f65657dffcffd7c1dd25ee2af423b27826|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-kmer_counter-0.1.2.gem|
|downloads|19565|
|name|bio-kmer_counter|
|version_downloads|3739|

</details>

## bio-freebayes

|key|value|
|---|-----|
|info|The API for Freebayes version #{Bio::Freebayes::EXTVERSION}
|url|http://rubygems.org/gems/bio-freebayes|
|authors|Natapol Pornputtapong|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-freebayes/0.1.6|
|metadata|{"allowed_push_host"=>"TODO: Set to 'http://mygemserver.com'"}|
|project_uri|https://rubygems.org/gems/bio-freebayes|
|version|0.1.6|
|sha|2126b998266da18cc7a9b111c907beadc803fa7d3cd99fd38e8273e31f30f146|
|platform|ruby|
|licenses|[]|
|gem_uri|https://rubygems.org/gems/bio-freebayes-0.1.6.gem|
|downloads|4734|
|name|bio-freebayes|
|version_downloads|2705|

</details>

## bio-hello

|key|value|
|---|-----|
|info|Encode any message string into a DNA sequence
|url|http://github.com/ktym/bioruby-hello|
|authors|Toshiaki Katayama|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-hello/0.0.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-hello|
|version|0.0.0|
|sha|3d2ad0c17e1a407b57e49d456f02f630ef129bac19d412e565be3279d2a4ca3c|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-hello-0.0.0.gem|
|downloads|4686|
|name|bio-hello|
|version_downloads|4686|

</details>

## bio-biolinux

|key|value|
|---|-----|
|info|Meta package for BioLinux distribution: bio-core, bio-core-ext
|url|http://github.com/helios/bioruby-biolinux|
|authors|Raoul J.P. Bonnal|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-biolinux/0.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-biolinux|
|version|0.1.0|
|sha|6bfa59585fd0701cdb311c221ccafbcdfa2bb73bb61757640f6242fe5ebb0d95|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-biolinux-0.1.0.gem|
|downloads|4412|
|name|bio-biolinux|
|version_downloads|4412|

</details>

## bio-ensembl

|key|value|
|---|-----|
|info|This API provides a complete set of methods and classes to access the Ensembl database using Ruby programming language
|url|http://github.com/fstrozzi/bioruby-ensembl|
|authors|Jan Aerts, Francesco Strozzi|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-ensembl/1.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-ensembl|
|version|1.1.0|
|sha|1a8037bfc6030278c8be07b9ee51af40e2901d91a16f4cd86298c1e7e4ffeac0|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-ensembl-1.1.0.gem|
|downloads|4390|
|name|bio-ensembl|
|version_downloads|4390|

</details>

## bio-dbsnp

|key|value|
|---|-----|
|info|decoding the dbSNP bitfield containg detaild information
|url|http://github.com/misshie/bioruby-dbsnp|
|authors|Hiroyuki Mishima|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-dbsnp/0.1.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-dbsnp|
|version|0.1.1|
|sha|4cefcd706da46d83a743ed9b919726e41fa596a07915350ed7f346ce49bf5cf5|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-dbsnp-0.1.1.gem|
|downloads|4347|
|name|bio-dbsnp|
|version_downloads|4347|

</details>

## bio-foo

|key|value|
|---|-----|
|info|longer description of your gem
|url|http://github.com/ktym/bioruby-foo|
|authors|Toshiaki Katayama|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-foo/0.0.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-foo|
|version|0.0.0|
|sha|7c73173934faff49a0e66373bde467986c5e8cf6544dd0b75b6c68b3c553cac1|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-foo-0.0.0.gem|
|downloads|4332|
|name|bio-foo|
|version_downloads|4332|

</details>

## bio-hydropathy

|key|value|
|---|-----|
|info|Hydropathy scale for BioRuby
|url|http://github.com/wwood/bioruby-hydropathy|
|authors|Ben J Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-hydropathy/0.0.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-hydropathy|
|version|0.0.1|
|sha|3534b79494d849022b446336d38cf130dd10bb4285cd1b6880bbb477b4b5c003|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-hydropathy-0.0.1.gem|
|downloads|4309|
|name|bio-hydropathy|
|version_downloads|4309|

</details>

## bio-phyloxml

|key|value|
|---|-----|
|info|Provides PhyloXML support for BioRuby.
|url|http://github.com/csw/bioruby-phyloxml|
|authors|Diana Jaunzeikare, Clayton Wheeler|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-phyloxml/0.9.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-phyloxml|
|version|0.9.0|
|sha|d56ac15a573582782ca25d697a839345c832a201180fc4a89d4cdacdf3662e8d|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-phyloxml-0.9.0.gem|
|downloads|4286|
|name|bio-phyloxml|
|version_downloads|4286|

</details>

## bio-exportpred

|key|value|
|---|-----|
|info|Wrapper around the ExportPred algorithm for predicting P. falciparum exported proteins. Requires local install of the program, which is available from http://bioinf.wehi.edu.au/exportpred/
|url|http://github.com/wwood/bioruby-exportpred|
|authors|Ben J Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-exportpred/0.0.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-exportpred|
|version|0.0.1|
|sha|49439427fb5c0e33eb43d6e71f067d87bd5693911b8cdbe54fb61969d4050b3d|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-exportpred-0.0.1.gem|
|downloads|4271|
|name|bio-exportpred|
|version_downloads|4271|

</details>

## bio-incanter

|key|value|
|---|-----|
|info|A Jruby wrapper to Incanter lib
|url|http://github.com/cerbero/bio-incanter|
|authors|Matteo Picciolini|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://github.com/cerbero/bioruby-incanter/tree/master/doc|
|metadata|{}|
|bug_tracker_uri||
|project_uri|https://rubygems.org/gems/bio-incanter|
|version|0.1.0|
|sha|81e1a2c905612b52eaf46280ef520cd07eef9352a655caff5621a744065979ed|
|platform|ruby|
|source_code_uri|https://github.com/cerbero/bioruby-incanter|
|gem_uri|https://rubygems.org/gems/bio-incanter-0.1.0.gem|
|downloads|4207|
|mailing_list_uri||
|name|bio-incanter|
|wiki_uri|https://github.com/cerbero/bioruby-incanter/wiki|
|version_downloads|4207|

</details>

## bio-fasta_retrieve

|key|value|
|---|-----|
|info|Gem that helps to retrieve DNA sequence from UCSC
|url|http://rubygems.org/gems/bio-fasta_retrieve.rb|
|authors|Andrei Rozanski|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://github.com/andreirozanski/bio-fasta_retrieve|
|metadata|{}|
|bug_tracker_uri|https://github.com/andreirozanski/bio-fasta_retrieve|
|project_uri|https://rubygems.org/gems/bio-fasta_retrieve|
|version|0.0.7|
|sha|c1b648653f1820e044a5637c1af6af469b002a035bb677776d47d1ccc67ead04|
|platform|ruby|
|source_code_uri|https://github.com/andreirozanski/bio-fasta_retrieve|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-fasta_retrieve-0.0.7.gem|
|downloads|16035|
|mailing_list_uri|https://github.com/andreirozanski/bio-fasta_retrieve|
|name|bio-fasta_retrieve|
|wiki_uri|https://github.com/andreirozanski/bio-fasta_retrieve|
|version_downloads|2593|

</details>

## bio-isoelectric_point

|key|value|
|---|-----|
|info|A bioruby plugin for calculating the isoelectric point of a protein 
|url|http://github.com/georgeG/bioruby-isoelectric_point|
|authors|George Githinji, Pascal Betz|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-isoelectric_point/0.1.3|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-isoelectric_point|
|version|0.1.3|
|sha|744f197fb06803c78d41dc6dbab1aec14070d059ab6dff0c778b2d32a9875659|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-isoelectric_point-0.1.3.gem|
|downloads|15937|
|name|bio-isoelectric_point|
|version_downloads|4784|

</details>

## bio-executables

|key|value|
|---|-----|
|info|BioRuby executables is a collection of miscellaneous utilities for bioinformatics that were formerly part of the BioRuby core ("bio" gem). Because of reducing complexity, they were moved to this gem.
|url|https://github.com/bioruby/bioruby-executables|
|authors|BioRuby project|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-executables/1.0.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-executables|
|version|1.0.0|
|sha|29f02115bc543d7f39b0c6d4c518719dcae9d112a913c215b988b599a6736b91|
|platform|ruby|
|licenses|[]|
|gem_uri|https://rubygems.org/gems/bio-executables-1.0.0.gem|
|downloads|3879|
|name|bio-executables|
|version_downloads|2338|

</details>

## bio-pangenome

|key|value|
|---|-----|
|info|Tools to find similarity between pangenomes.
|url|http://github.com/Uauy-Lab/bioruby-pangenome|
|authors|Ricardo H. Ramirez-Gonzalez|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-pangenome/0.1.3|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-pangenome|
|version|0.1.3|
|sha|325b2b4fbb37d0709d422a53160cf2ed93a190103ca53773ab2fe92b63d9974d|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-pangenome-0.1.3.gem|
|downloads|3758|
|name|bio-pangenome|
|version_downloads|1302|

</details>

## bio-stockholm

|key|value|
|---|-----|
|info|Parses stockholm sequence alignment format
|url|http://github.com/wwood/bioruby-stockholm|
|authors|Ben J. Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-stockholm/0.0.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-stockholm|
|version|0.0.1|
|sha|c6407901f14090d5fcaf805a2f02d399755bfb106debbc908247438fec49d8e7|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-stockholm-0.0.1.gem|
|downloads|3739|
|name|bio-stockholm|
|version_downloads|3739|

</details>

## bio-kseq

|key|value|
|---|-----|
|info|A fast FASTA/FASTQ parser based on kseq.h by Heng Li
|url|https://github.com/gusevfe/bio-kseq|
|authors|Fedor Gusev|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-kseq/0.0.2|
|metadata|{}|
|bug_tracker_uri|https://github.com/gusevfe/bio-kseq/issues|
|project_uri|https://rubygems.org/gems/bio-kseq|
|version|0.0.2|
|sha|3fcfd82492a0c59fd906b6c6251361ea290908386ecdcf0d9365c1698882caf7|
|platform|ruby|
|source_code_uri|https://github.com/gusevfe/bio-kseq|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-kseq-0.0.2.gem|
|downloads|3684|
|mailing_list_uri||
|name|bio-kseq|
|wiki_uri||
|version_downloads|3677|

</details>

## bio-velvet_underground

|key|value|
|---|-----|
|info|Bindings to some internals of the velvet assembler.
|url|http://github.com/wwood/bioruby-velvet_underground|
|authors|Ben Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-velvet_underground/0.3.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-velvet_underground|
|version|0.3.1|
|sha|1fd95b6a66e8308041d8a68c5094961d5b01fb7e3892a1d80de08464cdc1d7f1|
|platform|ruby|
|licenses|["GPL"]|
|gem_uri|https://rubygems.org/gems/bio-velvet_underground-0.3.1.gem|
|downloads|14321|
|name|bio-velvet_underground|
|version_downloads|2201|

</details>

## bio-raxml

|key|value|
|---|-----|
|info|Just a test
|url|http://github.com/fizquierdo/bioruby-raxml|
|authors|Fernando Izquierdo|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-raxml/0.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-raxml|
|version|0.1.0|
|sha|6d57e168d15bf1265bffb37a0b0ffd5ade3f1acc3b2641fd6b4930ee36f3224d|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-raxml-0.1.0.gem|
|downloads|3611|
|name|bio-raxml|
|version_downloads|3611|

</details>

## bioruby-bio

|key|value|
|---|-----|
|info|BioRuby is a library for bioinformatics (biology + information science).
|url|http://bioruby.org/|
|authors|BioRuby project|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bioruby-bio/1.3.1.5000|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bioruby-bio|
|version|1.3.1.5000|
|sha|6f8bf3defd75fd58f6943073f42216a841d2fa05090174e5a7428e2491dc326e|
|platform|ruby|
|licenses|[]|
|gem_uri|https://rubygems.org/gems/bioruby-bio-1.3.1.5000.gem|
|downloads|15203|
|name|bioruby-bio|
|version_downloads|3174|

</details>

## bio-genomic-interval

|key|value|
|---|-----|
|info|a BioRuby plugin: handling genomic intervals,such as "chr1:123-456", and overlap state between two intervals 
|url|http://github.com/misshie/bioruby-genomic-interval|
|authors|Hiroyuki Mishima|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://github.com/misshie/bioruby-genomic-interval|
|metadata|{}|
|bug_tracker_uri||
|project_uri|https://rubygems.org/gems/bio-genomic-interval|
|version|0.1.2|
|sha|deb647f1e0fc960f3f417ee8f70d33a60ee033de0d14d0c34c700afc17392dc4|
|platform|ruby|
|source_code_uri|https://github.com/misshie/bioruby-genomic-interval|
|gem_uri|https://rubygems.org/gems/bio-genomic-interval-0.1.2.gem|
|downloads|13620|
|mailing_list_uri|http://lists.open-bio.org/mailman/listinfo/bioruby|
|name|bio-genomic-interval|
|wiki_uri|http://bioruby.open-bio.org/wiki/Plugins|
|version_downloads|6452|

</details>

## bio-nwk

|key|value|
|---|-----|
|info|Some codes for parsing newick tree
|url||
|authors|Sishuo Wang|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-nwk/0.1.4|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-nwk|
|version|0.1.4|
|sha|d32005e7e74a43acf247f528bcca2f54cbb74954b7fb1c17eeecfb5bf4d7a7d5|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-nwk-0.1.4.gem|
|downloads|3443|
|name|bio-nwk|
|version_downloads|293|

</details>

## bio-gemma-wrapper

|key|value|
|---|-----|
|info|GEMMA wrapper adds LOCO and permutation support. Also runs in parallel and caches K between runs with LOCO support
|url|https://github.com/genetics-statistics/gemma-wrapper|
|authors|Pjotr Prins|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-gemma-wrapper/0.99.6|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-gemma-wrapper|
|version|0.99.6|
|sha|e38e17aeace59620460d1afc224c2103e1a3ceedc069d52d9ce42689aa37006c|
|platform|ruby|
|licenses|["GPL3"]|
|gem_uri|https://rubygems.org/gems/bio-gemma-wrapper-0.99.6.gem|
|downloads|12868|
|name|bio-gemma-wrapper|
|version_downloads|639|

</details>

## bio-exominer

|key|value|
|---|-----|
|info|Parse publications for gene names in a fuzzy fashion
|url|http://github.com/pjotrp/bioruby-exominer|
|authors|Pjotr Prins|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-exominer/0.0.3|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-exominer|
|version|0.0.3|
|sha|bb144499e6843d15879f6d54e2f07d4bfdb6ef32c9366ec476feaecb37316878|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-exominer-0.0.3.gem|
|downloads|3214|
|name|bio-exominer|
|version_downloads|3214|

</details>

## bio-faidx

|key|value|
|---|-----|
|info|ools to work with fasta files, indexed with samtools. The initial releases depend on bio-samtools, but that will change
|url|http://github.com/homonecloco/bioruby-faidx|
|authors|homonecloco|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-faidx/0.0.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-faidx|
|version|0.0.1|
|sha|047dfca5dad4b2caa4f74963bb8e43188f9f0f84a6d38df2ccb78ab48a6455ba|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-faidx-0.0.1.gem|
|downloads|3058|
|name|bio-faidx|
|version_downloads|3058|

</details>

## bio-statsample-timeseries

|key|value|
|---|-----|
|info|Statsample-timeseries is an extension to Statsample. It incorporates helpful timeseries functions and modules like ARMA, ARIMA, acf, pacf, lags etc.
|url|http://github.com/AnkurGel/bioruby-statsample-timeseries|
|authors|Ankur Goel, Claudio Bustos|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-statsample-timeseries/0.2.1|
|metadata|{}|
|bug_tracker_uri|https://github.com/AnkurGel/bioruby-statsample-timeseries/issues|
|project_uri|https://rubygems.org/gems/bio-statsample-timeseries|
|version|0.2.1|
|sha|a26a9af1d7bb775f98800ab688e26a09972428604fee63d11dcdec8157b8e3ea|
|platform|ruby|
|source_code_uri|https://github.com/AnkurGel/bioruby-statsample-timeseries|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-statsample-timeseries-0.2.1.gem|
|downloads|11713|
|mailing_list_uri||
|name|bio-statsample-timeseries|
|wiki_uri||
|version_downloads|3538|

</details>

## bio-pileup_iterator

|key|value|
|---|-----|
|info|Iterate through a samtools pileup file
|url|http://github.com/wwood/bioruby-pileup_iterator|
|authors|Ben J. Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-pileup_iterator/0.0.4|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-pileup_iterator|
|version|0.0.4|
|sha|5bbaa3a1aa4d603e359ca318e0556d88d00d1219d3ce1eb26edc4ca8ee5251fe|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-pileup_iterator-0.0.4.gem|
|downloads|11381|
|name|bio-pileup_iterator|
|version_downloads|4068|

</details>

## bio-picard

|key|value|
|---|-----|
|info|API of Picard tool
|url|http://rubygems.org/gems/bio-picard|
|authors|Natapol Pornputtapong|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-picard/1.131.5|
|metadata|{"allowed_push_host"=>"TODO: Set to 'http://mygemserver.com'"}|
|project_uri|https://rubygems.org/gems/bio-picard|
|version|1.131.5|
|sha|5c773ba5829bd0db9ebd15b3b5494d38c4e538a76bce62058410fc2523cc0d27|
|platform|ruby|
|licenses|["GPLv3"]|
|gem_uri|https://rubygems.org/gems/bio-picard-1.131.5.gem|
|downloads|2902|
|name|bio-picard|
|version_downloads|2726|

</details>

## bio-gff3-pltools

|key|value|
|---|-----|
|info|Ruby wrapper for the gff3-pltools.
|url|http://mamarjan.github.com/gff3-pltools/|
|authors|Marjan Povolni|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-gff3-pltools/0.3.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-gff3-pltools|
|version|0.3.0|
|sha|06e6a16047a54c6c73c2344e49736716460ba93bd0b0aa5fbf5947b8ac070a4a|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-gff3-pltools-0.3.0.gem|
|downloads|10655|
|name|bio-gff3-pltools|
|version_downloads|4043|

</details>

## bio-ucsc-util

|key|value|
|---|-----|
|info|Ruby FFI binding of the ucsc kent src. Implemented utilities include: bigWigInfo, bigWigSummary and wigToBigWig
|url|http://github.com/throwern/bio-ucsc-util|
|authors|throwern|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-ucsc-util/0.1.2|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-ucsc-util|
|version|0.1.2|
|sha|30c8ef8dec1325488c8138d313591746338175c348bf42d0262ac066bd849045|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-ucsc-util-0.1.2.gem|
|downloads|10443|
|name|bio-ucsc-util|
|version_downloads|4049|

</details>

## bio-kallisto

|key|value|
|---|-----|
|info|Basic wrapper for the Kallistio Mapper
|url|http://github.com/homonecloco/bioruby-kallisto|
|authors|homonecloco|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-kallisto/0.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-kallisto|
|version|0.1.0|
|sha|57f668128b29f38ff47e976776c6edb544c0095482adcf7aec9fbd7c41a52b6c|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-kallisto-0.1.0.gem|
|downloads|2665|
|name|bio-kallisto|
|version_downloads|2665|

</details>

## bio-jaspar

|key|value|
|---|-----|
|info|Basic tools for parsing, searching, and comparing JASPAR motifs; Based on Bio.motifs module in Biopython
|url|http://github.com/wassermanlab/jaspar-bioruby|
|authors|Jessica Lee, Wasserman Lab|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-jaspar/0.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-jaspar|
|version|0.1.0|
|sha|5442d3b50b93f4ed198305c5a8c2c45dff6d59af8abe581e49d1fc9cf03fe3e9|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-jaspar-0.1.0.gem|
|downloads|2622|
|name|bio-jaspar|
|version_downloads|2622|

</details>

## bio-conduit

|key|value|
|---|-----|
|info|A pipeline creator for bioruby
|url|http://rubygems.org/gems/bio-conduit|
|authors|Natapol Pornputtapong|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-conduit/1.0.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-conduit|
|version|1.0.0|
|sha|9e5abceceb9bb9c33efd64bca2be4c2a4211e8573092aa3ad37793d7878d4829|
|platform|ruby|
|licenses|["GPL"]|
|gem_uri|https://rubygems.org/gems/bio-conduit-1.0.0.gem|
|downloads|2610|
|name|bio-conduit|
|version_downloads|2610|

</details>

## ngoto-bio

|key|value|
|---|-----|
|info|BioRuby is a library for bioinformatics (biology + information science).
|url|http://bioruby.org/|
|authors|BioRuby project|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/ngoto-bio/1.3.1.5000|
|metadata|{}|
|project_uri|https://rubygems.org/gems/ngoto-bio|
|version|1.3.1.5000|
|sha|a13d7d3b16b794f65f9c397d34a4e7ac13b9de613d9eccc0781639ec179470b1|
|platform|ruby|
|licenses|[]|
|gem_uri|https://rubygems.org/gems/ngoto-bio-1.3.1.5000.gem|
|downloads|9815|
|name|ngoto-bio|
|version_downloads|2552|

</details>

## bio-plates

|key|value|
|---|-----|
|info|Methods for handling multiwell plate annotations, includes ranges and quadrants
|url|http://github.com/stveep/bioruby-plates|
|authors|Steve Pettitt|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-plates/0.2.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-plates|
|version|0.2.0|
|sha|9b57b1fabcbea3877305393dce7ba91215460e8b96fe42f2448d28eae4d4106f|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-plates-0.2.0.gem|
|downloads|2420|
|name|bio-plates|
|version_downloads|2420|

</details>

## bio-kb-illumina

|key|value|
|---|-----|
|info|Provide access to the IlluminaHumanAnnotations. Database comes form Bioconductor's package
     lumiHumanIDMapping
|url|http://github.com/helios/bioruby-kb-illumina|
|authors|Raoul J.P. Bonnal|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-kb-illumina/0.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-kb-illumina|
|version|0.1.0|
|sha|443c9c2bc501c4de4e2cad7a1f2f4ff138ee3763d6857e37bffb7e69d4ad119a|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-kb-illumina-0.1.0.gem|
|downloads|8646|
|name|bio-kb-illumina|
|version_downloads|4712|

</details>

## bio-biostars-analytics

|key|value|
|---|-----|
|info|Ruby script for data-mining biostars.org using web-crawling techniques as well as utilizing the Biostars RESTful API. Statistical analysis requires R (http://www.r-project.org).
|url|http://github.com/joejimbo/bioruby-biostars-analytics|
|authors|Joachim Baran|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-biostars-analytics/0.2.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-biostars-analytics|
|version|0.2.0|
|sha|29221bef2457adf1ea672aafc072fee194ff56a2b23a1319d21ddf4b2afe9171|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-biostars-analytics-0.2.0.gem|
|downloads|8062|
|name|bio-biostars-analytics|
|version_downloads|3209|

</details>

## bio-cnls_screenscraper

|key|value|
|---|-----|
|info|Programmatic interface to the cNLS nuclear localisation signal prediction software
|url|http://github.com/wwood/bioruby-cnls_screenscraper|
|authors|Ben J. Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-cnls_screenscraper/0.3.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-cnls_screenscraper|
|version|0.3.0|
|sha|89bad4f77e94e30da20f430c879bdedddbfcc5cb238c3c3b726773dcaee2fd88|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-cnls_screenscraper-0.3.0.gem|
|downloads|7809|
|name|bio-cnls_screenscraper|
|version_downloads|4161|

</details>

## bio-biosql

|key|value|
|---|-----|
|info|Provides BioSQL support for BioRuby.
|url|http://github.com/bioruby/bioruby-biosql|
|authors|BioRuby project|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-biosql/0.0.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-biosql|
|version|0.0.0|
|sha|30a3449afc8b2d7d14c1b6f35d54389bc90224b62a20858a230e2e4e39b86cef|
|platform|ruby|
|licenses|["Ruby"]|
|gem_uri|https://rubygems.org/gems/bio-biosql-0.0.0.gem|
|downloads|2131|
|name|bio-biosql|
|version_downloads|2131|

</details>

## bio-hmmer_model

|key|value|
|---|-----|
|info|Parse PFAM HMM definition files so that the models can be accessible programmatically
|url|http://github.com/wwood/bioruby-hmmer_model|
|authors|Ben J. Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-hmmer_model/0.0.2|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-hmmer_model|
|version|0.0.2|
|sha|860549f3db3717ade58766475b1429f348b7c2b3b097f786151b67d1cbe56b7e|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-hmmer_model-0.0.2.gem|
|downloads|7380|
|name|bio-hmmer_model|
|version_downloads|4056|

</details>

## bio-qtlHD

|key|value|
|---|-----|
|info|Ruby tools and scripts for QTL mapping, part of the qtlHD project
|url|http://github.com/pjotrp/bioruby-qtlHD|
|authors|Pjotr Prins, Danny Arends, Karl Broman|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-qtlHD/0.0.1.alpha.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-qtlHD|
|version|0.0.1.alpha.1|
|sha|13c26344c063e8def83b41a6cfc3bd13cc2afad110cb0fc387eed303adda1a13|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-qtlHD-0.0.1.alpha.1.gem|
|downloads|1995|
|name|bio-qtlHD|
|version_downloads|1995|

</details>

## bio-iqtree

|key|value|
|---|-----|
|info|Phylogenetic analysis using IQ-Tree via BioRuby
|url||
|authors|evolbeginner|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-iqtree/0.1.2|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-iqtree|
|version|0.1.2|
|sha|66f0353ebf7f403aed1105d9cf7722e1a3b0955825c0a710295162fc37d23397|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-iqtree-0.1.2.gem|
|downloads|1975|
|name|bio-iqtree|
|version_downloads|1858|

</details>

## bio-hmmer3_report

|key|value|
|---|-----|
|info|Enables programmatic parsing of HMMER version 3 reports
|url|http://github.com/wwood/bioruby-hmmer3_report|
|authors|Christian Zmasek, Ben J Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-hmmer3_report/0.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-hmmer3_report|
|version|0.1.0|
|sha|7fe415b44ed45eaa899a6b9a0caf70f6e3fe20170b281e20e23892838068fd9e|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-hmmer3_report-0.1.0.gem|
|downloads|6538|
|name|bio-hmmer3_report|
|version_downloads|3037|

</details>

## bio-polymarker_db_batch

|key|value|
|---|-----|
|info|Server for polymarker. To be used with cron
|url|http://github.com/homonecloco/bioruby-polymarker_db_batch|
|authors|Ricardo H.  Ramirez-Gonzalez|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-polymarker_db_batch/0.5.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-polymarker_db_batch|
|version|0.5.1|
|sha|fa66abdd7eb0b559b2576a6bc4abd4165f4b3f28c0cea1ab393aef854ef66dd6|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-polymarker_db_batch-0.5.1.gem|
|downloads|20970|
|name|bio-polymarker_db_batch|
|version_downloads|2679|

</details>

## bio-primer3

|key|value|
|---|-----|
|info|Primer3-related Ruby code - wrappers, Boulder IO, etc.
|url|http://github.com/wwood/bioruby-primer3|
|authors|Ben J Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-primer3/0.0.1.pre|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-primer3|
|version|0.0.1.pre|
|sha|c53cc716c84e267660d55991cf3254012900f2f8e6431a165a86f986c6d89dae|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-primer3-0.0.1.pre.gem|
|downloads|1799|
|name|bio-primer3|
|version_downloads|1799|

</details>

## bio-ensembl-rest

|key|value|
|---|-----|
|info|A Ruby library for the RESTful Ensembl API.
|url|http://github.com/ALTree/bio-ensembl-rest|
|authors|Alberto Donizetti|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://github.com/ALTree/bio-ensembl-rest/wiki|
|metadata|{}|
|bug_tracker_uri|https://github.com/ALTree/bio-ensembl-rest/issues|
|project_uri|https://rubygems.org/gems/bio-ensembl-rest|
|version|0.2.0|
|sha|8425c66120938186c35c5fbb9a57dec4081eb9e7d1393a914faab9056ae573d9|
|platform|ruby|
|source_code_uri|https://github.com/ALTree/bio-ensembl-rest|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-ensembl-rest-0.2.0.gem|
|downloads|6272|
|mailing_list_uri||
|name|bio-ensembl-rest|
|wiki_uri|https://github.com/ALTree/bio-ensembl-rest/wiki|
|version_downloads|3500|

</details>

## bio-statsample-glm

|key|value|
|---|-----|
|info|Statsample-GLM is an extension to Statsample, an advance statistics suite in Ruby. This gem includes modules for Regression techniques such as Poisson Regression, Logistic Regression and Exponential Regression
|url|http://github.com/AnkurGel/bioruby-statsample-glm|
|authors|Ankur Goel|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-statsample-glm/0.1.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-statsample-glm|
|version|0.1.1|
|sha|aebf30edd9bf6b0da842b4b10e00cc69a8aaf4f3f4265d0f42f8d734ec674fa0|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-statsample-glm-0.1.1.gem|
|downloads|6167|
|name|bio-statsample-glm|
|version_downloads|3475|

</details>

## muscle_bio

|key|value|
|---|-----|
|info|Multiple Sequence Alignment for Ruby scripts.  Muscle found at https://www.ebi.ac.uk/Tools/msa/muscle/.  For instructions on how to use, see https://github.com/clarkmu/muscle_bio
|url|http://swanstrom.web.unc.edu/|
|authors|Michael Clark|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/muscle_bio/0.5.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/muscle_bio|
|version|0.5.0|
|sha|99b6cdeb4e6a11b83f6c74f0dde6f451eafc5a8d40b3292206b7b718656e5162|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/muscle_bio-0.5.0.gem|
|downloads|6159|
|name|muscle_bio|
|version_downloads|543|

</details>

## bio-samtools-server

|key|value|
|---|-----|
|info|A minimal web service on the top of sinatra to query bam files
|url|http://github.com/homonecloco/bioruby-samtools-server|
|authors|homonecloco|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-samtools-server/0.1.2|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-samtools-server|
|version|0.1.2|
|sha|c5a2414400b2cb8b959dcbed18ba9912b00346c543b68d83c16108a7b9872339|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-samtools-server-0.1.2.gem|
|downloads|5638|
|name|bio-samtools-server|
|version_downloads|3178|

</details>

## bio-restriction_enzyme

|key|value|
|---|-----|
|info|Digests DNA based on restriction enzyme cut patterns
|url|http://github.com/trevor/bioruby-restriction_enzyme|
|authors|Trevor Wennblom|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-restriction_enzyme/1.0.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-restriction_enzyme|
|version|1.0.0|
|sha|f73f3f32b345ac41e143d1be86571f51b1a0b867c7f8a989bcb37d5aab563003|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-restriction_enzyme-1.0.0.gem|
|downloads|4618|
|name|bio-restriction_enzyme|
|version_downloads|4618|

</details>

## bio-core-ext

|key|value|
|---|-----|
|info|BioRuby plugins with dependecies from external library or tools
|url|http://github.com/helios/bioruby-core-ext|
|authors|Raoul J.P. Bonnal, Pjotr Prins|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-core-ext/0.0.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-core-ext|
|version|0.0.1|
|sha|3090fb5771ba36309c18582f9637034ca64503b78ff9254f5fb4f00f6c4a2e99|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-core-ext-0.0.1.gem|
|downloads|4420|
|name|bio-core-ext|
|version_downloads|4420|

</details>

## bio-cd-hit-report

|key|value|
|---|-----|
|info|A Ruby library for reading CD-HIT cluster reports
|url|http://github.com/georgeG/bioruby-cd-hit-report|
|authors|George Githinji|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-cd-hit-report/0.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-cd-hit-report|
|version|0.1.0|
|sha|2e518468a6de52991ec81288b9cefd6ed83a6e76104669c185d633bd2a074282|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-cd-hit-report-0.1.0.gem|
|downloads|13443|
|name|bio-cd-hit-report|
|version_downloads|3942|

</details>

## bio-aliphatic_index

|key|value|
|---|-----|
|info|A simple biogem that allows computation of the aliphatic index of a protein
|url|http://github.com/wwood/bioruby-aliphatic_index|
|authors|Ben J Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-aliphatic_index/0.0.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-aliphatic_index|
|version|0.0.1|
|sha|2b6402750b1d649e11efa6b760d728d080432f4207a4c7c8172443bc81e3b494|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-aliphatic_index-0.0.1.gem|
|downloads|4270|
|name|bio-aliphatic_index|
|version_downloads|4270|

</details>

## bio-twobit

|key|value|
|---|-----|
|info|This is a Ruby binding for lib2bit(https://github.com/dpryan79/lib2bit), which provides high-speed access to genomic data in 2bit file format.
|url|https://github.com/ruby-on-bioc/bio-twobit|
|authors|kojix2|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-twobit/0.1.2|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-twobit|
|version|0.1.2|
|sha|34b37a748e5a434423ac7896ca0db90c0d710b1088c1c639a6c3dc62c93cb380|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-twobit-0.1.2.gem|
|downloads|1189|
|name|bio-twobit|
|version_downloads|623|

</details>

## bio-img_metadata

|key|value|
|---|-----|
|info|Reads metadata from Integrated Microbial Genomes (IMG) metadata files into a programmaticly useful state.
|url|http://github.com/wwood/bioruby-img_metadata|
|authors|Ben J. Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-img_metadata/0.0.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-img_metadata|
|version|0.0.1|
|sha|4a7252f3f1c56aa24337aaf8b92511c009baaa14aa42a3023a5749243e14f13c|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-img_metadata-0.0.1.gem|
|downloads|3877|
|name|bio-img_metadata|
|version_downloads|3877|

</details>

## intermine-bio

|key|value|
|---|-----|
|info|= Biological Extensions to the InterMine Webservice Client Library 

This library is a set of extensions to the InterMine Webservices client, 
providing access for data in biological formats. It directly extends the 
InterMine classes, providing extra methods to the Query class.

== Example

Get all sequences for proteins on "h", "r", "eve", "bib" and "zen":

    require "rubygems"
    require "intermine/service"
    require "intermine/bio"

    s = Service.new("www.flymine.org/query")

    puts s.query("Gene").select("proteins").where(:symbol => %w{h r eve bib zen}).fasta

Process the locations of these genes one at a time:

    s.query.select("Gene").where(:symbol => %w{h r eve bib zen}).bed do |line|
      process line
    end

== Who is this for?

InterMine data warehouses are typically constructed to hold
Biological data, and as this library facilitates programmatic
access to these data, this install is primarily aimed at 
bioinformaticians. In particular, users of the following services
may find it especially useful:
 * FlyMine (http://www.flymine.org/query)
 * YeastMine (http://yeastmine.yeastgenome.org/yeastmine)
 * RatMine (http://ratmine.mcw.edu/ratmine)
 * modMine (http://intermine.modencode.org/release-23)
 * metabolicMine (http://www.metabolicmine.org/beta)

These extensions are aimed at bioinformaticians looking to integrate
these sources of data into other workflows.

For details on constructing queries, see the intermine documentation.

== Support

Support is available on our development mailing list: dev@intermine.org

== License

This code is Open Source under the LGPL. Source code for all InterMine code
can be checked out from svn://subversion.flymine.org/flymine

|url|http://www.intermine.org|
|authors|Alex Kalderimis|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/intermine-bio/0.98.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/intermine-bio|
|version|0.98.1|
|sha|c9a215bd92a7f52d538e5e26a6726b9b6ae81a86cd3a498b8f147af47fb3bc33|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/intermine-bio-0.98.1.gem|
|downloads|4111|
|name|intermine-bio|
|version_downloads|4111|

</details>

## bio-blat-tools

|key|value|
|---|-----|
|info|Tools to manipulate blat files from the command line. Uses the Bio::Blat object from BioRuby, but the parser has been modified to avoid reading the whole psl file, it yields just the current entry and it is up to the consumer to decide to store it or not."
  s.email = "ricardo.ramirez-gonzalez@tgac.ac.uk"
|url|http://github.com/homonecloco/bioruby-blat-tools|
|authors|homonecloco|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-blat-tools/0.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-blat-tools|
|version|0.1.0|
|sha|b859f95aa64ce69f1f10278f46b4d623c2d5be69b7804c3b3bc4c778475b9733|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-blat-tools-0.1.0.gem|
|downloads|3013|
|name|bio-blat-tools|
|version_downloads|3013|

</details>

## bio-location-rdfize

|key|value|
|---|-----|
|info|This is a BioRuby plugin providing FALDO location RDF output method for Bio::Location objects. A method Bio::Location#rdfize is added.
|url|http://github.com/ngoto/bioruby-location-rdfize|
|authors|Joachim Baran, Naohisa Goto|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-location-rdfize/0.0.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-location-rdfize|
|version|0.0.0|
|sha|a48ac28545b9d1335806a49038fe384c9a4df1a10b5a53eeca682cfb7337518d|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-location-rdfize-0.0.0.gem|
|downloads|2985|
|name|bio-location-rdfize|
|version_downloads|2985|

</details>

## bio-sparql-client

|key|value|
|---|-----|
|info|A ruby wrapper for bio SPARQL service
|url|http://rubygems.org/gems/bio-sparql-client|
|authors|Natapol Pornputtapong|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-sparql-client/0.0.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-sparql-client|
|version|0.0.1|
|sha|dfa5c8a7d3d4b17c9d6a283ac6b0d366e96c1daf6380f0df6fa93076dfc075bd|
|platform|ruby|
|licenses|["GPL-3.0"]|
|gem_uri|https://rubygems.org/gems/bio-sparql-client-0.0.1.gem|
|downloads|2844|
|name|bio-sparql-client|
|version_downloads|2844|

</details>

## bio-wolf_psort_wrapper

|key|value|
|---|-----|
|info|Enables the localisation predictor WoLF PSORT to be run locally. The algorithm is available at http://wolfpsort.org/
|url|http://github.com/wwood/bioruby-wolf_psort_wrapper|
|authors|Ben J Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-wolf_psort_wrapper/0.0.3|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-wolf_psort_wrapper|
|version|0.0.3|
|sha|ee1144bda0f73158336222f07f9dae32856ced05345d267d4891f41e55017c4a|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-wolf_psort_wrapper-0.0.3.gem|
|downloads|7822|
|name|bio-wolf_psort_wrapper|
|version_downloads|4291|

</details>

## jandot-bio

|key|value|
|---|-----|
|info|BioRuby is a library for bioinformatics (biology + information science).
|url|http://github.com/jandot/bioruby|
|authors|Jan Aerts|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/jandot-bio/1.2.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/jandot-bio|
|version|1.2.1|
|sha|592fdc936d86b6b51da49d39d39e9863a7b08f7e64747d7e97b8d9c3ae54df1a|
|platform|ruby|
|licenses|[]|
|gem_uri|https://rubygems.org/gems/jandot-bio-1.2.1.gem|
|downloads|2652|
|name|jandot-bio|
|version_downloads|2652|

</details>

## bio-sam-mutation

|key|value|
|---|-----|
|info|Simple classes for parsing SAM, CIGAR and MD:Z strings, including slices. Methods for calling mutations in HGVS format and looking up consequences using Ensembl VEP REST API. Developed for calling mutations at an expected position in an alignment - e.g. Amplicon sequencing of CRISPR-induced mutations.
|url|http://github.com/stveep/bioruby-sam-mutation|
|authors|Stephen Pettitt|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-sam-mutation/0.4.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-sam-mutation|
|version|0.4.1|
|sha|e2b7031dc32fa5bfd535c47be11e71fc8e4310052b991fdbba9c5edea7da8b29|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-sam-mutation-0.4.1.gem|
|downloads|2357|
|name|bio-sam-mutation|
|version_downloads|2357|

</details>

## bio-old-biofetch-emulator

|key|value|
|---|-----|
|info|Emulator that emulates Bio::Fetch object in BioRuby as if old BioRuby BioFetch server were still alive. It overrides methods and objects in Bio::Fetch, and if the old BioRuby BioFetch server's URL is given, it intercepts all requests and converts them into existing web services such as TogoWS, KEGG REST API, NCBI E-Utilities, and GenomeNet(genome.jp).
|url|http://github.com/ngoto/bioruby-old-biofetch-emulator|
|authors|Naohisa Goto|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-old-biofetch-emulator/1.0.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-old-biofetch-emulator|
|version|1.0.0|
|sha|2cebae441cd8d680cbf76df5301b7d45a3d086974cdc016b9148a15dfb8b6d8b|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-old-biofetch-emulator-1.0.0.gem|
|downloads|5950|
|name|bio-old-biofetch-emulator|
|version_downloads|3468|

</details>

## bio-express_beta_diversity

|key|value|
|---|-----|
|info|Interface for express beta diversity file formats
|url|http://github.com/wwood/bioruby-express_beta_diversity|
|authors|Ben J. Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-express_beta_diversity/0.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-express_beta_diversity|
|version|0.1.0|
|sha|18e1cf7c26e4d23a40550c38aa0f56986abe1cb71bc774ed0d21f53990396dd3|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-express_beta_diversity-0.1.0.gem|
|downloads|5944|
|name|bio-express_beta_diversity|
|version_downloads|3308|

</details>

## bio-blast-xmlparser

|key|value|
|---|-----|
|info|BioRuby BLAST XMLParser plugin: a plugin of BioRuby BLAST XML result parser by using the XMLParser extension. This was formerly a part of the BioRuby core ("bio" gem) and split to this gem because of the external XMLParser dependency.
|url|https://github.com/bioruby/bioruby-blast-xmlparser|
|authors|BioRuby project|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-blast-xmlparser/0.0.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-blast-xmlparser|
|version|0.0.0|
|sha|854ffaab93955fd32fbc41b1bdd294922d306f2c8f775a4ff5ddf065afc2ec07|
|platform|ruby|
|licenses|[]|
|gem_uri|https://rubygems.org/gems/bio-blast-xmlparser-0.0.0.gem|
|downloads|2034|
|name|bio-blast-xmlparser|
|version_downloads|2034|

</details>

## bio-bigwig

|key|value|
|---|-----|
|info|This is a Ruby binding for libBigWig (https://github.com/dpryan79/libBigWig), which provides high-speed access to bigWig or bigBed files.
|url|https://github.com/kojix2/bw-ruby|
|authors|kojix2|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-bigwig/0.0.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-bigwig|
|version|0.0.1|
|sha|9f592f32b6d99eba6c5f09d947b18f07d9e8722c95cdb66309aeea9a823f7d21|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-bigwig-0.0.1.gem|
|downloads|579|
|name|bio-bigwig|
|version_downloads|579|

</details>

## bio-data-restriction_enzyme

|key|value|
|---|-----|
|info|Restriction Enzyme Dataset from REBASE in yaml form
|url|http://github.com/trevor/bioruby-data-restriction_enzyme|
|authors|Sir Richard J. Roberts, Trevor Wennblom|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-data-restriction_enzyme/1.109.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-data-restriction_enzyme|
|version|1.109.0|
|sha|0b4698b20a2c082ad1137a2d491d52d9541ea14b0a4e9bdb52158d96a511181e|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-data-restriction_enzyme-1.109.0.gem|
|downloads|4369|
|name|bio-data-restriction_enzyme|
|version_downloads|4369|

</details>

## lorem_ipsum_bio

|key|value|
|---|-----|
|info|Lorem Ipsum Bio is a toolkit for generating varied random biological sequences (especially usable as a control group in analysis)
|url|https://github.com/pjanek/lorem_ipsum_bio|
|authors|Pawe Jankowski|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/lorem_ipsum_bio/0.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/lorem_ipsum_bio|
|version|0.1.0|
|sha|af3da433aabd069dae00109cdf8ac97730acce19bf32234aa2f85d9dada132fa|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/lorem_ipsum_bio-0.1.0.gem|
|downloads|7752|
|name|lorem_ipsum_bio|
|version_downloads|1596|

</details>

## bio-blast_seq_extract

|key|value|
|---|-----|
|info|Extract the sequence of the subjects' region based on blast search.
|url||
|authors|Sishuo Wang|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-blast_seq_extract/0.2.5|
|metadata|{"allowed_push_host"=>"https://rubygems.org"}|
|project_uri|https://rubygems.org/gems/bio-blast_seq_extract|
|version|0.2.5|
|sha|bd2e8acf736b66471edde2437def59a7242b65fbac79a7ad98371aeb32f471c8|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-blast_seq_extract-0.2.5.gem|
|downloads|3575|
|name|bio-blast_seq_extract|
|version_downloads|1571|

</details>

## bio-cgranges

|key|value|
|---|-----|
|info|Genomic interval overlap queries
|url|https://github.com/ruby-on-bioc/bio-cgranges|
|authors|kojix2|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-cgranges/0.0.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-cgranges|
|version|0.0.1|
|sha|5d540839a77505727cb13224e85192d4c91edc9c0d7cc51e0811645970e8c00f|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/bio-cgranges-0.0.1.gem|
|downloads|379|
|name|bio-cgranges|
|version_downloads|199|

</details>

## bio-emboss_six_frame_nucleotide_sequences

|key|value|
|---|-----|
|info|a method to get the nucleotide sequence of translations done by the EMBOSS bioinformatics package program transeq.
|url|http://github.com/wwood/bioruby-emboss_six_frame_nucleotide_sequences|
|authors|Ben J Woodcroft|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bio-emboss_six_frame_nucleotide_sequences/0.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bio-emboss_six_frame_nucleotide_sequences|
|version|0.1.0|
|sha|8dc290e6c37f6e6eeba2433910d54c811bc513ba10304ca26b74a1c650588c02|
|platform|ruby|
|gem_uri|https://rubygems.org/gems/bio-emboss_six_frame_nucleotide_sequences-0.1.0.gem|
|downloads|4442|
|name|bio-emboss_six_frame_nucleotide_sequences|
|version_downloads|4442|

</details>

## geni-svgenes

|key|value|
|---|-----|
|info|Modified version of the bio-svgenes gem, for use in https://geni-act.org education website.
|url|http://github.com/southpawlife/geni-svgenes|
|authors|Kristopher Williams|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/geni-svgenes/1.1.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/geni-svgenes|
|version|1.1.0|
|sha|34f65f06b33d0451e4e1dff25645d18b14f0f17a80799581462e8714f2a543b9|
|platform|ruby|
|licenses|["MIT"]|
|gem_uri|https://rubygems.org/gems/geni-svgenes-1.1.0.gem|
|downloads|2103|
|name|geni-svgenes|
|version_downloads|2076|

</details>

## bioruby-phyloxml

|key|value|
|---|-----|
|info|Provides PhyloXML support for BioRuby. This bioruby-phyloxml gem replaces old unmaintained bio-phyloxml gem.
|url|http://github.com/bioruby/bioruby-phyloxml|
|authors|Diana Jaunzeikare, Clayton Wheeler, BioRuby project|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bioruby-phyloxml/1.0.0|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bioruby-phyloxml|
|version|1.0.0|
|sha|00d87cba440c80812ad94ae6af455e18df52a39216126ddad66e385b4c80fc46|
|platform|ruby|
|licenses|["Ruby"]|
|gem_uri|https://rubygems.org/gems/bioruby-phyloxml-1.0.0.gem|
|downloads|3129|
|name|bioruby-phyloxml|
|version_downloads|3129|

</details>

## eutils

|key|value|
|---|-----|
|info|Lightweight Ruby API for NCBI Eutils. This gem only provides API for NCBI Eutils. If you need full access to other NCBI databases, try BioRuby (bio gem) instead.
|url|http://github.com/seouri/eutils|
|authors|Joon Lee, aka seouri|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/eutils/0.2.1|
|metadata|{}|
|project_uri|https://rubygems.org/gems/eutils|
|version|0.2.1|
|sha|ed151b01ecc2f9c7e321afab0a7b4c228c3b3eb7c736a98bc57b2e0b80c92ca3|
|platform|ruby|
|licenses|[]|
|gem_uri|https://rubygems.org/gems/eutils-0.2.1.gem|
|downloads|16393|
|name|eutils|
|version_downloads|3344|

</details>

## bioroebe

|key|value|
|---|-----|
|info|
Welcome to the bioroebe-0.10.x release series! It is recommended to
use ruby 3.x for this series, as it is the most tested version of 
ruby in regards to this project.

The bioroebe-project is a software suite focusing on bio-related
life sciences in general - in particular molecular biology, molecular
genetics, bioinformatics, biotechnology, system biology, synthetic
biology and related topics close to the life sciences in general.

The primary objective for the bioroebe project is to provide a
"practical glue" between these different topics while also focusing
on "getting real work done", as a toolset-project, trying to
efficiently solve existing problems.

For additional information about this project, have a look at the
link called "documentation" on the rubygems webpage of this gem,
on the very bottom right side (The direct URL is: 
https://www.rubydoc.info/gems/bioroebe/ ).


|url|https://rubygems.org/gems/bioroebe|
|authors|Robert A. Heiler|

<details>
<summary><b>details</b></summary>

|key|value|
|---|-----|
|documentation_uri|https://www.rubydoc.info/gems/bioroebe/0.10.79|
|metadata|{}|
|project_uri|https://rubygems.org/gems/bioroebe|
|version|0.10.79|
|sha|9d948e4ac01d6a9d05433634a8ded48726dcbb2092c3a16a69d7afbb899ebc60|
|platform|ruby|
|licenses|["LGPL-2.1"]|
|gem_uri|https://rubygems.org/gems/bioroebe-0.10.79.gem|
|downloads|276514|
|name|bioroebe|
|version_downloads|159|

</details>

